<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Masahiro  Kanai | Publications</title>
    <meta name="description" content="Research Fellow at the Massachusetts General Hospital & the Broad Institute of MIT and Harvard.
">
    <!-- Open Graph -->
    <!-- Bootstrap & MDB -->
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />
    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&display=swap">
    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.statically.io/gh/jwarby/pygments-css/master/github.css" />
    <!-- Styles -->
    <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>ðŸ§¬</text></svg>">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/publications/">
  <!-- Webp polyfill -->
  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=HTMLPictureElement" crossorigin="anonymous"></script>
  <!-- JQuery -->
  <!-- jQuery -->
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
  <!-- Theming-->
  <script src="/assets/js/theme.js"></script>
  <!-- Load DarkMode JS -->
  <script src="/assets/js/dark_mode.js"></script>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-BFKLBV2ZNN"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'G-BFKLBV2ZNN');
  </script>
</head>
<body class="fixed-top-nav ">
  <!-- Header -->
  <header>
    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
      <div class="container">
        <a class="navbar-brand title font-weight-lighter" href="https://mkanai.github.io/">
          <span class="font-weight-bold">Masahiro</span>   Kanai
        </a>
        <!-- Navbar Toggle -->
        <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar top-bar"></span>
          <span class="icon-bar middle-bar"></span>
          <span class="icon-bar bottom-bar"></span>
        </button>
        <div class="collapse navbar-collapse text-right" id="navbarNav">
          <ul class="navbar-nav ml-auto flex-nowrap">
            <!-- About -->
            <li class="nav-item ">
              <a class="nav-link" href="/">
                About
              </a>
            </li>
            <!-- CV -->
            <li class="nav-item ">
              <a class="nav-link" href="/cv/">
                CV
              </a>
            </li>
            <!-- Other pages -->
            <li class="nav-item active">
              <a class="nav-link" href="/publications/">
                Publications
                <span class="sr-only">(current)</span>
              </a>
            </li>
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                <i class="fas fa-globe"></i>
                English
              </a>
              <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                <a class="nav-link dropdown-item" href="/ja/publications/">æ—¥æœ¬èªž</a>
              </div>
            </li>
            <div class = "toggle-container">
              <a id = "light-toggle">
                <i class="fas fa-moon"></i>
                <i class="fas fa-sun"></i>
              </a>
            </div>
          </ul>
        </div>
      </div>
    </nav>
  </header>
  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>
  <!-- Content -->
  <div class="container mt-5">
    <div class="post">
      <header class="post-header">
        <h1 class="post-title">Publications</h1>
        <p class="post-description"></p>
      </header>
      <article>
        <div class="row">
          <div class="col-sm-9">
            <ul class="list-unstyled mb-0">
              <li>
                <p>Selected publications are listed <a href="/">here</a>. <b>*</b> denotes equal contribution</p>
              </li>
              <li>Link to
                <a href="https://scholar.google.com/citations?user=jb3AUMsAAAAJ" target="_blank" rel="noopener"><i class="ai ai-google-scholar"></i> Google Scholar</a> | 
                <a href="https://orcid.org/0000-0001-5165-4408" target="_blank" rel="noopener"><i class="ai ai-orcid"></i> ORCID</a> | 
                <a href="https://www.ncbi.nlm.nih.gov/myncbi/masahiro.kanai.1/bibliography/public/" target="_blank" rel="noopener"><i class="ai ai-pubmed"></i> PubMed</a>
              </li>
              <li>Jump to
                <a href="#Preprints">Preprints</a> |
                <a href="#2026">2026</a> |
                <a href="#2025">2025</a> |
                <a href="#2024">2024</a> |
                <a href="#2023">2023</a> |
                <a href="#2022">2022</a> |
                <a href="#2021">2021</a> |
                <a href="#2020">2020</a> |
                <a href="#2019">2019</a> |
                <a href="#2018">2018</a> |
                <a href="#2017">2017</a> |
                <a href="#2016">2016</a> |
                <a href="#2015">2015</a> |
              </li>
            </ul>
          </div>
          <div class="col-sm-3 mt-auto pt-3">
            <input type="text" class="form-control" id="search-input" placeholder="Search..." />
          </div>
        </div>
        <div class="publications">
          <h2 class="year" id="Preprints">Preprints</h2>
          <ol class="bibliography reset">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2025.11.25.25340489" target="_blank" rel="noopener">Population-scale multiome immune cell atlas reveals complex disease drivers</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Delorey, T. M.,
                    </span>
                    <span class="coauthor">
                      Honkanen, J.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Rodosthenous, R. S.,
                      Juvila, J.,
                      Murphy, S.,
                      Teixeira-Soldano, I.,
                      Hwang, H. S.,
                      Karjalainen, J.,
                      Halonen, J.,
                      Panagiotaropoulou, G.,
                      Zhang, Y.,
                      McCabe, C.,
                      Chen, E.,
                      Nanki, K.,
                      Yoshida, T.,
                      Liu, K.,
                      Glean, M.,
                      Mehrotra, N.,
                      Finan, E. P.,
                      Chafamo, D.,
                      Zhu, Y.,
                      Arvas, M.,
                      Ruotsalainen, S.,
                      Zheng, Z.,
                      FinnGen.,
                      Reeve, M. P.,
                      Kurki, M.,
                      Porter, C. B. M.,
                      Ashenberg, O.,
                      Zhou, W.,
                      PitkÃ¤nen, K.,
                      Partanen, J.,
                      Palotie, A.,
                      Graham, D. B.,
                      Daly, M. J.,
                    </span>
                    and Xavier, R. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>medRxiv</em>
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2025.11.25.25340489" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2025.11.25.25340489" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Most genetic variants associated with complex diseases lie in non-coding regions, yet mechanistic insights have been limited by the lack of an empirical framework for characterizing the molecular consequences of regulatory variation. Single-cell profiling of molecular quantitative trait loci (QTL) can connect variants to gene regulation, but prior studies lacked the sample size to detect variants at disease-relevant genes and the simultaneous measurements across regulatory layers needed to trace complete mechanisms from chromatin state to gene expression. Here we show that population-scale simultaneous profiling of chromatin accessibility and gene expression across immune cell types reveals multi-layered regulatory pathways connecting genetic variants to disease. We generated paired single-nucleus ATAC-seq and RNA-seq profiles from 10 million peripheral blood mononuclear cells across 1,108 Finnish individuals, identifying 51,083 cis-eQTLs for 20,829 genes, 338,100 cis-caQTLs for 210,584 peaks, 119,094 fine-mapped variants, and 496,488 enhancerâ€“gene links. Systematic classification of regulatory mechanisms revealed that variants with complete chromatin-to-expression cascades show twice the disease colocalization of chromatin-only effects, establishing a hierarchy where mechanistic cascade predicts disease relevance. Analysis of evolutionarily constrained genes revealed multi-layered regulatory buffering where chromatin accessibility changes occur with normal effect sizes, but transmission to gene expression is attenuated through systematically weaker enhancerâ€“gene links, reconciling why disease variants preferentially target these genes despite apparent eQTL depletion. We incorporated base editing to experimentally validate causal variants and mechanisms at Finnish-enriched disease loci such as TNRC18. This resource provides testable mechanistic hypotheses for over half of immune disease associations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2025.11.25.25340489" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Liu2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2025.03.12.642879" target="_blank" rel="noopener">Sensitive Transcriptomics and Genotyping reveals function of genetic variants in immunity</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Liu, K.,
                    </span>
                    <span class="coauthor">
                      Rasmussen, A.,
                    </span>
                    <span class="coauthor">
                      Han, W.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Gong, Q.,
                      Bohley, S.,
                      Chen, D.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Maronick, R.,
                      Yin, Z.,
                      Hu, C.,
                      Jain, T.,
                      Martin-Rufino, J. D.,
                      Sankaran, V. G.,
                      Ashenberg, O.,
                      Daly, M. J.,
                      Chen, F.,
                      Graham, D. B.,
                    </span>
                    and Xavier, R. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>bioRxiv</em>
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2025.03.12.642879" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2025.03.12.642879" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Advancements in genomics have revolutionized human genetics by defining the genetic architecture of disease. However, identifying causal variants and their mechanisms of action remains a challenge in translating genetics into therapeutic interventions. Here, we developed Sensitive Transcriptomics And Genotyping by sequencing (STAG-seq), a high-throughput platform designed to define mechanistic genotype-phenotype relationships through simultaneous single-cell measurements of genomic DNA and RNA transcripts. Combined with base-editing, STAG-seq enables functionalization of variants in relevant cellular contexts. We demonstrate the applicability of this approach in several settings. First, we screened genetic perturbations to identify monoallelic and biallelic variant effects in primary human macrophages treated with innate immune stimuli. Next, we phenotyped clinically relevant missense variants associated with immunodeficiency and autoimmunity. Finally, we defined a noncoding variant in a pleiotropic autoimmunity locus that governs TNRC18 expression in primary T cells. STAG-seq thus enables variant phenotyping at scale to advance functional genomics and disease biology.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2025.03.12.642879" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Finucane2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2024.09.17.613327" target="_blank" rel="noopener">Variant scoring performance across selection regimes depends on variant-to-gene and gene-to-disease components</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Finucane, H. K.,
                    </span>
                    <span class="coauthor">
                      Parsa, S.,
                    </span>
                    <span class="coauthor">
                      Guez, J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Satterstrom, F. K.,
                      Nkambule, L. L.,
                      Daly, M. J.,
                      Seed, C.,
                    </span>
                    and Karczewski, K. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>bioRxiv</em>
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2024.09.17.613327" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2024.09.17.613327" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Variant scoring methods (VSMs) aid in the interpretation of coding mutations and their potential impact on health, but their evaluation in the context of human genetics applications remains inconsistent. Here, we describe GeneticsGym, a systematic approach to evaluating the real-world impact of VSMs on human genetic analysis across selection regimes. We show that the relative performance of VSMs varies across the spectrum of variant impact, as well as by gene function, and that both variant-to-gene and gene-to-disease components contribute.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2024.09.17.613327" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Zhou2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2024.05.15.24307317" target="_blank" rel="noopener">Efficient and accurate mixed model association tool for single-cell eQTL analysis</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Zhou, W.,
                    </span>
                    <span class="coauthor">
                      Cuomo, A.,
                    </span>
                    <span class="coauthor">
                      Xue, A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Chau, G.,
                      Krishna, C.,
                      Xavier, R. J.,
                      MacArthur, D. G.,
                      Powell, J. E.,
                      Daly, M. J.,
                    </span>
                    and Neale, B. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>medRxiv</em>
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2024.05.15.24307317" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <a href="https://github.com/weizhou0/qtl" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://zenodo.org/records/10811106" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2024.05.15.24307317" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Understanding the genetic basis of gene expression can help us understand the molecular underpinnings of human traits and disease. Expression quantitative trait locus (eQTL) mapping can help in studying this relationship but have been shown to be very cell-type specific, motivating the use of single-cell RNA sequencing and single-cell eQTLs to obtain a more granular view of genetic regulation. Current methods for single-cell eQTL mapping either rely on the pseudobulk approach and traditional pipelines for bulk transcriptomics or do not scale well to large datasets. Here, we propose SAIGE-QTL, a robust and scalable tool that can directly map eQTLs using single-cell profiles without needing aggregation at the pseudobulk level. Additionally, SAIGE-QTL allows for testing the effects of less frequent/rare genetic variation through set-based tests, which is traditionally excluded from eQTL mapping studies. We evaluate the performance of SAIGE-QTL on both real and simulated data and demonstrate the improved power for eQTL mapping over existing pipelines.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2024.05.15.24307317" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2021.09.03.21262975" target="_blank" rel="noopener">Insights from complex trait fine-mapping across diverse populations</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Ulirsch, J. C.,
                    </span>
                    <span class="coauthor">
                      Karjalainen, J.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kurki, M.,
                      Karczewski, K. J.,
                      Fauman, E. B.,
                      Wang, Q. S.,
                      Jacobs, H.,
                      Aguet, F.,
                      Ardlie, K. G.,
                      Kerimov, N.,
                      Alasoo, K.,
                      Benner, C.,
                      Ishigaki, K.,
                      Sakaue, S.,
                      Reilly, S.,
                      The BioBank Japan Project.,
                      FinnGen.,
                      Kamatani, Y.,
                      Matsuda, K.,
                      Palotie, A.,
                      Neale, B. M.,
                      Tewhey, R.,
                      Sabeti, P. C.,
                      Okada, Y.,
                      Daly, M. J.,
                    </span>
                    and Finucane, H. K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>medRxiv</em>
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2021.09.03.21262975" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <a href="https://github.com/mkanai/finemapping-insights" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.finucanelab.org/data" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2021.09.03.21262975" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Despite the great success of genome-wide association studies (GWAS) in identifying genetic loci significantly associated with diseases, the vast majority of causal variants underlying disease-associated loci have not been identified. To create an atlas of causal variants, we performed and integrated fine-mapping across 148 complex traits in three large-scale biobanks (BioBank Japan, FinnGen, and UK Biobank; total n = 811,261), resulting in 4,518 variant-trait pairs with high posterior probability (&gt; 0.9) of causality. Of these, we found 285 high-confidence variant-trait pairs replicated across multiple populations, and we characterized multiple contributors to the surprising lack of overlap among fine-mapping results from different biobanks. By studying the bottlenecked Finnish and Japanese populations, we identified 21 and 26 putative causal coding variants with extreme allele frequency enrichment (&gt; 10-fold) in these two populations, respectively. Aggregating data across populations enabled identification of 1,492 unique fine-mapped coding variants and 176 genes in which multiple independent coding variants influence the same trait (i.e., with an allelic series of coding variants). Our results demonstrate that fine-mapping in diverse populations enables novel insights into the biology of complex traits by pinpointing high-confidence causal variants for further characterization.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2021.09.03.21262975" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Koskela2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2021.05.07.21255988" target="_blank" rel="noopener">Genetic variant in SPDL1 reveals novel mechanism linking pulmonary fibrosis risk and cancer protection</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Koskela, J. T.,
                    </span>
                    <span class="coauthor">
                      Happola, P.,
                    </span>
                    <span class="coauthor">
                      Liu, A.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      FinnGen.,
                      Partanen, J.,
                      Genovese, G.,
                      Artomov, M.,
                      Myllymaki, M. N. M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, W.,
                      Karjalainen, J.,
                      Palviainen, T.,
                      Ronkainen, J.,
                      Sebert, S.,
                      Tukiainen, T.,
                      Palta, P.,
                      Kaprio, J.,
                      Kurki, M.,
                      Ganna, A.,
                      Palotie, A.,
                      Laitinen, T.,
                      Myllarniemi, M.,
                    </span>
                    and Daly, M. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>medRxiv</em>
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2021.05.07.21255988" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2021.05.07.21255988" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Idiopathic Pulmonary Fibrosis (IPF) is a rare disease with poor prognosis. By contrast, cancer is common in any elderly population and a leading killer, but is now often curable. Of note, whereas IPF is driven by cellular senescence, cancer is characterized by uncontrolled cell division. Using data available from two large biobank-based studies (Finnish FinnGen study and UK biobank), we conducted a comprehensive analysis of the shared genetic background of IPF and cancer. In a population sample of 218,792 Finns with complete longitudinal health histories, we estimated the effect of individual genetic variants to the lifetime risk of IPF and cancer. We extend the analysis from IPF-GWAS to pan-cancer meta-analysis over FinnGen and UK Biobank and finally to the identification of genetic drivers of somatic chromosomal alterations. We detected six loci (SPDL1, MAD1L1, MAP2K1, RTEL1-STMN3, TERC-ACTRT3, OBFC1) associated with both IPF and cancer, all closely related to cellular division. However, each individual signal is found with opposite effects over the two diseases, termed as antagonistic pleiotropy. Several of these loci (TERC-ACTRT3, RTEL1-STMN3, OBFC1) are among the strongest inherited factors for constitutive telomere length variation and consistently indicate that shorter telomere length would increase the risk for IPF but protect from malignancy. However, a Finnish enriched SPDL1 missense variant and a common MAD1L1 intronic variant had no effect on telomere length but were shown to protect individuals from accumulation of somatic mutations. The decreased risk of cancer in SPDL1 and MAD1L1 variant carriers might result from a lower number of chromosomal alterations accumulated over time, conversely leading to fibrosis in the lung due to cellular senescence-induced inflammation. We hypothesize that the SPDL1 missense variant functions as gain-of-function mutation, leading to cellular senescence, a barrier to cancer and a driver of fibrosis in IPF. If translated to therapy, these findings might not only be able to offer relief to individuals with IPF, but also to protect from onset of cancer.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2021.05.07.21255988" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Turley2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1101/2021.04.23.441003" target="_blank" rel="noopener">Multi-Ancestry Meta-Analysis yields novel genetic discoveries and ancestry-specific associations</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Turley, P.,
                    </span>
                    <span class="coauthor">
                      Martin, A. R.,
                    </span>
                    <span class="coauthor">
                      Goldman, G.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Li, H.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Walters, R. K.,
                      Jala, J. B.,
                      Lin, K.,
                      Millwood, I. Y.,
                      Carey, C. E.,
                      Palmer, D. S.,
                      Zacher, M.,
                      Atkinson, E. G.,
                      Chen, Z.,
                      Li, L.,
                      Akiyama, M.,
                      Okada, Y.,
                      Kamatani, Y.,
                      Walters, R. G.,
                      Callier, S.,
                      Laibson, D.,
                      Meyer, M. N.,
                      Cesarini, D.,
                      Daly, M.,
                      Benjamin, D. J.,
                    </span>
                    and Neale, B. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>bioRxiv</em>
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://doi.org/10.1101/2021.04.23.441003" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PDF</a>
                    <a href="https://github.com/JonJala/mama" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1101/2021.04.23.441003" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>We present a new method, Multi-Ancestry Meta-Analysis (MAMA), which combines genome-wide association study (GWAS) summary statistics from multiple populations to produce new summary statistics for each population, identifying novel loci that would not have been discovered in either set of GWAS summary statistics alone. In simulations, MAMA increases power with less bias and generally lower type-1 error rate than other multi-ancestry meta-analysis approaches. We apply MAMA to 23 phenotypes in East-Asian- and European-ancestry populations and find substantial gains in power. In an independent sample, novel genetic discoveries from MAMA replicate strongly. ### Competing Interest Statement A.R.M has consulted for 23andMe and Illumina, and she has received speaker fees from Genentech, Illumina, and Pfizer. B.M.N. is a member of the scientific advisory board at Deep Genomics and RBNC Therapeutics, Member of the scientific advisory committee at Milken and a consultant for Camp4 Therapeutics, Takeda Pharmaceutical and Biogen. D.S.P. is an employee of Genomics plc, all contributions were performed prior to him joining the company.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/2021.04.23.441003" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2026">2026</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Reeve2026" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-026-02521-1" target="_blank" rel="noopener">Genome-wide association analyses of autoimmune hypothyroidism reveal autoimmune and thyroid-specific contributions and an inverse relationship with cancer risk</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Reeve, M. P.,
                    </span>
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Graham, D. B.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karjalainen, J.,
                      Luo, S.,
                      Kolosov, N.,
                      Adams, C.,
                      Ritari, J.,
                      Karczewski, K. J.,
                      Kiiskinen, T.,
                      Jiang, Y.,
                      Fuller, Z.,
                      Mehtonen, J.,
                      Kurki, M. I.,
                      Khan, Z.,
                      FinnGen.,
                      Daly, M. J.,
                      Partanen, J.,
                      McCarthy, M. I.,
                      Artomov, M.,
                      Palotie, A.,
                      Tuomi, T.,
                      Pirinen, M.,
                      Kero, J.,
                      Xavier, R. J.,
                      Daly, M. J.,
                    </span>
                    and Ripatti, S.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    (2026)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/e5P2E" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-026-02521-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The high prevalence (&gt;5%) of autoimmune hypothyroidism (AIHT) provides a unique opportunity to dissect genetic contributions to systemic and organ-specific autoimmunity. Here we performed a genome-wide association meta-analysis of 81,718 AIHT cases in FinnGen and the UK Biobank, identifying 418 independent signals ( P âˆ’8 ). At 48 of these loci, a protein-coding variant is, or is highly correlated ( r 2 &gt; 0.95) with, the lead variant, including Finnish-enriched coding variants in LAG3 , ZAP70 and TG . We demonstrated that ZAP70 :T155M reduces T cell activation and broadly compare large-scale scans of nonthyroid autoimmunity and thyroid-stimulating hormone levels with a Bayesian classifier to assign loci into distinct groupings, estimating that 38% are involved in general autoimmunity whereas 20% are thyroid specific. We further identified substantial antagonistic pleiotropy, with 10% of AIHT loci showing a consistent protective effect against skin cancer. The AIHT results, including numerous genes encoding checkpoint proteins, support the causal role of natural immune variation influencing cancer outcomes.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-026-02521-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Siraj2026" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-026-10121-6" target="_blank" rel="noopener">Functional dissection of complex trait variants at single-nucleotide resolution</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Siraj, L.,
                    </span>
                    <span class="coauthor">
                      Castro, R. I.,
                    </span>
                    <span class="coauthor">
                      Dewey, H. B.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kales, S.,
                      Butts, J. C.,
                      Nguyen, T. T. L.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Berenzy, D.,
                      Mouri, K.,
                      Wang, Q. S.,
                      Fiziev, P. P.,
                      Tsuo, K.,
                      McCaw, Z. R.,
                      Gosai, S. J.,
                      Aguet, F.,
                      Cui, R.,
                      Kassam, I.,
                      McRae, J.,
                      Vockley, C. M.,
                      Lareau, C. A.,
                      Abramov, S.,
                      Boystov, A.,
                      Vierstra, J.,
                      Okada, Y.,
                      Gusev, A.,
                      Jones, T. R.,
                      Lander, E. S.,
                      Sabeti, P. C.,
                      Finucane, H. K.,
                      Reilly, S. K.,
                      Ulirsch, J. C.,
                    </span>
                    and Tewhey, R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    (2026)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/e5PZS" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://doi.org/10.1101/2024.05.05.592437" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://github.com/julirsch/finemapped_mpra/" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://doi.org/10.5281/zenodo.15297965" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-026-10121-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Identifying the causal variants and mechanisms that drive complex traits and diseases remains a core problem in human genetics. Most of these variants individually have weak effects 6 and lie in non-coding gene-regulatory elements, for which we lack a complete understanding of how single-nucleotide alterations modulate transcriptional processes to affect human phenotypes. To address this problem, we measured the activity of 221,412 fine-mapped trait-associated variants using a massively parallel reporter assay in 5 diverse cell types. We show that this assay effectively discriminates between likely causal variants and controls, and identified 13,121 regulatory variants with high precision. Although the effects of these variants largely agree with orthogonal measures of function, only 69% of them can plausibly be explained by the disruption of a known transcription factor binding motif. We investigated the mechanisms of 136 variants using saturation mutagenesis and assigned affected transcription factors for 91% of variants without a clear canonical mechanism. Finally, we detected regulatory epistasis at 11% of tested regulatory variants in close proximity and identified multiple functional variants on the same haplotype at a small, but important, subset of trait-associated loci. Overall, our study provides a systematic functional characterization of likely causal common variants that underlie complex and molecular human traits, enabling new insights into the regulatory grammar underlying disease risk.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-026-10121-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sato2026" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41591-026-04213-z" target="_blank" rel="noopener">Genetic regulation across germline and somatic variation on the Y chromosome contributes to type 2 diabetes</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Sato, G.,
                    </span>
                    <span class="coauthor">
                      Yamamoto, Y.,
                    </span>
                    <span class="coauthor">
                      Sonehara, K.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Saiki, R.,
                      Ojima, T.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Liu, A.,
                      Edahiro, R.,
                      Shirai, Y.,
                      Namba, S.,
                      Namkoong, H.,
                      Hasaegawa, T.,
                      Koyanagi, Y. N.,
                      Kasugai, Y.,
                      Yamaji, T.,
                      Nakano, S.,
                      Genovese, G.,
                      SipilÃ¤, T. P.,
                      Ghazal, A.,
                      Tanaka, H.,
                      Azekawa, S.,
                      Uwamino, Y.,
                      Yamamoto, K.,
                      Suzuki, K.,
                      Hata, T.,
                      Uemura, M.,
                      Takeda, Y.,
                      Kanai, A.,
                      Higashiue, S.,
                      Kobayashi, S.,
                      Afuso, H.,
                      Matsuura, K.,
                      Mitsumoto, Y.,
                      Fujita, Y.,
                      FinnGen.,
                      the Japan COVID-19 Task Force.,
                      the Biobank Japan Project.,
                      Oda, Y.,
                      Suzuki, Y.,
                      Morisaki, T.,
                      Ishii, M.,
                      Kitagawa, Y.,
                      Koike, R.,
                      Kimura, A.,
                      Imoto, S.,
                      Miyano, S.,
                      Kanai, T.,
                      Takayama, J.,
                      Iwasaki, M.,
                      Sawada, N.,
                      Fukunaga, K.,
                      Matsuo, K.,
                      Kumanogoh, A.,
                      Doki, Y.,
                      Eguchi, H.,
                      Nakagome, S.,
                      Tamiya, G.,
                      Ganna, A.,
                      Palotie, A.,
                      Daly, M. J.,
                      Wilson, J. F.,
                      Yamamoto, M.,
                      Matsuda, K.,
                      Ogawa, S.,
                      Yamauchi, T.,
                      Kadowaki, T.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Medicine</em>
                    (2026)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/e5kjX" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/41731096" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41591-026-04213-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Our understanding of the biological role of the Y chromosome remains limited. Here, we systematically profile germline Y haplogroups and somatic loss of the Y chromosome (LOY) in 122,683 East Asian males from BioBank Japan and 181,472 European males from the UK Biobank. A phenome-wide scan uncovers male-specific genetic regulation of complex traits, including pleiotropic effects of the Japanese-specific haplogroup D on height and type 2 diabetes (T2D). LOY increases T2D risk in East Asians but is associated with reduced T2D risk in Europeans. In East Asians, LOY contributes to T2D incidence particularly among males with lower polygenic risk scores, providing a compensatory explanation for disease risk beyond germline genetics. Incorporating sex-chromosome variation improves polygenic prediction of T2D risk in both sexes. Single-cell analyses reveal cell type-specific accumulation of LOY across tissues and disease contexts, with LOY in pancreatic Î² cells potentially impairing glucose metabolism. Our study demonstrates the clinical relevance of Y chromosome variation for diabetes risk prediction and management. A genetic study in over 300,000 males reveals that Y chromosome loss affects type 2 diabetes risk differently in East Asians and Europeans. Multi-omics data suggest that this may be due to impaired glucose metabolism in Y-deficient pancreatic Î² cells.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41591-026-04213-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Rossen2026" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-025-02450-5" target="_blank" rel="noopener">MultiSuSiE improves multi-ancestry fine-mapping in All of Us whole-genome sequencing data</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Rossen, J.,
                    </span>
                    <span class="coauthor">
                      Shi, H.,
                    </span>
                    <span class="coauthor">
                      Strober, B. J.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhang, M. J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      McCaw, Z. R.,
                      Liang, L.,
                      Weissbrod, O.,
                    </span>
                    and Price, A. L.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>58</b>,
                    67â€“76
                    (2026)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/eYVdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/41491094" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2024.05.13.24307291" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/jordanero/MultiSuSiE" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://doi.org/10.5281/zenodo.17370173" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-025-02450-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Leveraging multi-ancestry data can improve fine-mapping power. We propose MultiSuSiE, an extension of Sum of Single Effects (SuSiE), to multiple ancestries that allows causal effect sizes to vary across ancestries. We evaluated MultiSuSiE using whole-genome sequencing data from 47,000 African-ancestry, 36,000 Latino-ancestry and 116,000 European-ancestry individuals from All of Us. In simulations, MultiSuSiE applied to Afr36k + Lat36k + Eur36k was well-calibrated and attained higher power than SuSiE applied to Eur109k; compared to recent multi-ancestry methods (SuSiEx and MESuSiE), MultiSuSiE attained higher power and lower computational cost. In analyses of 14 quantitative traits, MultiSuSiE applied to Afr47k + Lat36k + Eur116k identified 348 fine-mapped variants with posterior inclusion probability (PIP) &gt; 0.9, and MultiSuSiE applied to Afr36k + Lat36k + Eur36k identified 59% more PIP &gt; 0.9 variants than SuSiE applied to Eur109k; MultiSuSiE identified 29% more PIP &gt; 0.9 variants than SuSiEx, and MESuSiE was not included due to its high computational cost. We validated these findings through functional enrichment of fine-mapped variants and highlighted examples implicating biologically plausible fine-mapped variants.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-025-02450-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2025">2025</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Jacobs2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-025-65476-7" target="_blank" rel="noopener">Widespread naturally variable human exons aid genetic interpretation</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Jacobs, H. N.,
                    </span>
                    <span class="coauthor">
                      Gorissen, B. L.,
                    </span>
                    <span class="coauthor">
                      Guez, J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Gupta, K.,
                      Finucane, H. K.,
                      Karczewski, K. J.,
                    </span>
                    and Burge, C. B.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>16</b>,
                    11345
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/eUUVt" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/41402286" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2024.09.09.612029" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-025-65476-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Most mammalian genes undergo alternative splicing. The splicing of some exons has been acquired or lost in specific mammalian lineages, but differences in splicing within the human population are poorly understood. Using GTEx tissue transcriptomes from 838 individuals, we identified 57,271 â€œnaturally variable exonsâ€ (NVEs) - exons which are included in mRNAs in some individuals but entirely excluded from others (or vice versa). NVEs impact three quarters of protein-coding genes, occur at all population frequencies, and are often absent from reference annotations. NVEs are more abundant in genes depleted of genetic loss-of-function mutations and aid in the interpretation of causal genetic variants. Genetic variants modulate the splicing of many NVEs, and 5â€™ untranslated region and coding-region NVEs are often associated with increased and decreased gene expression, respectively. Together, our findings characterize abundant splicing variation in the human population, with implications for a range of human genetic analyses.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-025-65476-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Karczewski2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-025-02335-7" target="_blank" rel="noopener">Pan-UK Biobank GWAS improves discovery, analysis of genetic architecture, and resolution into ancestry-enriched effects</a></div>
                  <div class="author">
                    <span class="coauthor">
                      *Karczewski, K. J.,
                    </span>
                    <span class="coauthor">
                      *Gupta, R.,
                    </span>
                    <em>*Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Lu, W.,
                      Tsuo, K.,
                      Wang, Y.,
                      Walters, R. K.,
                      Turley, P.,
                      Callier, S.,
                      Shah, N. N.,
                      Baya, N.,
                      Palmer, D. S.,
                      Goldstein, J. I.,
                      Sarma, G.,
                      Solomonson, M.,
                      Cheng, N.,
                      Bryant, S.,
                      Churchhouse, C.,
                      Cusick, C. M.,
                      Poterba, T.,
                      Compitello, J.,
                      King, D.,
                      Zhou, W.,
                      Seed, C.,
                      Finucane, H. K.,
                      Daly, M. J.,
                      Neale, B. M.,
                      Atkinson, E. G.,
                    </span>
                    and Martin, A. R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>57</b>,
                    2408â€“2417
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/eG9Kz" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/40968291" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2024.03.13.24303864" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/atgu/ukbb_pan_ancestry" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://pan.ukbb.broadinstitute.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-025-02335-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Large biobanks, such as the UK Biobank (UKB), enable massive phenome by genome-wide association studies that elucidate genetic etiology of complex traits. However, people from diverse genetic ancestry groups are often excluded from association analyses due to concerns about population structure introducing false positive associations. Here we generate mixed model associations and meta-analyses across genetic ancestry groups, inclusive of a larger fraction of the UK Biobank than previous efforts, to produce freely available summary statistics for 7,266 traits. We build a quality control and analysis framework informed by genetic architecture. Overall, we identify 14,676 significant loci (P &lt; 5 Ã— 10âˆ’8) in the meta-analysis that were not found in the EUR genetic ancestry group alone, including new associations, for example between CAMK2D and triglycerides. We also highlight associations from ancestry-enriched variation, including a known pleiotropic missense variant in G6PD associated with several biomarker traits. We release these results publicly alongside frequently asked questions that describe caveats for interpretation of results, enhancing available resources for interpretation of risk variants across diverse populations. Genome-wide analyses for 7,266 traits leveraging data from several genetic ancestry groups in UK Biobank identify new associations and enhance resources for interpreting risk variants across diverse populations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-025-02335-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Smit2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41591-025-03827-z" target="_blank" rel="noopener">Polygenic prediction of body mass index and obesity through the life course and across ancestries</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Smit, R. A. J.,
                    </span>
                    <span class="coauthor">
                      Wade, K. H.,
                    </span>
                    <span class="coauthor">
                      Hui, Q.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Arias, J. D.,
                      Yin, X.,
                      Christiansen, M. R.,
                      Yengo, L.,
                      Preuss, M. H.,
                      Nakabuye, M.,
                      Rocheleau, G.,
                      Graham, S. E.,
                      Buchanan, V. L.,
                      Chittoor, G.,
                      Graff, M.,
                      Guindo-MartÃ­nez, M.,
                      Lu, Y.,
                      Marouli, E.,
                      Sakaue, S.,
                      Spracklen, C. N.,
                      Vedantam, S.,
                      Wilson, E. P.,
                      Chen, S.-H.,
                      Ferreira, T.,
                      Ji, Y.,
                      Karaderi, T.,
                      LÃ¼ll, K.,
                      Machado, M.,
                      Malden, D. E.,
                      Medina-Gomez, C.,
                      Moore, A.,
                      RÃ¼eger, S.,
                      Akiyama, M.,
                      Allison, M. A.,
                      Alvarez, M.,
                      Andersen, M. K.,
                      Appadurai, V.,
                      Arbeeva, L.,
                      Bartell, E.,
                      Bhaskar, S.,
                      Bielak, L. F.,
                      Bis, J. C.,
                      Bollepalli, S.,
                      Bork-Jensen, J.,
                      Bradfield, J. P.,
                      Bradford, Y.,
                      Brandl, C.,
                      Braund, P. S.,
                      Brody, J. A.,
                      Broeckel, U.,
                      Burgdorf, K. S.,
                      Cade, B. E.,
                      Cai, Q.,
                      Camarda, S.,
                      Campbell, A.,
                      CaÃ±adas-Garre, M.,
                      Chai, J.-F.,
                      Chesi, A.,
                      Choi, S. H.,
                      Christofidou, P.,
                      Couture, C.,
                      Cuellar-Partida, G.,
                      Danning, R.,
                      Degenhardt, F.,
                      Delgado, G. E.,
                      Delitala, A.,
                      Demirkan, A.,
                      Deng, X.,
                      Dietl, A.,
                      Dimitriou, M.,
                      Dimitrov, L.,
                      Dorajoo, R.,
                      Eichelmann, F.,
                      Eliasen, A. U.,
                      Engmann, J. E.,
                      Erdos, M. R.,
                      Fairhurst-Hunter, Z.,
                      Farmaki, A.-E.,
                      Faul, J. D.,
                      Fernandez-Lopez, J.-C.,
                      Forer, L.,
                      Frank, M.,
                      Freitag-Wolf, S.,
                      Fritsche, L. G.,
                      Fuchsberger, C.,
                      Galesloot, T. E.,
                      Gao, Y.,
                      Geller, F.,
                      Giannakopoulou, O.,
                      Giulianini, F.,
                      Gjesing, A. P.,
                      Goel, A.,
                      Gordon, S. D.,
                      Gorski, M.,
                      Grove, J.,
                      Guo, X.,
                      Gustafsson, S.,
                      Haessler, J.,
                      Hansen, T. F.,
                      Havulinna, A. S.,
                      Haworth, S. J.,
                      Heard-Costa, N.,
                      Hemerich, D.,
                      Highland, H. M.,
                      Hindy, G.,
                      Ho, Y.-L.,
                      Hofer, E.,
                      Holliday, E.,
                      Horn, K.,
                      Hornsby, W. E.,
                      Hottenga, J.-J.,
                      Huang, H.,
                      Huang, J.,
                      Huerta-Chagoya, A.,
                      Huo, S.,
                      Hwang, M. Y.,
                      Hwu, C.-M.,
                      Iha, H.,
                      Ikeda, D. D.,
                      Isono, M.,
                      Jackson, A. U.,
                      Jansen, I. E.,
                      Jiang, Y.,
                      Johansson, I.,
                      Jonsson, A.,
                      JÃ¸rgensen, T.,
                      Kalafati, I. P.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kanoni, S.,
                      KÃ¥rhus, L. L.,
                      Kasturiratne, A.,
                      Katsuya, T.,
                      Kawaguchi, T.,
                      Kember, R. L.,
                      Kentistou, K. A.,
                      Kim, D.,
                      Kim, H.-N.,
                      Kim, Y. J.,
                      Kleber, M. E.,
                      Knol, M. J.,
                      Kurbasic, A.,
                      Lauzon, M.,
                      Le, P.,
                      Lea, R.,
                      Lee, J.-Y.,
                      Lee, W.-J.,
                      Leonard, H. L.,
                      Li, H.,
                      Li, S. A.,
                      Li, X.,
                      Li, X.,
                      Liang, J.,
                      Lin, H.,
                      Lin, K.,
                      Liu, J.,
                      Liu, X.,
                      Lo, K. S.,
                      Long, J.,
                      Lores-Motta, L.,
                      Luan, J.,
                      Lyssenko, V.,
                      LyytikÃ¤inen, L.-P.,
                      Mahajan, A.,
                      Malik, M. Z.,
                      Mamakou, V.,
                      Mangino, M.,
                      Manichaikul, A.,
                      Marten, J.,
                      Mattheisen, M.,
                      McDaid, A. F.,
                      Mei, Q.,
                      Meiselbach, H.,
                      Melendez, T. L.,
                      Milaneschi, Y.,
                      Miller, J. E.,
                      Millwood, I. Y.,
                      Mishra, P. P.,
                      Mitchell, R. E.,
                      MÃ¸llehave, L. T.,
                      Mononen, N.,
                      Mucha, S.,
                      Munz, M.,
                      MykkÃ¤nen, J.,
                      Nakatochi, M.,
                      Nardone, G. G.,
                      Nelson, C. P.,
                      Nethander, M.,
                      Nho, C. W.,
                      Nielsen, A. A.,
                      Nolte, I. M.,
                      Nongmaithem, S. S.,
                      Noordam, R.,
                      Ntalla, I.,
                      Nutile, T.,
                      Pandit, A.,
                      Pauper, M.,
                      Petersen, E. R. B.,
                      Petersen, L. V.,
                      Piluso, F.,
                      PolaÅ¡ek, O.,
                      Poveda, A.,
                      Pyarajan, S.,
                      Raffield, L. M.,
                      Rakugi, H.,
                      Ramirez, J.,
                      Rasheed, A.,
                      Raven, D.,
                      Rayner, N. W.,
                      Riveros, C.,
                      Rohde, R.,
                      Ruggiero, D.,
                      Ruotsalainen, S. E.,
                      Ryan, K. A.,
                      Sabater-Lleal, M.,
                      Santin, A.,
                      Saxena, R.,
                      Scholz, M.,
                      Shen, B.,
                      Shi, J.,
                      Shin, J. H.,
                      Sidore, C.,
                      Sidorenko, J.,
                      Sim, X.,
                      Slieker, R. C.,
                      Smith, A. V.,
                      Smith, J. A.,
                      Smyth, L. J.,
                      Southam, L.,
                      Steinthorsdottir, V.,
                      Sun, L.,
                      Takeuchi, F.,
                      Taylor, K. D.,
                      Tayo, B. O.,
                      Tcheandjieu, C.,
                      Terzikhan, N.,
                      Tesolin, P.,
                      Teumer, A.,
                      Theusch, E.,
                      Thompson, D. J.,
                      Thorleifsson, G.,
                      Timmers, P. R. H. J.,
                      Trompet, S.,
                      Turman, C.,
                      Vaccargiu, S.,
                      Laan, S. W.,
                      Most, P. J.,
                      Klinken, J. B.,
                      Setten, J.,
                      Verma, S. S.,
                      Verweij, N.,
                      Veturi, Y.,
                      Wang, C. A.,
                      Wang, C.,
                      Wang, J.-S.,
                      Wang, L.,
                      Wang, Y. X.,
                      Wang, Z.,
                      Warren, H. R.,
                      Bin Wei, W.,
                      Wen, W.,
                      Wheeler, W. A.,
                      Wickremasinghe, A. R.,
                      Wielscher, M.,
                      Winsvold, B. S.,
                      Wong, A.,
                      Wuttke, M.,
                      Xia, R.,
                      Yamamoto, K.,
                      Yang, J.,
                      Yao, J.,
                      Young, H.,
                      Yousri, N. A.,
                      Yu, L.,
                      Zeng, L.,
                      Zhang, W.,
                      Zhang, X.,
                      Zhao, J.-H.,
                      Zhao, W.,
                      Zhou, W.,
                      Zimmermann, M. E.,
                      Zoledziewska, M.,
                      Hart, L. M.,
                      Adair, L. S.,
                      Adams, H. H. H.,
                      Aguilar-Salinas, C. A.,
                      Al-Mulla, F.,
                      Arnett, D. K.,
                      Asselbergs, F. W.,
                      Ã…svold, B. O.,
                      Attia, J.,
                      Banas, B.,
                      Bandinelli, S.,
                      Beilin, L. J.,
                      Bennett, D. A.,
                      Bergler, T.,
                      Bharadwaj, D.,
                      Biino, G.,
                      Boerwinkle, E.,
                      BÃ¶ger, C. A.,
                      Borja, J. B.,
                      Bouchard, C.,
                      Bowden, D. W.,
                      Brandslund, I.,
                      Brumpton, B.,
                      Buring, J. E.,
                      Caulfield, M. J.,
                      Chambers, J. C.,
                      Chandak, G. R.,
                      Chanock, S. J.,
                      Chaturvedi, N.,
                      Ida Chen, Y.-D.,
                      Chen, Z.,
                      Cheng, C.-Y.,
                      Cho, Y. S.,
                      Christensen, K.,
                      Christophersen, I. E.,
                      Ciullo, M.,
                      Cole, J. W.,
                      Collins, F. S.,
                      Concas, M. P.,
                      Cooper, R. S.,
                      Cruz, M.,
                      Cucca, F.,
                      Cutler, M. J.,
                      Damrauer, S. M.,
                      Dantoft, T. M.,
                      Borst, G. J.,
                      Geus, E. J. C.,
                      Groot, L. C. P. G. M.,
                      De Jager, P. L.,
                      Kleijn, D. P. V.,
                      Silva, H. J.,
                      Dedoussis, G. V.,
                      Hollander, A. I.,
                      Du, S.,
                      Easton, D. F.,
                      Eckardt, K.-U.,
                      Elders, P. J. M.,
                      Eliassen, A. H.,
                      Ellinor, P. T.,
                      ElmstÃ¥hl, S.,
                      Erdmann, J.,
                      Evans, M. K.,
                      Fatkin, D.,
                      Feenstra, B.,
                      Feitosa, M. F.,
                      Ferrucci, L.,
                      Florez, J. C.,
                      Ford, I.,
                      Fornage, M.,
                      Franke, A.,
                      Franks, P. W.,
                      Freedman, B. I.,
                      Gieger, C.,
                      Girotto, G.,
                      Golightly, Y. M.,
                      Gonzalez-Villalpando, C.,
                      Gordon-Larsen, P.,
                      Grallert, H.,
                      Grant, S. F. A.,
                      Grarup, N.,
                      Griffiths, L.,
                      Gudnason, V.,
                      Haiman, C.,
                      Hakonarson, H.,
                      Hansen, T.,
                      Hartman, C. A.,
                      Hattersley, A. T.,
                      Hayward, C.,
                      Heid, I. M.,
                      Heng, C.-K.,
                      Hengstenberg, C.,
                      Herzig, K.-H.,
                      Hewitt, A. W.,
                      Hishigaki, H.,
                      Hougaard, D. M.,
                      Hoyng, C. B.,
                      Huang, P. L.,
                      Huang, W.,
                      Huang, W.-Y.,
                      Huffman, J. E.,
                      Hunt, S. C.,
                      Hutri, N.,
                      Hveem, K.,
                      HyppÃ¶nen, E.,
                      Iacono, W. G.,
                      Ichihara, S.,
                      Ikram, M. A.,
                      Isasi, C. R.,
                      Jarvelin, M.-R.,
                      Jin, Z.-B.,
                      JÃ¶ckel, K.-H.,
                      Jonas, J. B.,
                      Joshi, P. K.,
                      Jousilahti, P.,
                      Jukema, J. W.,
                      KÃ¤hÃ¶nen, M.,
                      Kamatani, Y.,
                      Kang, K. D.,
                      Kaprio, J.,
                      Kardia, S. L. R.,
                      Karpe, F.,
                      Kato, N.,
                      Kavousi, M.,
                      Kee, F.,
                      Kessler, T.,
                      Khera, A. V.,
                      Khor, C. C.,
                      Kiemeney, L. A. L. M.,
                      Kim, B.-J.,
                      Kim, E. K.,
                      Kim, H.-L.,
                      Kirchhof, P.,
                      Kivimaki, M.,
                      Koh, W.-P.,
                      Koistinen, H. A.,
                      Kokkinos, A.,
                      Kooner, J. S.,
                      Kooperberg, C.,
                      Kovacs, P.,
                      Kraaijeveld, A.,
                      Kraft, P.,
                      Krauss, R. M.,
                      Kumari, M.,
                      Kutalik, Z.,
                      Laakso, M.,
                      Lange, L. A.,
                      Langenberg, C.,
                      Launer, L. J.,
                      Lee, H.,
                      Lee, N. R.,
                      LehtimÃ¤ki, T.,
                      Lemaitre, R. N.,
                      Li, H.,
                      Li, L.,
                      Lieb, W.,
                      Lin, X.,
                      Lind, L.,
                      Linneberg, A.,
                      Liu, C.-T.,
                      Liu, J.,
                      Loeffler, M.,
                      London, B.,
                      Lu, F.,
                      Lubitz, S. A.,
                      Mackey, D. A.,
                      Magnusson, P. K. E.,
                      Manson, J. E.,
                      Marcus, G. M.,
                      Marques Vidal, P.,
                      Martin, N. G.,
                      MÃ¤rz, W.,
                      Matsuda, F.,
                      McCarthy, M. I.,
                      McGarrah, R. W.,
                      McGue, M.,
                      McKnight, A. J.,
                      Medland, S. E.,
                      MellstrÃ¶m, D.,
                      Metspalu, A.,
                      Mitchell, B. D.,
                      Mitchell, P.,
                      Mook-Kanamori, D. O.,
                      Mori, T. A.,
                      Morris, A. D.,
                      Mucci, L. A.,
                      Munroe, P. B.,
                      Nalls, M. A.,
                      Nazarian, S.,
                      Nelson, A. E.,
                      Neville, M. J.,
                      Newton-Cheh, C.,
                      Nielsen, C. S.,
                      Niinikoski, H.,
                      Nikus, K.,
                      NÃ¶then, M. M.,
                      Ogunniyi, A.,
                      Ohlsson, C.,
                      Oldehinkel, A. J.,
                      Orozco, L.,
                      Pahkala, K.,
                      Pajukanta, P.,
                      Palmer, C. N. A.,
                      Parra, E. J.,
                      Pattaro, C.,
                      Pedersen, O.,
                      Pennell, C. E.,
                      Penninx, B. W. J. H.,
                      Perusse, L.,
                      Peters, A.,
                      Peyser, P. A.,
                      Porteous, D. J.,
                      Posthuma, D.,
                      Power, C.,
                      Pramstaller, P. P.,
                      Province, M. A.,
                      Psaty, B. M.,
                      Qi, Q.,
                      Qu, J.,
                      Rader, D. J.,
                      Raitakari, O. T.,
                      Rallidis, L. S.,
                      Rao, D. C.,
                      Redline, S.,
                      Reilly, D. F.,
                      Reiner, A. P.,
                      Rhee, S. Y.,
                      Ridker, P. M.,
                      Rienstra, M.,
                      Ripatti, S.,
                      Ritchie, M. D.,
                      Rivadeneira, F.,
                      Roden, D. M.,
                      Rosendaal, F. R.,
                      Rotter, J. I.,
                      Rudan, I.,
                      Rutters, F.,
                      Ryu, S.,
                      Sabanayagam, C.,
                      Salako, B.,
                      Saleheen, D.,
                      Salomaa, V.,
                      Samani, N. J.,
                      Sanghera, D. K.,
                      Sattar, N.,
                      Schmidt, B.,
                      Schmidt, H.,
                      Schmidt, R.,
                      Schulze, M. B.,
                      Schunkert, H.,
                      Scott, L. J.,
                      Scott, R. J.,
                      Sever, P.,
                      Sheu, W. H. H.,
                      Shoemaker, M. B.,
                      Shu, X.-O.,
                      Simonsick, E. M.,
                      Sims, M.,
                      Singleton, A. B.,
                      Sinner, M. F.,
                      Smith, J. G.,
                      Snieder, H.,
                      Spector, T. D.,
                      Spedicati, B.,
                      Stampfer, M. J.,
                      Stark, K. J.,
                      Strachan, D. P.,
                      Tabara, Y.,
                      Tai, E. S.,
                      Tang, H.,
                      Tardif, J.-C.,
                      Thanaraj, T. A.,
                      TÃ¶njes, A.,
                      Tuomi, T.,
                      Tuomilehto, J.,
                      TusiÃ©-Luna, M.-T.,
                      Dam, R. M.,
                      Harst, P.,
                      Velde, N.,
                      Duijn, C. M.,
                      Schoor, N. M.,
                      Vitart, V.,
                      Vohl, M.-C.,
                      VÃ¶lker, U.,
                      Vollenweider, P.,
                      VÃ¶lzke, H.,
                      Vrieze, S.,
                      Wacher-Rodarte, N. H.,
                      Walker, M.,
                      Wander, G. S.,
                      Wareham, N. J.,
                      Watanabe, R. M.,
                      Watkins, H.,
                      Weir, D. R.,
                      Werge, T. M.,
                      Widen, E.,
                      Willemsen, G.,
                      Willett, W. C.,
                      Wilson, J. F.,
                      Wilson, P. W. F.,
                      Wong, T. Y.,
                      Woo, J.-T.,
                      Wright, A. F.,
                      Xu, H.,
                      Yajnik, C. S.,
                      Yang, J.,
                      Yokota, M.,
                      Yuan, J.-M.,
                      Zeggini, E.,
                      Zemel, B. S.,
                      Zheng, W.,
                      Zhu, X.,
                      Zillikens, M. C.,
                      Zonderman, A. B.,
                      Zwart, J.-A.,
                      23andMe Research Team.,
                      DiscovEHR (DiscovEHR and MyCode Community Health Initiative).,
                      eMERGE (Electronic Medical Records and Genomics Network).,
                      GPC-UGR.,
                      The PRACTICAL Consortium.,
                      Understanding Society Scientific Group.,
                      VA Million Veteran Program.,
                      Abecasis, G. R.,
                      Assimes, T. L.,
                      Auton, A.,
                      Boehnke, M.,
                      Chasman, D. I.,
                      Esko, T.,
                      Stefansson, K.,
                      Lettre, G.,
                      Lindgren, C. M.,
                      Ng, M. C. Y.,
                      Oâ€™Donnell, C. J.,
                      Thorsteinsdottir, U.,
                      Visscher, P. M.,
                      Walters, R. G.,
                      Winkler, T. W.,
                      Wood, A. R.,
                      Deloukas, P.,
                      Frayling, T. M.,
                      Justice, A. E.,
                      KilpelÃ¤inen, T. O.,
                      Locke, A. E.,
                      Mohlke, K. L.,
                      North, K. E.,
                      Okada, Y.,
                      Willer, C. J.,
                      Young, K. L.,
                      Fatumo, S.,
                      McCaffery, J. M.,
                      Timpson, N. J.,
                      Hirschhorn, J. N.,
                      Sun, Y. V.,
                      Berndt, S. I.,
                    </span>
                    and Loos, R. J. F.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Medicine</em>
                    <b>31</b>,
                    3151â€“3168
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/exaF6" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/40691366" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41591-025-03827-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic scores (PGSs) for body mass index (BMI) may guide early prevention and targeted treatment of obesity. Using genetic data from up to 5.1 million people (4.6% African ancestry, 14.4% American ancestry, 8.4% East Asian ancestry, 71.1% European ancestry and 1.5% South Asian ancestry) from the GIANT consortium and 23andMe, Inc., we developed ancestry-specific and multi-ancestry PGSs. The multi-ancestry score explained 17.6% of BMI variation among UK Biobank participants of European ancestry. For other populations, this ranged from 16% in East Asian-Americans to 2.2% in rural Ugandans. In the ALSPAC study, children with higher PGSs showed accelerated BMI gain from age 2.5 years to adolescence, with earlier adiposity rebound. Adding the PGS to predictors available at birth nearly doubled explained variance for BMI from age 5 onward (for example, from 11% to 21% at age 8). Up to age 5, adding the PGS to early-life BMI improved prediction of BMI at age 18 (for example, from 22% to 35% at age 5). Higher PGSs were associated with greater adult weight gain. In intensive lifestyle intervention trials, individuals with higher PGSs lost modestly more weight in the first year (0.55 kg per s.d.) but were more likely to regain it. Overall, these data show that PGSs have the potential to improve obesity prediction, particularly when implemented early in life.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41591-025-03827-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Lammi2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-025-02100-w" target="_blank" rel="noopener">Genome-wide association study of long COVID</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Lammi, V.,
                    </span>
                    <span class="coauthor">
                      Nakanishi, T.,
                    </span>
                    <span class="coauthor">
                      Jones, S. E.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Andrews, S. J.,
                      Karjalainen, J.,
                      CortÃ©s, B.,
                      Oâ€™Brien, H. E.,
                      Ochoa-Guzman, A.,
                      Fulton-Howard, B. E.,
                      Broberg, M.,
                      Haapaniemi, H. H.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Pirinen, M.,
                      Schmidt, A.,
                      Mitchell, R. E.,
                      Mousas, A.,
                      Mangino, M.,
                      Huerta-Chagoya, A.,
                      Sinnott-Armstrong, N.,
                      Cirulli, E. T.,
                      Vaudel, M.,
                      Kwong, A. S. F.,
                      Maiti, A. K.,
                      Marttila, M. M.,
                      Posner, D. C.,
                      Rodriguez, A. A.,
                      Batini, C.,
                      Minnai, F.,
                      Dearman, A. R.,
                      Warmerdam, C. A. R.,
                      Sequeros, C. B.,
                      Winkler, T. W.,
                      Jordan, D. M.,
                      ReÅ¡cenko, R.,
                      Miano, L.,
                      Lane, J. M.,
                      Chung, R. K.,
                      Guillen-Guio, B.,
                      Leavy, O. C.,
                      Carvajal-Silva, L.,
                      Aguilar-ValdÃ©s, K.,
                      Frangione, E.,
                      Guare, L.,
                      Vergasova, E.,
                      Marouli, E.,
                      Striano, P.,
                      Zainulabid, U. A.,
                      Kumar, A.,
                      Ahmad, H. F.,
                      Edahiro, R.,
                      Azekawa, S.,
                      Long COVID Host Genetics Initiative.,
                      FinnGen.,
                      VA Million Veteran Program.,
                      MexGen-COVID Initiative.,
                      DBDS Genomic Consortium.,
                      GEN-COVID Multicenter Study.,
                      PHOSP-COVID Collaborative Group.,
                      GENCOV Study.,
                      Estonian Biobank Research Team.,
                      Luoh, S.-W.,
                      Erikstrup, C.,
                      Pedersen, O. B. V.,
                      Lerner-Ellis, J.,
                      Colombo, A.,
                      Grzymski, J. J.,
                      Ishii, M.,
                      Okada, Y.,
                      Beckmann, N. D.,
                      Kumari, M.,
                      Wagner, R.,
                      Heid, I. M.,
                      John, C.,
                      Short, P. J.,
                      Magnus, P.,
                      Ansone, L.,
                      Valenti, L. V. C.,
                      Lee, S. A.,
                      Wain, L. V.,
                      Verdugo, R. A.,
                      Banasik, K.,
                      Geller, F.,
                      Franke, L. H.,
                      Rakitko, A.,
                      Duncan, E. L.,
                      Renieri, A.,
                      Tsilidis, K. K.,
                      Cid, R.,
                      Niavarani, A.,
                      Abner, E.,
                      TusiÃ©-Luna, T.,
                      Verma, S. S.,
                      Smith, G. D.,
                      Timpson, N. J.,
                      Madduri, R. K.,
                      Cho, K.,
                      Daly, M. J.,
                      Ganna, A.,
                      Schulte, E. C.,
                      Richards, J. B.,
                      Ludwig, K. U.,
                      Marks-HultstrÃ¶m, M.,
                      Zeberg, H.,
                    </span>
                    and Ollila, H. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>57</b>,
                    1402â€“1417
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/em16G" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/40399555" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2023.06.29.23292056" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-025-02100-w" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Infections can lead to persistent symptoms and diseases such as shingles after varicella zoster or rheumatic fever after streptococcal infections. Similarly, severe acute respiratory syndrome coronavirus 2 (SARSâ€‘CoVâ€‘2) infection can result in long coronavirus disease (COVID), typically manifesting as fatigue, pulmonary symptoms and cognitive dysfunction. The biological mechanisms behind long COVID remain unclear. We performed a genome-wide association study for long COVID including up to 6,450 long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We discovered an association of FOXP4 with long COVID, independent of its previously identified association with severe COVID-19. The signal was replicated in 9,500 long COVID cases and 798,835 population controls. Given the transcription factor FOXP4â€™s role in lung physiology and pathology, our findings highlight the importance of lung function in the pathophysiology of long COVID. A genome-wide study by the Long COVID Host Genetics Initiative identifies an association between the FOXP4 locus and long COVID, implicating altered lung function in its pathophysiology.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-025-02100-w" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sonehara2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2025.100783" target="_blank" rel="noopener">Germline variants and mosaic chromosomal alterations affect COVID-19 vaccine immunogenicity</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Sonehara, K.,
                    </span>
                    <span class="coauthor">
                      Uwamino, Y.,
                    </span>
                    <span class="coauthor">
                      Saiki, R.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Takeshita, M.,
                      Namba, S.,
                      Uno, S.,
                      Nakanishi, T.,
                      Nishimura, T.,
                      Naito, T.,
                      Sato, G.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Liu, A.,
                      Uchida, S.,
                      Kurafuji, T.,
                      Tanabe, A.,
                      Arai, T.,
                      Ohno, A.,
                      Shibata, A.,
                      Tanaka, S.,
                      Wakui, M.,
                      Kashimura, S.,
                      Tomi, C.,
                      Hara, A.,
                      Yoshikawa, S.,
                      Gotanda, K.,
                      Misawa, K.,
                      Tanaka, H.,
                      Azekawa, S.,
                      Wang, Q. S.,
                      Edahiro, R.,
                      Shirai, Y.,
                      Yamamoto, K.,
                      Nagao, G.,
                      Suzuki, T.,
                      Kiyoshi, M.,
                      Ishii-Watabe, A.,
                      Higashiue, S.,
                      Kobayashi, S.,
                      Yamaguchi, H.,
                      Okazaki, Y.,
                      Matsumoto, N.,
                      Masumoto, A.,
                      Koga, H.,
                      Kanai, A.,
                      Japan COVID-19 Task Force.,
                      Biobank Japan Project.,
                      Oda, Y.,
                      Suzuki, Y.,
                      Matsuda, K.,
                      Kitagawa, Y.,
                      Koike, R.,
                      Kimura, A.,
                      Kumanogoh, A.,
                      Yoshimura, A.,
                      Imoto, S.,
                      Miyano, S.,
                      Kanai, T.,
                      Fukunaga, K.,
                      Hasegawa, N.,
                      Murata, M.,
                      Matsushita, H.,
                      Ogawa, S.,
                      Okada, Y.,
                    </span>
                    and Namkoong, H.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>5</b>,
                    100783
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(25)00039-4" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/40043710" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2025.100783" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Vaccine immunogenicity is influenced by the vaccineeâ€™s genetic background. Here, we perform a genome-wide association study of vaccine-induced SARS-CoV-2-specific immunoglobulin G (IgG) antibody titers and T cell immune responses in 1,559 mRNA-1273 and 537 BNT162b2 vaccinees of Japanese ancestry. SARS-CoV-2-specific antibody titers are associated with the immunoglobulin heavy chain (IGH) and major histocompatibility complex (MHC) locus, and T cell responses are associated with MHC. The lead variants at IGH contain a population-specific missense variant (rs1043109-C; p.Leu192Val) in the immunoglobulin heavy constant gamma 1 gene (IGHG1), with a strong decreasing effect (Î² = -0.54). Antibody-titer-associated variants modulate circulating immune regulatory proteins (e.g., LILRB4 and FCRL6). Age-related hematopoietic expanded mosaic chromosomal alterations (mCAs) affecting MHC and IGH also impair antibody production. MHC-/IGH-affecting mCAs confer infectious and immune disease risk, including sepsis and Gravesâ€™ disease. Impacts of expanded mosaic loss of chromosomes X/Y on these phenotypes were examined. Altogether, both germline and somatic mutations contribute to adaptive immunity functions.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2025.100783" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Adams2025" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.cell.2024.12.002" target="_blank" rel="noopener">Trans-ancestry genome-wide study of depression identifies 697 associations implicating cell types and pharmacotherapies</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Adams, M. J.,
                    </span>
                    <span class="coauthor">
                      Streit, F.,
                    </span>
                    <span class="coauthor">
                      Meng, X.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Awasthi, S.,
                      Adey, B. N.,
                      Choi, K. W.,
                      Chundru, V. K.,
                      Coleman, J. R. I.,
                      Ferwerda, B.,
                      Foo, J. C.,
                      Gerring, Z. F.,
                      Giannakopoulou, O.,
                      Gupta, P.,
                      Hall, A. S. M.,
                      Harder, A.,
                      Howard, D. M.,
                      HÃ¼bel, C.,
                      Kwong, A. S. F.,
                      Levey, D. F.,
                      Mitchell, B. L.,
                      Ni, G.,
                      Ota, V. K.,
                      Pain, O.,
                      Pathak, G. A.,
                      Schulte, E. C.,
                      Shen, X.,
                      Thorp, J. G.,
                      Walker, A.,
                      Yao, S.,
                      Zeng, J.,
                      Zvrskovec, J.,
                      Aarsland, D.,
                      Actkins, K. V.,
                      Adli, M.,
                      Agerbo, E.,
                      Aichholzer, M.,
                      Aiello, A.,
                      Air, T. M.,
                      Als, T. D.,
                      Andersson, E.,
                      Andlauer, T. F. M.,
                      Arolt, V.,
                      Ask, H.,
                      BÃ¤ckman, J.,
                      Badola, S.,
                      Ballard, C.,
                      Banasik, K.,
                      Bass, N. J.,
                      Beekman, A. T. F.,
                      Belangero, S.,
                      Bigdeli, T. B.,
                      Binder, E. B.,
                      Bjerkeset, O.,
                      Bjornsdottir, G.,
                      BÃ¸rte, S.,
                      BrÃ¤nn, E.,
                      Braun, A.,
                      Brodersen, T.,
                      BrÃ¼ckl, T. M.,
                      Brunak, S.,
                      Bruun, M. T.,
                      Burmeister, M.,
                      Buspavanich, P.,
                      Bybjerg-Grauholm, J.,
                      Byrne, E. M.,
                      Cai, J.,
                      Campbell, A.,
                      Campbell, M. L.,
                      Campos, A. I.,
                      Castelao, E.,
                      Cervilla, J.,
                      Chaumette, B.,
                      Chen, C.-Y.,
                      Chen, H.-C.,
                      Chen, Z.,
                      Cichon, S.,
                      Colodro-Conde, L.,
                      Corbett, A.,
                      Corfield, E. C.,
                      Couvy-Duchesne, B.,
                      Craddock, N.,
                      Dannlowski, U.,
                      Davies, G.,
                      Geus, E. J. C.,
                      Deary, I. J.,
                      Degenhardt, F.,
                      Dehghan, A.,
                      DePaulo, J. R.,
                      Deuschle, M.,
                      Didriksen, M.,
                      Dinh, K. M.,
                      Direk, N.,
                      Djurovic, S.,
                      Docherty, A. R.,
                      Domschke, K.,
                      Dowsett, J.,
                      Drange, O. K.,
                      Dunn, E. C.,
                      Eaton, W.,
                      Einarsson, G.,
                      Eley, T. C.,
                      Elsheikh, S. S. M.,
                      Engelmann, J.,
                      Benros, M. E.,
                      Erikstrup, C.,
                      Escott-Price, V.,
                      Fabbri, C.,
                      Fang, Y.,
                      Finer, S.,
                      Frank, J.,
                      Free, R. C.,
                      Gallo, L.,
                      Gao, H.,
                      Gill, M.,
                      Gilles, M.,
                      Goes, F. S.,
                      Gordon, S. D.,
                      Grove, J.,
                      Gudbjartsson, D. F.,
                      Gutierrez, B.,
                      Hahn, T.,
                      Hall, L. S.,
                      Hansen, T. F.,
                      Haraldsson, M.,
                      Hartman, C. A.,
                      Havdahl, A.,
                      Hayward, C.,
                      Heilmann-Heimbach, S.,
                      Herms, S.,
                      Hickie, I. B.,
                      Hjalgrim, H.,
                      Hjerling-Leffler, J.,
                      Hoffmann, P.,
                      Homuth, G.,
                      Horn, C.,
                      Hottenga, J.-J.,
                      Hougaard, D. M.,
                      Hovatta, I.,
                      Huang, Q. Q.,
                      Hucks, D.,
                      Huider, F.,
                      Hunt, K. A.,
                      Ialongo, N. S.,
                      Ising, M.,
                      IsometsÃ¤, E.,
                      Jansen, R.,
                      Jiang, Y.,
                      Jones, I.,
                      Jones, L. A.,
                      Jonsson, L.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karlsson, R.,
                      Kasper, S.,
                      Kendler, K. S.,
                      Kessler, R. C.,
                      Kloiber, S.,
                      Knowles, J. A.,
                      Koen, N.,
                      Kraft, J.,
                      Kranzler, H. R.,
                      Krebs, K.,
                      Kallak, T. K.,
                      Kutalik, Z.,
                      Lahtela, E.,
                      Lake, M.,
                      Larsen, M. H.,
                      Lenze, E. J.,
                      Lewins, M.,
                      Lewis, G.,
                      Li, L.,
                      Lin, B. D.,
                      Lin, K.,
                      Lind, P. A.,
                      Liu, Y.-L.,
                      MacIntyre, D. J.,
                      MacKinnon, D. F.,
                      Maher, B. S.,
                      Maier, W.,
                      Marshe, V. S.,
                      Martinez-Levy, G. A.,
                      Matsuda, K.,
                      Mbarek, H.,
                      McGuffin, P.,
                      Medland, S. E.,
                      Meinert, S.,
                      Mikkelsen, C.,
                      Mikkelsen, S.,
                      Milaneschi, Y.,
                      Millwood, I. Y.,
                      Molina, E.,
                      Mondimore, F. M.,
                      Mortensen, P. B.,
                      Mulsant, B. H.,
                      Naamanka, J.,
                      Najman, J. M.,
                      Nauck, M.,
                      NenadiÄ‡, I.,
                      Nielsen, K. R.,
                      Nolt, I. M.,
                      Nordentoft, M.,
                      NÃ¶then, M. M.,
                      Nyegaard, M.,
                      Oâ€™Donovan, M. C.,
                      Oddsson, A.,
                      Oliveira, A. M.,
                      Olsen, C. M.,
                      Oskarsson, H.,
                      Ostrowski, S. R.,
                      Owen, M. J.,
                      Packer, R.,
                      Palviainen, T.,
                      Pan, P. M.,
                      Pato, C. N.,
                      Pato, M. T.,
                      Pedersen, N. L.,
                      Pedersen, O. B.,
                      Peyrot, W. J.,
                      Potash, J. B.,
                      Preisig, M.,
                      Preuss, M. H.,
                      Quiroz, J. A.,
                      Renteria, M. E.,
                      Reynolds, C. F.,
                      Rice, J. P.,
                      Sakaue, S.,
                      Santoro, M. L.,
                      Schoevers, R. A.,
                      Schork, A.,
                      Schulze, T. G.,
                      Send, T. S.,
                      Shi, J.,
                      Sigurdsson, E.,
                      Singh, K.,
                      Sinnamon, G. C. B.,
                      Sirignano, L.,
                      Smeland, O. B.,
                      Smith, D. J.,
                      Sofer, T.,
                      SÃ¸rensen, E.,
                      Srinivasan, S.,
                      Stefansson, H.,
                      Stefansson, K.,
                      Straub, P.,
                      Su, M.-H.,
                      Tadic, A.,
                      Teismann, H.,
                      Teumer, A.,
                      Thapar, A.,
                      Thomson, P. A.,
                      ThÃ¸rner, L. W.,
                      Topaloudi, A.,
                      Tsai, S.-J.,
                      Tzoulaki, I.,
                      Uhl, G.,
                      Uitterlinden, A. G.,
                      Ullum, H.,
                      Umbricht, D.,
                      Ursano, R. J.,
                      Auwera, S.,
                      Hemert, A. M.,
                      Veluchamy, A.,
                      Viktorin, A.,
                      VÃ¶lzke, H.,
                      Walters, G. B.,
                      Wang, X.,
                      Wani, A.,
                      Weissman, M. M.,
                      Wellmann, J.,
                      Whiteman, D. C.,
                      Wildman, D.,
                      Willemsen, G.,
                      Williams, A. T.,
                      Winsvold, B. S.,
                      Witt, S. H.,
                      Xiong, Y.,
                      Zillich, L.,
                      Zwart, J.-A.,
                      Twenty-Three and Me Research Team.,
                      China Kadoorie Biobank Collaborative Group.,
                      Estonian Biobank Research Team.,
                      Genes &amp; Health Research Team.,
                      HUNT All-In Psychiatry.,
                      The BioBank Japan Project.,
                      VA Million Veteran Program.,
                      Andreassen, O. A.,
                      Baune, B. T.,
                      Berger, K.,
                      Boomsma, D. I.,
                      BÃ¸rglum, A. D.,
                      Breen, G.,
                      Cai, N.,
                      Coon, H.,
                      Copeland, W. E.,
                      Creese, B.,
                      Cruz-Fuentes, C. S.,
                      Czamara, D.,
                      Davis, L. K.,
                      Derks, E. M.,
                      Domenici, E.,
                      Elliott, P.,
                      Forstner, A. J.,
                      Gawlik, M.,
                      Gelernter, J.,
                      Grabe, H. J.,
                      Hamilton, S. P.,
                      Hveem, K.,
                      John, C.,
                      Kaprio, J.,
                      Kircher, T.,
                      Krebs, M.-O.,
                      Kuo, P.-H.,
                      LandÃ©n, M.,
                      Lehto, K.,
                      Levinson, D. F.,
                      Li, Q. S.,
                      Lieb, K.,
                      Loos, R. J. F.,
                      Lu, Y.,
                      Lucae, S.,
                      Luykx, J. J.,
                      Maes, H. H. M.,
                      Magnusson, P. K.,
                      Martin, H. C.,
                      Martin, N. G.,
                      McQuillin, A.,
                      Middeldorp, C. M.,
                      Milani, L.,
                      Mors, O.,
                      MÃ¼ller, D. J.,
                      MÃ¼ller-Myhsok, B.,
                      Okada, Y.,
                      Oldehinkel, A. J.,
                      Paciga, S. A.,
                      Palmer, C. N. A.,
                      Paschou, P.,
                      Penninx, B. W. J. H.,
                      Perlis, R. H.,
                      Peterson, R. E.,
                      Pistis, G.,
                      Polimanti, R.,
                      Porteous, D. J.,
                      Posthuma, D.,
                      Rabinowitz, J. A.,
                      Reichborn-Kjennerud, T.,
                      Reif, A.,
                      Rice, F.,
                      Ricken, R.,
                      Rietschel, M.,
                      Rivera, M.,
                      RÃ¼ck, C.,
                      Salum, G. A.,
                      Schaefer, C.,
                      Sen, S.,
                      Serretti, A.,
                      Skalkidou, A.,
                      Smoller, J. W.,
                      Stein, D. J.,
                      Stein, F.,
                      Stein, M. B.,
                      Sullivan, P. F.,
                      Tesli, M.,
                      Thorgeirsson, T. E.,
                      Tiemeier, H.,
                      Timpson, N. J.,
                      Uddin, M.,
                      Uher, R.,
                      Heel, D. A.,
                      Verweij, K. J. H.,
                      Walters, R. G.,
                      Wassertheil-Smoller, S.,
                      Wendland, J. R.,
                      Werge, T.,
                      Zwinderman, A. H.,
                      Kuchenbaecker, K.,
                      Wray, N. R.,
                      Ripke, S.,
                      Lewis, C. M.,
                    </span>
                    and McIntosh, A. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell</em>
                    <b>188</b>,
                    640â€“652.e9
                    (2025)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/39814019" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2024.04.29.24306535" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.cell.2024.12.002" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>In a genome-wide association study (GWAS) meta-analysis of 688,808 individuals with major depression (MD) and 4,364,225 controls from 29 countries across diverse and admixed ancestries, we identify 697 associations at 635 loci, 293 of which are novel. Using fine-mapping and functional tools, we find 308 high-confidence gene associations and enrichment of postsynaptic density and receptor clustering. A neural cell-type enrichment analysis utilizing single-cell data implicates excitatory, inhibitory, and medium spiny neurons and the involvement of amygdala neurons in both mouse and human single-cell analyses. The associations are enriched for antidepressant targets and provide potential repurposing opportunities. Polygenic scores trained using European or multi-ancestry data predicted MD status across all ancestries, explaining up to 5.8% of MD liability variance in Europeans. These findings advance our global understanding of MD and reveal biological targets that may be used to target and develop pharmacotherapies addressing the unmet need for effective treatment.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.cell.2024.12.002" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2024">2024</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Pozarickij2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-024-50297-x" target="_blank" rel="noopener">Causal relevance of different blood pressure traits on risk of cardiovascular diseases: GWAS and Mendelian randomisation in 100,000 Chinese adults</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Pozarickij, A.,
                    </span>
                    <span class="coauthor">
                      Gan, W.,
                    </span>
                    <span class="coauthor">
                      Lin, K.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Clarke, R.,
                      Fairhurst-Hunter, Z.,
                      Koido, M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Okada, Y.,
                      Kamatani, Y.,
                      Bennett, D.,
                      Du, H.,
                      Chen, Y.,
                      Yang, L.,
                      Avery, D.,
                      Guo, Y.,
                      Yu, M.,
                      Yu, C.,
                      Schmidt Valle, D.,
                      Lv, J.,
                      Chen, J.,
                      Peto, R.,
                      Collins, R.,
                      Li, L.,
                      Chen, Z.,
                      Millwood, I. Y.,
                      Walters, R. G.,
                    </span>
                    and China Kadoorie Biobank Collaborative Group.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>15</b>,
                    6265
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dPbrd" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/39048560" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2023.01.20.23284709" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-024-50297-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Elevated blood pressure (BP) is major risk factor for cardiovascular diseases (CVD). Genome-wide association studies (GWAS) conducted predominantly in populations of European ancestry have identified &gt;2,000 BP-associated loci, but other ancestries have been less well-studied. We conducted GWAS of systolic, diastolic, pulse, and mean arterial BP in 100,453 Chinese adults. We identified 128 non-overlapping loci associated with one or more BP traits, including 74 newly-reported associations. Despite strong genetic correlations between populations, we identified appreciably higher heritability and larger variant effect sizes in Chinese compared with European or Japanese ancestry populations. Using instruments derived from these GWAS, multivariable Mendelian randomisation demonstrated that BP traits contribute differently to the causal associations of BP with CVD. In particular, only pulse pressure was independently causally associated with carotid plaque. These findings reinforce the need for studies in diverse populations to understand the genetic determinants of BP traits and their roles in disease risk.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-024-50297-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Carey2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41562-024-01909-5" target="_blank" rel="noopener">Principled distillation of UK Biobank phenotype data reveals underlying structure in human variation</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Carey, C. E.,
                    </span>
                    <span class="coauthor">
                      Shafee, R.,
                    </span>
                    <span class="coauthor">
                      Wedow, R.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Elliott, A.,
                      Palmer, D. S.,
                      Compitello, J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Abbott, L.,
                      Schultz, P.,
                      Karczewski, K. J.,
                      Bryant, S. C.,
                      Cusick, C. M.,
                      Churchhouse, C.,
                      Howrigan, D. P.,
                      King, D.,
                      Davey Smith, G.,
                      Neale, B. M.,
                      Walters, R. K.,
                    </span>
                    and Robinson, E. B.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Human Behaviour</em>
                    <b>8</b>,
                    1599â€“1615
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dM1kx" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38965376" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.09.02.22279546" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/ce-carey/ukb-factor-analysis" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41562-024-01909-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Data within biobanks capture broad yet detailed indices of human variation, but biobank-wide insights can be difficult to extract due to complexity and scale. Here, using large-scale factor analysis, we distill hundreds of variables (diagnoses, assessments and survey items) into 35 latent constructs, using data from unrelated individuals with predominantly estimated European genetic ancestry in UK Biobank. These factors recapitulate known disease classifications, disentangle elements of socioeconomic status, highlight the relevance of psychiatric constructs to health and improve measurement of pro-health behaviours. We go on to demonstrate the power of this approach to clarify genetic signal, enhance discovery and identify associations between underlying phenotypic structure and health outcomes. In building a deeper understanding of ways in which constructs such as socioeconomic status, trauma, or physical activity are structured in the dataset, we emphasize the importance of considering the interwoven nature of the human phenome when evaluating public health patterns.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41562-024-01909-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Trsan2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.devcel.2024.06.001" target="_blank" rel="noopener">The centrosomal protein FGFR1OP controls myosin function in murine intestinal epithelial cells</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Trsan, T.,
                    </span>
                    <span class="coauthor">
                      Peng, V.,
                    </span>
                    <span class="coauthor">
                      Krishna, C.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ohara, T. E.,
                      Beatty, W. L.,
                      Sudan, R.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Krishnamoorthy, P.,
                      Rodrigues, P. F.,
                      Fachi, J. L.,
                      Grajales-Reyes, G.,
                      Jaeger, N.,
                      Fitzpatrick, J. A. J.,
                      Cella, M.,
                      Gilfillan, S.,
                      Nakata, T.,
                      Jaiswal, A.,
                      Stappenbeck, T. S.,
                      Daly, M. J.,
                      Xavier, R. J.,
                    </span>
                    and Colonna, M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Developmental Cell</em>
                    <b>59</b>,
                    2460â€“2476
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/developmental-cell/pdfExtended/S1534-5807(24)00379-4" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38942017" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.devcel.2024.06.001" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Recent advances in human genetics have shed light on the genetic factors contributing to inflammatory diseases, particularly Crohnâ€™s disease (CD), a prominent form of inflammatory bowel disease. Certain risk genes associated with CD directly influence cytokine biology and cell-specific communication networks. Current CD therapies primarily rely on anti-inflammatory drugs, which are inconsistently effective and lack strategies for promoting epithelial restoration and mucosal balance. To understand CDâ€™s underlying mechanisms, we investigated the link between CD and the FGFR1OP gene, which encodes a centrosome protein. FGFR1OP deletion in mouse intestinal epithelial cells disrupted crypt architecture, resulting in crypt loss, inflammation, and fatality. FGFR1OP insufficiency hindered epithelial resilience during colitis. FGFR1OP was crucial for preserving non-muscle myosin II activity, ensuring the integrity of the actomyosin cytoskeleton and crypt cell adhesion. This role of FGFR1OP suggests that its deficiency in genetically predisposed individuals may reduce epithelial renewal capacity, heightening susceptibility to inflammation and disease.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.devcel.2024.06.001" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Jermy2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-024-48938-2" target="_blank" rel="noopener">A unified framework for estimating country-specific cumulative incidence for 18 diseases stratified by polygenic risk</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Jermy, B.,
                    </span>
                    <span class="coauthor">
                      LÃ¤ll, K.,
                    </span>
                    <span class="coauthor">
                      Wolford, B. N.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Wang, Y.,
                      Zguro, K.,
                      Cheng, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kanoni, S.,
                      Yang, Z.,
                      Hartonen, T.,
                      Monti, R.,
                      Wanner, J.,
                      Youssef, O.,
                      Estonian Biobank research team.,
                      FinnGen.,
                      Lippert, C.,
                      Heel, D.,
                      Okada, Y.,
                      McCartney, D. L.,
                      Hayward, C.,
                      Marioni, R. E.,
                      Furini, S.,
                      Renieri, A.,
                      Martin, A. R.,
                      Neale, B. M.,
                      Hveem, K.,
                      MÃ¤gi, R.,
                      Palotie, A.,
                      Heyne, H.,
                      Mars, N.,
                      Ganna, A.,
                    </span>
                    and Ripatti, S.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>15</b>,
                    5007
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dKCBb" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38866767" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2023.06.12.23291186" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/intervene-EU-H2020/flagship" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.interveneproject.eu/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-024-48938-2" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic scores (PGSs) offer the ability to predict genetic risk for complex diseases across the life course; a key benefit over short-term prediction models. To produce risk estimates relevant to clinical and public health decision-making, it is important to account for varying effects due to age and sex. Here, we develop a novel framework to estimate country-, age-, and sex-specific estimates of cumulative incidence stratified by PGS for 18 high-burden diseases. We integrate PGS associations from seven studies in four countries (N = 1,197,129) with disease incidences from the Global Burden of Disease. PGS has a significant sex-specific effect for asthma, hip osteoarthritis, gout, coronary heart disease and type 2 diabetes (T2D), with all but T2D exhibiting a larger effect in men. PGS has a larger effect in younger individuals for 13 diseases, with effects decreasing linearly with age. We show for breast cancer that, relative to individuals in the bottom 20% of polygenic risk, the top 5% attain an absolute risk for screening eligibility 16.3 years earlier. Our framework increases the generalizability of results from biobank studies and the accuracy of absolute risk estimates by appropriately accounting for age- and sex-specific PGS effects. Our results highlight the potential of PGS as a screening tool which may assist in the early prevention of common diseases. Here the authors present a framework for estimating disease risk using PGS accounting for country, age and sex. They find that PGSs have a significant sex-specific effect on common diseases, and their effect is typically larger in young individuals.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-024-48938-2" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sakaue2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-024-01682-1" target="_blank" rel="noopener">Tissue-specific enhancerâ€“gene maps from multimodal single-cell data identify causal disease alleles</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Sakaue, S.,
                    </span>
                    <span class="coauthor">
                      Weinand, K.,
                    </span>
                    <span class="coauthor">
                      Isaac, S.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Dey, K. K.,
                      Jagadeesh, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Watts, G. F. M.,
                      Zhu, Z.,
                      Accelerating Medicines Partnership RA/SLE Program and Network.,
                      Brenner, M. B.,
                      McDavid, A.,
                      Donlin, L. T.,
                      Wei, K.,
                      Price, A. L.,
                    </span>
                    and Raychaudhuri, S.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>56</b>,
                    615â€“626
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dEgB3" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38594305" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.10.27.22281574" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/immunogenomics/SCENT" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-024-01682-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Translating genome-wide association study (GWAS) loci into causal variants and genes requires accurate cell-type-specific enhancerâ€“gene maps from disease-relevant tissues. Building enhancerâ€“gene maps is essential but challenging with current experimental methods in primary human tissues. Here we developed a nonparametric statistical method, SCENT (single-cell enhancer target gene mapping), that models association between enhancer chromatin accessibility and gene expression in single-cell or nucleus multimodal RNA sequencing and ATAC sequencing data. We applied SCENT to 9 multimodal datasets including &gt;120,000 single cells or nuclei and created 23 cell-type-specific enhancerâ€“gene maps. These maps were highly enriched for causal variants in expression quantitative loci and GWAS for 1,143 diseases and traits. We identified likely causal genes for both common and rare diseases and linked somatic mutation hotspots to target genes. We demonstrate that application of SCENT to multimodal data from disease-relevant human tissue enables the scalable construction of accurate cell-type-specific enhancerâ€“gene maps, essential for defining noncoding variant function. SCENT is a nonparametric method that models association between chromatin accessibility and gene expression in single-cell multimodal datasets, enabling construction of cell-type-specific enhancerâ€“gene maps to aid mapping of candidate causal variants and genes for common diseases.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-024-01682-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="De_Vincentis2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1111/apt.17953" target="_blank" rel="noopener">Poor accuracy and sustainability of the first-step FIB4 EASL pathway for stratifying steatotic liver disease risk in the general population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      De Vincentis, A.,
                    </span>
                    <span class="coauthor">
                      Tavaglione, F.,
                    </span>
                    <span class="coauthor">
                      Namba, S.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Okada, Y.,
                      Kamatani, Y.,
                      Maurotti, S.,
                      Pedone, C.,
                      Antonelli Incalzi, R.,
                      Valenti, L.,
                      Romeo, S.,
                    </span>
                    and Vespasiani-Gentilucci, U.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Alimentary Pharmacology &amp; Therapeutics</em>
                    <b>59</b>,
                    1402â€“1412
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38497224" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1111/apt.17953" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>BACKGROUND AND AIMS: The European Association for the Study of the Liver introduced a clinical pathway (EASL CP) for screening significant/advanced fibrosis in people at risk of steatotic liver disease (SLD). We assessed the performance of the first-step FIB4 EASL CP in the general population across different SLD risk groups (MASLD, Met-ALD and ALD) and various age classes. METHODS: We analysed a total of 3372 individuals at risk of SLD from the 2017-2018 National Health and Nutrition Examination Survey (NHANES17-18), projected to 152.3 million U.S. adults, 300,329 from the UK Biobank (UKBB) and 57,644 from the Biobank Japan (BBJ). We assessed liver stiffness measurement (LSM) \geq8 kPa and liver-related events occurring within 3 and 10 years (3/10 year-LREs) as outcomes. We defined MASLD, MetALD, and ALD according to recent international recommendations. RESULTS: FIB4 sensitivity for LSM â‰¥8 kPa was low (27.7%), but it ranged approximately 80%-90% for 3-year LREs. Using FIB4, 22%-57% of subjects across the three cohorts were identified as candidates for vibration-controlled transient elastography (VCTE), which was mostly avoidable (positive predictive value of FIB4 â‰¥1.3 for LSM â‰¥8 kPa ranging 9.5%-13% across different SLD categories). Sensitivity for LSM â‰¥8 kPa and LREs increased with increasing alcohol intake (ALD&gt;MetALD&gt;MASLD) and age classes. For individuals aged \geq65 years, using the recommended age-adjusted FIB4 cut-off (\geq2) substantially reduced sensitivity for LSM â‰¥8 kPa and LREs. CONCLUSIONS: The first-step FIB4 EASL CP is poorly accurate and feasible for individuals at risk of SLD in the general population. It is crucial to enhance the screening strategy with a first-step approach able to reduce unnecessary VCTEs and optimise their yield.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1111/apt.17953" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Lo_Faro2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xcrm.2024.101430" target="_blank" rel="noopener">Novel ancestry-specific primary open-angle glaucoma loci and shared biology with vascular mechanisms and cell proliferation</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Lo Faro, V.,
                    </span>
                    <span class="coauthor">
                      Bhattacharya, A.,
                    </span>
                    <span class="coauthor">
                      Zhou, W.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, D.,
                      Wang, Y.,
                      LÃ¤ll, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Lopera-Maya, E.,
                      Straub, P.,
                      Pawar, P.,
                      Tao, R.,
                      Zhong, X.,
                      Namba, S.,
                      Global Biobank Meta-analysis Initiative.,
                      Sanna, S.,
                      Nolte, I. M.,
                      Okada, Y.,
                      Ingold, N.,
                      MacGregor, S.,
                      Snieder, H.,
                      Surakka, I.,
                      Shortt, J.,
                      Gignoux, C.,
                      Rafaels, N.,
                      Crooks, K.,
                      Verma, A.,
                      Verma, S. S.,
                      Guare, L.,
                      Rader, D. J.,
                      Willer, C.,
                      Martin, A. R.,
                      Brantley, M. A.,
                      Gamazon, E. R.,
                      Jansonius, N. M.,
                      Joos, K.,
                      Cox, N. J.,
                    </span>
                    and Hirbo, J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Reports Medicine</em>
                    <b>5</b>,
                    101430
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-reports-medicine/pdfExtended/S2666-3791(24)00053-3" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38382466" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.12.16.21267891" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xcrm.2024.101430" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Primary open-angle glaucoma (POAG), a leading cause of irreversible blindness globally, shows disparity in prevalence and manifestations across ancestries. We perform meta-analysis across 15 biobanks (of the Global Biobank Meta-analysis Initiative) (n = 1,487,441: cases = 26,848) and merge with previous multi-ancestry studies, with the combined dataset representing the largest and most diverse POAG study to date (n = 1,478,037: cases = 46,325) and identify 17 novel significant loci, 5 of which were ancestry specific. Gene-enrichment and transcriptome-wide association analyses implicate vascular and cancer genes, a fifth of which are primary ciliary related. We perform an extensive statistical analysis of SIX6 and CDKN2B-AS1 loci in human GTEx data and across large electronic health records showing interaction between SIX6 gene and causal variants in the chr9p21.3 locus, with expression effect on CDKN2A/B. Our results suggest that some POAG risk variants may be ancestry specific, sex specific, or both, and support the contribution of genes involved in programmed cell death in POAG pathogenesis.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xcrm.2024.101430" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Scholz2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-024-44709-1" target="_blank" rel="noopener">X-chromosome and kidney function: evidence from a multi-trait genetic analysis of 908,697 individuals reveals sex-specific and sex-differential findings in genes regulated by androgen response elements</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Scholz, M.,
                    </span>
                    <span class="coauthor">
                      Horn, K.,
                    </span>
                    <span class="coauthor">
                      Pott, J.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Wuttke, M.,
                      KÃ¼hnapfel, A.,
                      Nasr, M. K.,
                      Kirsten, H.,
                      Li, Y.,
                      Hoppmann, A.,
                      Gorski, M.,
                      Ghasemi, S.,
                      Li, M.,
                      Tin, A.,
                      Chai, J.-F.,
                      Cocca, M.,
                      Wang, J.,
                      Nutile, T.,
                      Akiyama, M.,
                      Ã…svold, B. O.,
                      Bansal, N.,
                      Biggs, M. L.,
                      Boutin, T.,
                      Brenner, H.,
                      Brumpton, B.,
                      Burkhardt, R.,
                      Cai, J.,
                      Campbell, A.,
                      Campbell, H.,
                      Chalmers, J.,
                      Chasman, D. I.,
                      Chee, M. L.,
                      Chee, M. L.,
                      Chen, X.,
                      Cheng, C.-Y.,
                      Cifkova, R.,
                      Daviglus, M.,
                      Delgado, G.,
                      Dittrich, K.,
                      Edwards, T. L.,
                      Endlich, K.,
                      Michael Gaziano, J.,
                      Giri, A.,
                      Giulianini, F.,
                      Gordon, S. D.,
                      Gudbjartsson, D. F.,
                      Hallan, S.,
                      Hamet, P.,
                      Hartman, C. A.,
                      Hayward, C.,
                      Heid, I. M.,
                      Hellwege, J. N.,
                      Holleczek, B.,
                      Holm, H.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Hveem, K.,
                      Isermann, B.,
                      Jonas, J. B.,
                      Joshi, P. K.,
                      Kamatani, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kastarinen, M.,
                      Khor, C. C.,
                      Kiess, W.,
                      Kleber, M. E.,
                      KÃ¶rner, A.,
                      Kovacs, P.,
                      Krajcoviechova, A.,
                      Kramer, H.,
                      KrÃ¤mer, B. K.,
                      Kuokkanen, M.,
                      KÃ¤hÃ¶nen, M.,
                      Lange, L. A.,
                      Lash, J. P.,
                      LehtimÃ¤ki, T.,
                      Li, H.,
                      Lin, B. M.,
                      Liu, J.,
                      Loeffler, M.,
                      LyytikÃ¤inen, L.-P.,
                      Magnusson, P. K. E.,
                      Martin, N. G.,
                      Matsuda, K.,
                      Milaneschi, Y.,
                      Mishra, P. P.,
                      Mononen, N.,
                      Montgomery, G. W.,
                      Mook-Kanamori, D. O.,
                      Mychaleckyj, J. C.,
                      MÃ¤rz, W.,
                      Nauck, M.,
                      Nikus, K.,
                      Nolte, I. M.,
                      Noordam, R.,
                      Okada, Y.,
                      Olafsson, I.,
                      Oldehinkel, A. J.,
                      Penninx, B. W. J. H.,
                      Perola, M.,
                      Pirastu, N.,
                      Polasek, O.,
                      Porteous, D. J.,
                      Poulain, T.,
                      Psaty, B. M.,
                      Rabelink, T. J.,
                      Raffield, L. M.,
                      Raitakari, O. T.,
                      Rasheed, H.,
                      Reilly, D. F.,
                      Rice, K. M.,
                      Richmond, A.,
                      Ridker, P. M.,
                      Rotter, J. I.,
                      Rudan, I.,
                      Sabanayagam, C.,
                      Salomaa, V.,
                      Schneiderman, N.,
                      SchÃ¶ttker, B.,
                      Sims, M.,
                      Snieder, H.,
                      Stark, K. J.,
                      Stefansson, K.,
                      Stocker, H.,
                      Stumvoll, M.,
                      Sulem, P.,
                      Sveinbjornsson, G.,
                      Svensson, P. O.,
                      Tai, E.-S.,
                      Taylor, K. D.,
                      Tayo, B. O.,
                      Teren, A.,
                      Tham, Y.-C.,
                      Thiery, J.,
                      Thio, C. H. L.,
                      Thomas, L. F.,
                      Tremblay, J.,
                      TÃ¶njes, A.,
                      Most, P. J.,
                      Vitart, V.,
                      VÃ¶lker, U.,
                      Wang, Y. X.,
                      Wang, C.,
                      Wei, W. B.,
                      Whitfield, J. B.,
                      Wild, S. H.,
                      Wilson, J. F.,
                      Winkler, T. W.,
                      Wong, T.-Y.,
                      Woodward, M.,
                      Sim, X.,
                      Chu, A. Y.,
                      Feitosa, M. F.,
                      Thorsteinsdottir, U.,
                      Hung, A. M.,
                      Teumer, A.,
                      Franceschini, N.,
                      Parsa, A.,
                      KÃ¶ttgen, A.,
                      Schlosser, P.,
                    </span>
                    and Pattaro, C.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>15</b>,
                    586
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dwDZ5" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38233393" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-024-44709-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>X-chromosomal genetic variants are understudied but can yield valuable insights into sexually dimorphic human traits and diseases. We performed a sex-stratified cross-ancestry X-chromosome-wide association meta-analysis of seven kidney-related traits (n = 908,697), identifying 23 loci genome-wide significantly associated with two of the traits: 7 for uric acid and 16 for estimated glomerular filtration rate (eGFR), including four novel eGFR loci containing the functionally plausible prioritized genes ACSL4, CLDN2, TSPAN6 and the female-specific DRP2. Further, we identified five novel sex-interactions, comprising male-specific effects at FAM9B and AR/EDA2R, and three sex-differential findings with larger genetic effect sizes in males at DCAF12L1 and MST4 and larger effect sizes in females at HPRT1. All prioritized genes in loci showing significant sex-interactions were located next to androgen response elements (ARE). Five ARE genes showed sex-differential expressions. This study contributes new insights into sex-dimorphisms of kidney traits along with new prioritized gene targets for further molecular research.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-024-44709-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Meng2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-023-01596-4" target="_blank" rel="noopener">Multi-ancestry genome-wide association study of major depression aids locus discovery, fine mapping, gene prioritization and causal inference</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Meng, X.,
                    </span>
                    <span class="coauthor">
                      Navoly, G.,
                    </span>
                    <span class="coauthor">
                      Giannakopoulou, O.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Levey, D. F.,
                      Koller, D.,
                      Pathak, G. A.,
                      Koen, N.,
                      Lin, K.,
                      Adams, M. J.,
                      RenterÄ±Ìa Miguel E.,
                      Feng, Y.,
                      Gaziano, J. M.,
                      Stein, D. J.,
                      Zar, H. J.,
                      Campbell, M. L.,
                      Heel, D. A.,
                      Trivedi, B.,
                      Finer, S.,
                      McQuillin, A.,
                      Bass, N.,
                      Chundru, V. K.,
                      Martin, H. C.,
                      Huang, Q. Q.,
                      Valkovskaya, M.,
                      Chu, C.-Y.,
                      Kanjira, S.,
                      Kuo, P.-H.,
                      Chen, H.-C.,
                      Tsai, S.-J.,
                      Liu, Y.-L.,
                      Kendler, K. S.,
                      Peterson, R. E.,
                      Cai, N.,
                      Fang, Y.,
                      Sen, S.,
                      Scott, L. J.,
                      Burmeister, M.,
                      Loos, R. J. F.,
                      Preuss, M. H.,
                      Actkins, K. V.,
                      Davis, L. K.,
                      Uddin, M.,
                      Wani, A. H.,
                      Wildman, D. E.,
                      Aiello, A. E.,
                      Ursano, R. J.,
                      Kessler, R. C.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Okada, Y.,
                      Sakaue, S.,
                      Rabinowitz, J. A.,
                      Maher, B. S.,
                      Uhl, G.,
                      Eaton, W.,
                      Cruz-Fuentes, C. S.,
                      Martinez-Levy, G. A.,
                      Campos, A. I.,
                      Millwood, I. Y.,
                      Chen, Z.,
                      Li, L.,
                      Wassertheil-Smoller, S.,
                      Jiang, Y.,
                      Tian, C.,
                      Martin, N. G.,
                      Mitchell, B. L.,
                      Byrne, E. M.,
                      Awasthi, S.,
                      Coleman, J. R. I.,
                      Ripke, S.,
                      PGC-MDD Working Group.,
                      China Kadoorie Biobank Collaborative Group.,
                      23andMe Research Team.,
                      Genes and Health Research Team.,
                      BioBank Japan Project.,
                      Sofer, T.,
                      Walters, R. G.,
                      McIntosh, A. M.,
                      Polimanti, R.,
                      Dunn, E. C.,
                      Stein, M. B.,
                      Gelernter, J.,
                      Lewis, C. M.,
                    </span>
                    and Kuchenbaecker, K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>56</b>,
                    222â€“233
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dv985" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38177345" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.07.20.500802" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-023-01596-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Most genome-wide association studies (GWAS) of major depression (MD) have been conducted in samples of European ancestry. Here we report a multi-ancestry GWAS of MD, adding data from 21 cohorts with 88,316 MD cases and 902,757 controls to previously reported data. This analysis used a range of measures to define MD and included samples of African (36% of effective sample size), East Asian (26%) and South Asian (6%) ancestry and Hispanic/Latin American participants (32%). The multi-ancestry GWAS identified 53 significantly associated novel loci. For loci from GWAS in European ancestry samples, fewer than expected were transferable to other ancestry groups. Fine mapping benefited from additional sample diversity. A transcriptome-wide association study identified 205 significantly associated novel genes. These findings suggest that, for MD, increasing ancestral and global diversity in genetic studies may be particularly important to ensure discovery of core genes and inform about transferability of findings.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-023-01596-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Chen2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-023-06045-0" target="_blank" rel="noopener">A genomic mutational constraint map using variation in 76,156 human genomes</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Chen, S.,
                    </span>
                    <span class="coauthor">
                      Francioli, L. C.,
                    </span>
                    <span class="coauthor">
                      Goodrich, J. K.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Collins, R. L.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Wang, Q.,
                      AlfÃ¶ldi, J.,
                      Watts, N. A.,
                      Vittal, C.,
                      Gauthier, L. D.,
                      Poterba, T.,
                      Wilson, M. W.,
                      Tarasova, Y.,
                      Phu, W.,
                      Grant, R.,
                      Yohannes, M. T.,
                      Koenig, Z.,
                      Farjoun, Y.,
                      Banks, E.,
                      Donnelly, S.,
                      Gabriel, S.,
                      Gupta, N.,
                      Ferriera, S.,
                      Tolonen, C.,
                      Novod, S.,
                      Bergelson, L.,
                      Roazen, D.,
                      Ruano-Rubio, V.,
                      Covarrubias, M.,
                      Llanwarne, C.,
                      Petrillo, N.,
                      Wade, G.,
                      Jeandet, T.,
                      Munshi, R.,
                      Tibbetts, K.,
                      Genome Aggregation Database Consortium.,
                      Oâ€™Donnell-Luria, A.,
                      Solomonson, M.,
                      Seed, C.,
                      Martin, A. R.,
                      Talkowski, M. E.,
                      Rehm, H. L.,
                      Daly, M. J.,
                      Tiao, G.,
                      Neale, B. M.,
                      MacArthur, D. G.,
                    </span>
                    and Karczewski, K. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>625</b>,
                    92â€“100
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/ds6zL" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38057664" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.03.20.485034" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://gnomad.broadinstitute.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-023-06045-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The depletion of disruptive variation caused by purifying natural selection (constraint) has been widely used to investigate protein-coding genes underlying human disorders1-4, but attempts to assess constraint for non-protein-coding regions have proved more difficult. Here we aggregate, process and release a dataset of 76,156 human genomes from the Genome Aggregation Database (gnomAD)-the largest public open-access human genome allele frequency reference dataset-and use it to build a genomic constraint map for the whole genome (genomic non-coding constraint of haploinsufficient variation (Gnocchi)). We present a refined mutational model that incorporates local sequence context and regional genomic features to detect depletions of variation. As expected, the average constraint for protein-coding sequences is stronger than that for non-coding regions. Within the non-coding genome, constrained regions are enriched for known regulatory elements and variants that are implicated in complex human diseases and traits, facilitating the triangulation of biological annotation, disease association and natural selection to non-coding DNA analysis. More constrained regulatory elements tend to regulate more constrained protein-coding genes, which in turn suggests that non-coding constraint can aid the identification of constrained genes that are as yet unrecognized by current gene constraint metrics. We demonstrate that this genome-wide constraint map improves the identification and interpretation of functional human genetic variation.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-023-06045-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Cui2024" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-023-01597-3" target="_blank" rel="noopener">Improving fine-mapping by modeling infinitesimal effects</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Cui, R.,
                    </span>
                    <span class="coauthor">
                      Elzur, R. A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ulirsch, J. C.,
                      Weissbrod, O.,
                      Daly, M. J.,
                      Neale, B. M.,
                      Fan, Z.,
                    </span>
                    and Finucane, H. K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>56</b>,
                    162â€“169
                    (2024)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dswl2" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/38036779" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.10.21.513123" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://github.com/FinucaneLab/fine-mapping-inf" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-023-01597-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Fine-mapping aims to identify causal genetic variants for phenotypes. Bayesian fine-mapping algorithms (for example, SuSiE, FINEMAP, ABF and COJO-ABF) are widely used, but assessing posterior probability calibration remains challenging in real data, where model misspecification probably exists, and true causal variants are unknown. We introduce replication failure rate (RFR), a metric to assess fine-mapping consistency by downsampling. SuSiE, FINEMAP and COJO-ABF show high RFR, indicating potential overconfidence in their output. Simulations reveal that nonsparse genetic architecture can lead to miscalibration, while imputation noise, nonuniform distribution of causal variants and quality control filters have minimal impact. Here we present SuSiE-inf and FINEMAP-inf, fine-mapping methods modeling infinitesimal effects alongside fewer larger causal effects. Our methods show improved calibration, RFR and functional enrichment, competitive recall and computational efficiency. Notably, using our methodsâ€™ posterior effect sizes substantially increases polygenic risk score accuracy over SuSiE and FINEMAP. Our work improves causal variant identification for complex traits, a fundamental goal of human genetics. A downsampling approach to assess causal variant fine-mapping, replication failure rate, finds that commonly used methods may be miscalibrated. Simulations suggest this is probably due to a nonsparse genetic architecture model misspecification. Incorporating infinitesimal effects in the SuSiE and FINEMAP frameworks improves performance.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-023-01597-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2023">2023</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kerimov2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1371/journal.pgen.1010932" target="_blank" rel="noopener">eQTL Catalogue 2023: New datasets, X chromosome QTLs, and improved detection and visualisation of transcript-level QTLs</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Kerimov, N.,
                    </span>
                    <span class="coauthor">
                      Tambets, R.,
                    </span>
                    <span class="coauthor">
                      Hayhurst, J. D.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Rahu, I.,
                      Kolberg, P.,
                      Raudvere, U.,
                      Kuzmin, I.,
                      Chowdhary, A.,
                      Vija, A.,
                      Teras, H. J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ulirsch, J.,
                      Ryten, M.,
                      Hardy, J.,
                      Guelfi, S.,
                      Trabzuni, D.,
                      Kim-Hellmuth, S.,
                      Rayner, W.,
                      Finucane, H.,
                      Peterson, H.,
                      Mosaku, A.,
                      Parkinson, H.,
                    </span>
                    and Alasoo, K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>PLoS Genetics</em>
                    <b>19</b>,
                    e1010932
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://journals.plos.org/plosgenetics/article/file?id=10.1371/journal.pgen.1010932&amp;type=printable" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37721944" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2023.04.06.535816" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://www.ebi.ac.uk/eqtl/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1371/journal.pgen.1010932" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The eQTL Catalogue is an open database of uniformly processed human molecular quantitative trait loci (QTLs). We are continuously updating the resource to further increase its utility for interpreting genetic associations with complex traits. Over the past two years, we have increased the number of uniformly processed studies from 21 to 31 and added X chromosome QTLs for 19 compatible studies. We have also implemented Leafcutter to directly identify splice-junction usage QTLs in all RNA sequencing datasets. Finally, to improve the interpretability of transcript-level QTLs, we have developed static QTL coverage plots that visualise the association between the genotype and average RNA sequencing read coverage in the region for all 1.7 million fine mapped associations. To illustrate the utility of these updates to the eQTL Catalogue, we performed colocalisation analysis between vitamin D levels in the UK Biobank and all molecular QTLs in the eQTL Catalogue. Although most GWAS loci colocalised both with eQTLs and transcript-level QTLs, we found that visual inspection could sometimes be used to distinguish primary splicing QTLs from those that appear to be secondary consequences of large-effect gene expression QTLs. While these visually confirmed primary splicing QTLs explain just 6/53 of the colocalising signals, they are significantly less pleiotropic than eQTLs and identify a prioritised causal gene in 4/6 cases.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1371/journal.pgen.1010932" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Wang_Y2023b" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2023.100408" target="_blank" rel="noopener">Polygenic prediction across populations is influenced by ancestry, genetic architecture, and methodology</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Wang, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Tan, T.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kamariza, M.,
                      Tsuo, K.,
                      Yuan, K.,
                      Zhou, W.,
                      Okada, Y.,
                      BioBank Japan Project.,
                      Huang, H.,
                      Turley, P.,
                      Atkinson, E. G.,
                    </span>
                    and Martin, A. R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>3</b>,
                    100408
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(23)00222-7" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37868036" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.12.29.522270" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2023.100408" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores (PRSs) developed from multi-ancestry genome-wide association studies (GWASs), PRSmulti, hold promise for improving PRS accuracy and generalizability across populations. To establish best practices for leveraging the increasing diversity of genomic studies, we investigated how various factors affect the performance of PRSmulti compared with PRSs constructed from single-ancestry GWASs (PRSsingle). Through extensive simulations and empirical analyses, we showed that PRSmulti overall outperformed PRSsingle in understudied populations, except when the understudied population represented a small proportion of the multi-ancestry GWAS. Furthermore, integrating PRSs based on local ancestry-informed GWASs and large-scale, European-based PRSs improved predictive performance in understudied African populations, especially for less polygenic traits with large-effect ancestry-enriched variants. Our work highlights the importance of diversifying genomic studies to achieve equitable PRS performance across ancestral populations and provides guidance for developing PRSs from multiple studies.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2023.100408" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="The_COVID-19_Host_Genetics_Initiative2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-023-06355-3" target="_blank" rel="noopener">A second update on mapping the human genetic architecture of COVID-19</a></div>
                  <div class="author">
                    The COVID-19 Host Genetics Initiative.
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>621</b>,
                    E7â€“E26
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dlwih" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37674002" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.12.24.22283874" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.covid19hg.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-023-06355-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Investigating the role of host genetic factors in COVID-19 severity and susceptibility can inform our understanding of the underlying biological mechanisms that influence adverse outcomes and drug development. Here we present a second updated genome-wide association study (GWAS) on COVID-19 severity and infection susceptibility to SARS-CoV-2 from the COVID-19 Host Genetic Initiative (data release 7). We performed a meta-analysis of up to 219,692 cases and over 3â€‰million controls, identifying 51 distinct genome-wide significant lociâ€”adding 28 loci from the previous data release. The increased number of candidate genes at the identified loci helped to map three major biological pathways that are involved in susceptibility and severity: viral entry, airway defence in mucus and type I interferon.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-023-06355-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="International_League_Against_Epilepsy_Consortium_on_Complex_Epilepsies2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-023-01485-w" target="_blank" rel="noopener">GWAS meta-analysis of over 29,000 people with epilepsy identifies 26 risk loci and subtype-specific genetic architecture</a></div>
                  <div class="author">
                    International League Against Epilepsy Consortium on Complex Epilepsies.
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>55</b>,
                    1471â€“1482
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dkWho" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37653029" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.06.08.22276120" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-023-01485-w" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Epilepsy is a highly heritable disorder affecting over 50 million people worldwide, of which about one-third are resistant to current treatments. Here we report a multi-ancestry genome-wide association study including 29,944 cases, stratified into three broad categories and seven subtypes of epilepsy, and 52,538 controls. We identify 26 genome-wide significant loci, 19 of which are specific to genetic generalized epilepsy (GGE). We implicate 29 likely causal genes underlying these 26 loci. SNP-based heritability analyses show that common variants explain between 39.6% and 90% of genetic risk for GGE and its subtypes. Subtype analysis revealed markedly different genetic architectures between focal and generalized epilepsies. Gene-set analyses of GGE signals implicate synaptic processes in both excitatory and inhibitory neurons in the brain. Prioritized candidate genes overlap with monogenic epilepsy genes and with targets of current antiseizure medications. Finally, we leverage our results to identify alternate drugs with predicted efficacy if repurposed for epilepsy treatment. Genome-wide association meta-analyses identify 26 risk loci for epilepsy, including 19 loci specific to genetic generalized epilepsy. Prioritized candidate genes implicate synaptic processes and overlap with targets of antiseizure medications.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-023-01485-w" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Gupta2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-023-06426-5" target="_blank" rel="noopener">Nuclear genetic control of mtDNA copy number and heteroplasmy in humans</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Gupta, R.,
                    </span>
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Durham, T. J.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Tsuo, K.,
                      McCoy, J. G.,
                      Kotrys, A. V.,
                      Zhou, W.,
                      Chinnery, P. F.,
                      Karczewski, K. J.,
                      Calvo, S. E.,
                      Neale, B. M.,
                    </span>
                    and Mootha, V. K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>620</b>,
                    839â€“848
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/djF0J" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37587338" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2023.01.19.23284696" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/rahulg603/mtSwirl" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.broadinstitute.org/news/nuclear-dna-influences-variation-mitochondrial-dna" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-023-06426-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Mitochondrial DNA (mtDNA) is a maternally inherited, high-copy-number genome required for oxidative phosphorylation1. Heteroplasmy refers to the presence of a mixture of mtDNA alleles in an individual and has been associated with disease and ageing. Mechanisms underlying common variation in human heteroplasmy, and the influence of the nuclear genome on this variation, remain insufficiently explored. Here we quantify mtDNA copy number (mtCN) and heteroplasmy using blood-derived whole-genome sequences from 274,832 individuals and perform genome-wide association studies to identify associated nuclear loci. Following blood cell composition correction, we find that mtCN declines linearly with age and is associated with variants at 92 nuclear loci. We observe that nearly everyone harbours heteroplasmic mtDNA variants obeying two principles: (1) heteroplasmic single nucleotide variants tend to arise somatically and accumulate sharply after the age of 70 years, whereas (2) heteroplasmic indels are maternally inherited as mixtures with relative levels associated with 42 nuclear loci involved in mtDNA replication, maintenance and novel pathways. These loci may act by conferring a replicative advantage to certain mtDNA alleles. As an illustrative example, we identify a length variant carried by more than 50% of humans at position chrM:302 within a G-quadruplex previously proposed to mediate mtDNA transcription/replication switching2,3. We find that this variant exerts cis-acting genetic control over mtDNA abundance and is itself associated in-trans with nuclear loci encoding machinery for this regulatory switch. Our study suggests that common variation in the nuclear genome can shape variation in mtCN and heteroplasmy dynamics across the human population. We quantify mitochondrial DNA copy number and heteroplasmy levels and study their association with nuclear genetic loci in population-scale biobanks.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-023-06426-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41591-023-02423-3" target="_blank" rel="noopener">Leveraging fine-scale population structures for precision healthcare</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>
                  </div>
                  <div class="periodical">
                    <em>Nature Medicine</em>
                    <b>29</b>,
                    1611â€“1612
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dg9ly" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37464034" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41591-023-02423-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>A study investigates the links between fine-scale populations and outcomes including healthcare use and disease risk in a Los Angeles biobank, unveiling crucial insights into healthcare disparities in the USA.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41591-023-02423-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Weeks2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-023-01443-6" target="_blank" rel="noopener">Leveraging polygenic enrichments of gene features to predict genes underlying complex traits and diseases</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Weeks, E. M.,
                    </span>
                    <span class="coauthor">
                      Ulirsch, J. C.,
                    </span>
                    <span class="coauthor">
                      Cheng, N. Y.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Trippe, B. L.,
                      Fine, R. S.,
                      Miao, J.,
                      Patwardhan, T. A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Nasser, J.,
                      Fulco, C. P.,
                      Tashman, K. C.,
                      Aguet, F.,
                      Li, T.,
                      Ordovas-Montanes, J.,
                      Smillie, C. S.,
                      Biton, M.,
                      Shalek, A. K.,
                      Ananthakrishnan, A. N.,
                      Xavier, R. J.,
                      Regev, A.,
                      Gupta, R. M.,
                      Lage, K.,
                      Ardlie, K. G.,
                      Hirschhorn, J. N.,
                      Lander, E. S.,
                      Engreitz, J. M.,
                    </span>
                    and Finucane, H. K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>55</b>,
                    1267â€“1276
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/dg9jj" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/37443254" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.09.08.20190561" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/FinucaneLab/pops" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.finucanelab.org/data" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-023-01443-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Genome-wide association studies (GWASs) are a valuable tool for understanding the biology of complex human traits and diseases, but associated variants rarely point directly to causal genes. In the present study, we introduce a new method, polygenic priority score (PoPS), that learns trait-relevant gene features, such as cell-type-specific expression, to prioritize genes at GWAS loci. Using a large evaluation set of genes with fine-mapped coding variants, we show that PoPS and the closest gene individually outperform other gene prioritization methods, but observe the best overall performance by combining PoPS with orthogonal methods. Using this combined approach, we prioritize 10,642 unique geneâ€“trait pairs across 113 complex traits and diseases with high precision, finding not only well-established geneâ€“trait relationships but nominating new genes at unresolved loci, such as LGR4 for estimated glomerular filtration rate and CCR7 for deep vein thrombosis. Overall, we demonstrate that PoPS provides a powerful addition to the gene prioritization toolbox. Polygenic Priority Score (PoPS) prioritizes candidate effector genes at complex trait loci by integrating genome-wide association summary statistics with other data types. Combining PoPS with methods that leverage local genetic signals further improves the performance.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-023-01443-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Palmer2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1126/science.abn8455" target="_blank" rel="noopener">Analysis of genetic dominance in the UK Biobank</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Palmer, D. S.,
                    </span>
                    <span class="coauthor">
                      Zhou, W.,
                    </span>
                    <span class="coauthor">
                      Abbott, L.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Wigdor, E. M.,
                      Baya, N.,
                      Churchhouse, C.,
                      Seed, C.,
                      Poterba, T.,
                      King, D.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Bloemendal, A.,
                    </span>
                    and Neale, B. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Science</em>
                    <b>379</b>,
                    1341â€“1348
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36996212" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.08.15.456387" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/astheeggeggs/d-ldsc" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://docs.google.com/spreadsheets/d/1w0czvXqr_tgITgyrs53ji-3aOIkq8uGu9ZJHUauHO7E/edit#gid=1191380170" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1126/science.abn8455" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Classical statistical genetics theory defines dominance as any deviation from a purely additive, or dosage, effect of a genotype on a trait, which is known as the dominance deviation. Dominance is well documented in plant and animal breeding. Outside of rare monogenic traits, however, evidence in humans is limited. We systematically examined common genetic variation across 1060 traits in a large population cohort (UK Biobank, N = 361,194 samples analyzed) for evidence of dominance effects. We then developed a computationally efficient method to rapidly assess the aggregate contribution of dominance deviations to heritability. Lastly, observing that dominance associations are inherently less correlated between sites at a genomic locus than their additive counterparts, we explored whether they may be leveraged to identify causal variants more confidently.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1126/science.abn8455" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Ogawa2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.jid.2023.01.037" target="_blank" rel="noopener">A cross-trait genetic correlation study identified eight diseases and traits associated with psoriasis</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Ogawa, K.,
                    </span>
                    <span class="coauthor">
                      Tsoi, L. C.,
                    </span>
                    <span class="coauthor">
                      Tanaka, H.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Stuart, P. E.,
                      Nair, R. P.,
                      Tanaka, Y.,
                      Mochizuki, H.,
                      Elder, J. T.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Journal of Investigative Dermatology</em>
                    <b>143</b>,
                    1813â€“1816.e2
                    (2023)
                  </div>
                  <div class="links">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36906125" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.jid.2023.01.037" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.jid.2023.01.037" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Ramo2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-022-32936-3" target="_blank" rel="noopener">Genome-wide screen of otosclerosis in population biobanks: 27 loci and shared associations with skeletal structure</a></div>
                  <div class="author">
                    <span class="coauthor">
                      RÃ¤mÃ¶, J. T.,
                    </span>
                    <span class="coauthor">
                      Kiiskinen, T.,
                    </span>
                    <span class="coauthor">
                      Seist, R.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Krebs, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karjalainen, J.,
                      Kurki, M.,
                      HÃ¤mÃ¤lÃ¤inen, E.,
                      HÃ¤ppÃ¶lÃ¤, P.,
                      Havulinna, A. S.,
                      Hautakangas, H.,
                      FinnGen.,
                      MÃ¤gi, R.,
                      Palta, P.,
                      Esko, T.,
                      Metspalu, A.,
                      Pirinen, M.,
                      Karczewski, K. J.,
                      Ripatti, S.,
                      Milani, L.,
                      Stankovic, K. M.,
                      MÃ¤kitie, A.,
                      Daly, M. J.,
                    </span>
                    and Palotie, A.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>14</b>,
                    157
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/c4Jpm" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36653343" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.11.15.20227868" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-022-32936-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Otosclerosis is one of the most common causes of conductive hearing loss, affecting 0.3% of the population. It typically presents in adulthood and half of the patients have a positive family history. The pathophysiology of otosclerosis is poorly understood. A previous genome-wide association study (GWAS) identified a single association locus in an intronic region of RELN. Here, we report a meta-analysis of GWAS studies of otosclerosis in three population-based biobanks comprising 3504 cases and 861,198 controls. We identify 23 novel risk loci (p &lt; 5 \times 10-8) and report an association in RELN and three previously reported candidate gene or linkage regions (TGFB1, MEPE, and OTSC7). We demonstrate developmental stage-dependent immunostaining patterns of MEPE and RUNX2 in mouse otic capsules. In most association loci, the nearest protein-coding genes are implicated in bone remodelling, mineralization or severe skeletal disorders. We highlight multiple genes involved in transforming growth factor beta signalling for follow-up studies.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-022-32936-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kurki2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-022-05473-8" target="_blank" rel="noopener">FinnGen provides genetic insights from a well-phenotyped isolated population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Kurki, M. I.,
                    </span>
                    <span class="coauthor">
                      Karjalainen, J.,
                    </span>
                    <span class="coauthor">
                      Palta, P.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      SipilÃ¤, T. P.,
                      Kristiansson, K.,
                      Donner, K. M.,
                      Reeve, M. P.,
                      Laivuori, H.,
                      Aavikko, M.,
                      Kaunisto, M. A.,
                      Loukola, A.,
                      Lahtela, E.,
                      Mattsson, H.,
                      Laiho, P.,
                      Della Briotta Parolo, P.,
                      Lehisto, A. A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Mars, N.,
                      RÃ¤mÃ¶, J.,
                      Kiiskinen, T.,
                      Heyne, H. O.,
                      Veerapen, K.,
                      RÃ¼eger, S.,
                      LemmelÃ¤, S.,
                      Zhou, W.,
                      Ruotsalainen, S.,
                      PÃ¤rn, K.,
                      Hiekkalinna, T.,
                      Koskelainen, S.,
                      Paajanen, T.,
                      Llorens, V.,
                      Gracia-Tabuenca, J.,
                      Siirtola, H.,
                      Reis, K.,
                      Elnahas, A. G.,
                      Sun, B.,
                      Foley, C. N.,
                      Aalto-SetÃ¤lÃ¤, K.,
                      Alasoo, K.,
                      Arvas, M.,
                      Auro, K.,
                      Biswas, S.,
                      Bizaki-Vallaskangas, A.,
                      Carpen, O.,
                      Chen, C.-Y.,
                      Dada, O. A.,
                      Ding, Z.,
                      Ehm, M. G.,
                      Eklund, K.,
                      FÃ¤rkkilÃ¤, M.,
                      Finucane, H.,
                      Ganna, A.,
                      Ghazal, A.,
                      Graham, R. R.,
                      Green, E. M.,
                      Hakanen, A.,
                      Hautalahti, M.,
                      Hedman, Ã…. K.,
                      Hiltunen, M.,
                      Hinttala, R.,
                      Hovatta, I.,
                      Hu, X.,
                      Huertas-Vazquez, A.,
                      Huilaja, L.,
                      Hunkapiller, J.,
                      Jacob, H.,
                      Jensen, J.-N.,
                      Joensuu, H.,
                      John, S.,
                      Julkunen, V.,
                      Jung, M.,
                      Junttila, J.,
                      Kaarniranta, K.,
                      KÃ¤hÃ¶nen, M.,
                      Kajanne, R.,
                      Kallio, L.,
                      KÃ¤lviÃ¤inen, R.,
                      Kaprio, J.,
                      FinnGen.,
                      Kerimov, N.,
                      Kettunen, J.,
                      KilpelÃ¤inen, E.,
                      Kilpi, T.,
                      Klinger, K.,
                      Kosma, V.-M.,
                      Kuopio, T.,
                      Kurra, V.,
                      Laisk, T.,
                      Laukkanen, J.,
                      Lawless, N.,
                      Liu, A.,
                      Longerich, S.,
                      MÃ¤gi, R.,
                      MÃ¤kelÃ¤, J.,
                      MÃ¤kitie, A.,
                      Malarstig, A.,
                      Mannermaa, A.,
                      Maranville, J.,
                      Matakidou, A.,
                      Meretoja, T.,
                      Mozaffari, S. V.,
                      Niemi, M. E. K.,
                      Niemi, M.,
                      Niiranen, T.,
                      O Donnell, C. J.,
                      Obeidat, M. E.,
                      Okafo, G.,
                      Ollila, H. M.,
                      PalomÃ¤ki, A.,
                      Palotie, T.,
                      Partanen, J.,
                      Paul, D. S.,
                      Pelkonen, M.,
                      Pendergrass, R. K.,
                      Petrovski, S.,
                      PitkÃ¤ranta, A.,
                      Platt, A.,
                      Pulford, D.,
                      Punkka, E.,
                      Pussinen, P.,
                      Raghavan, N.,
                      Rahimov, F.,
                      Rajpal, D.,
                      Renaud, N. A.,
                      Riley-Gillis, B.,
                      Rodosthenous, R.,
                      Saarentaus, E.,
                      Salminen, A.,
                      Salminen, E.,
                      Salomaa, V.,
                      Schleutker, J.,
                      Serpi, R.,
                      Shen, H.-Y.,
                      Siegel, R.,
                      Silander, K.,
                      Siltanen, S.,
                      Soini, S.,
                      Soininen, H.,
                      Sul, J. H.,
                      Tachmazidou, I.,
                      Tasanen, K.,
                      Tienari, P.,
                      Toppila-Salmi, S.,
                      Tukiainen, T.,
                      Tuomi, T.,
                      Turunen, J. A.,
                      Ulirsch, J. C.,
                      Vaura, F.,
                      Virolainen, P.,
                      Waring, J.,
                      Waterworth, D.,
                      Yang, R.,
                      Nelis, M.,
                      Reigo, A.,
                      Metspalu, A.,
                      Milani, L.,
                      Esko, T.,
                      Fox, C.,
                      Havulinna, A. S.,
                      Perola, M.,
                      Ripatti, S.,
                      Jalanko, A.,
                      Laitinen, T.,
                      MÃ¤kelÃ¤, T. P.,
                      Plenge, R.,
                      McCarthy, M.,
                      Runz, H.,
                      Daly, M. J.,
                    </span>
                    and Palotie, A.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>613</b>,
                    508â€“518
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/c4Jpk" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36653562" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.03.03.22271360" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.finngen.fi/en/access_results" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <a href="https://www.finngen.fi/en" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <a href="https://www.finngen.fi/en/power-genetic-isolate-hundreds-novel-genetic-discoveries-finngen-study" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-022-05473-8" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% â‰¤minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored1,2. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 \times 10-11) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-022-05473-8" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Wang_Y2023" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2022.100241" target="_blank" rel="noopener">Global Biobank analyses provide lessons for developing polygenic risk scores across diverse cohorts</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Wang, Y.,
                    </span>
                    <span class="coauthor">
                      Namba, S.,
                    </span>
                    <span class="coauthor">
                      Lopera, E.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kerminen, S.,
                      Tsuo, K.,
                      LÃ¤ll, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, W.,
                      Wu, K.-H.,
                      FavÃ©, M.-J.,
                      Bhatta, L.,
                      Awadalla, P.,
                      Brumpton, B.,
                      Deelen, P.,
                      Hveem, K.,
                      Lo Faro, V.,
                      MÃ¤gi, R.,
                      Murakami, Y.,
                      Sanna, S.,
                      Smoller, J. W.,
                      Uzunovic, J.,
                      Wolford, B. N.,
                      Global Biobank Meta-analysis Initiative.,
                      Willer, C.,
                      Gamazon, E. R.,
                      Cox, N. J.,
                      Surakka, I.,
                      Okada, Y.,
                      Martin, A. R.,
                    </span>
                    and Hirbo, J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>3</b>,
                    100241
                    (2023)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(22)00204-X" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36777179" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://www.pgscatalog.org/publication/PGP000262/" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <a href="https://www.globalbiobankmeta.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2022.100241" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores (PRSs) have been widely explored in precision medicine. However, few studies have thoroughly investigated their best practices in global populations across different diseases. We here utilized data from Global Biobank Meta-analysis Initiative (GBMI) to explore methodological considerations and PRS performance in 9 different biobanks for 14 disease endpoints. Specifically, we constructed PRSs using pruning and thresholding (P + T) and PRS-continuous shrinkage (CS). For both methods, using a European-based linkage disequilibrium (LD) reference panel resulted in comparable or higher prediction accuracy compared with several other non-European-based panels. PRS-CS overall outperformed the classic P + T method, especially for endpoints with higher SNP-based heritability. Notably, prediction accuracy is heterogeneous across endpoints, biobanks, and ancestries, especially for asthma, which has known variation in disease prevalence across populations. Overall, we provide lessons for PRS construction, evaluation, and interpretation using GBMI resources and highlight the importance of best practices for PRS in the biobank-scale genomics era.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2022.100241" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2022">2022</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanoni2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1186/s13059-022-02837-1" target="_blank" rel="noopener">Implicating genes, pleiotropy, and sexual dimorphism at blood lipid loci through multi-ancestry meta-analysis</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Kanoni, S.,
                    </span>
                    <span class="coauthor">
                      Graham, S. E.,
                    </span>
                    <span class="coauthor">
                      Wang, Y.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Surakka, I.,
                      Ramdas, S.,
                      Zhu, X.,
                      Clarke, S. L.,
                      Bhatti, K. F.,
                      Vedantam, S.,
                      Winkler, T. W.,
                      Locke, A. E.,
                      Marouli, E.,
                      Zajac, G. J. M.,
                      Wu, K.-H. H.,
                      Ntalla, I.,
                      Hui, Q.,
                      Klarin, D.,
                      Hilliard, A. T.,
                      Wang, Z.,
                      Xue, C.,
                      Thorleifsson, G.,
                      Helgadottir, A.,
                      Gudbjartsson, D. F.,
                      Holm, H.,
                      Olafsson, I.,
                      Hwang, M. Y.,
                      Han, S.,
                      Akiyama, M.,
                      Sakaue, S.,
                      Terao, C.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, W.,
                      Brumpton, B. M.,
                      Rasheed, H.,
                      Havulinna, A. S.,
                      Veturi, Y.,
                      Pacheco, J. A.,
                      Rosenthal, E. A.,
                      Lingren, T.,
                      Feng, Q.,
                      Kullo, I. J.,
                      Narita, A.,
                      Takayama, J.,
                      Martin, H. C.,
                      Hunt, K. A.,
                      Trivedi, B.,
                      Haessler, J.,
                      Giulianini, F.,
                      Bradford, Y.,
                      Miller, J. E.,
                      Campbell, A.,
                      Lin, K.,
                      Millwood, I. Y.,
                      Rasheed, A.,
                      Hindy, G.,
                      Faul, J. D.,
                      Zhao, W.,
                      Weir, D. R.,
                      Turman, C.,
                      Huang, H.,
                      Graff, M.,
                      Choudhury, A.,
                      Sengupta, D.,
                      Mahajan, A.,
                      Brown, M. R.,
                      Zhang, W.,
                      Yu, K.,
                      Schmidt, E. M.,
                      Pandit, A.,
                      Gustafsson, S.,
                      Yin, X.,
                      Luan, J.,
                      Zhao, J.-H.,
                      Matsuda, F.,
                      Jang, H.-M.,
                      Yoon, K.,
                      Medina-Gomez, C.,
                      Pitsillides, A.,
                      Hottenga, J. J.,
                      Wood, A. R.,
                      Ji, Y.,
                      Gao, Z.,
                      Haworth, S.,
                      Yousri, N. A.,
                      Mitchell, R. E.,
                      Chai, J. F.,
                      Aadahl, M.,
                      Bjerregaard, A. A.,
                      Yao, J.,
                      Manichaikul, A.,
                      Hwu, C.-M.,
                      Hung, Y.-J.,
                      Warren, H. R.,
                      Ramirez, J.,
                      Bork-Jensen, J.,
                      KÃ¥rhus, L. L.,
                      Goel, A.,
                      Sabater-Lleal, M.,
                      Noordam, R.,
                      Mauro, P.,
                      Matteo, F.,
                      McDaid, A. F.,
                      Marques-Vidal, P.,
                      Wielscher, M.,
                      Trompet, S.,
                      Sattar, N.,
                      MÃ¸llehave, L. T.,
                      Munz, M.,
                      Zeng, L.,
                      Huang, J.,
                      Yang, B.,
                      Poveda, A.,
                      Kurbasic, A.,
                      Lamina, C.,
                      Forer, L.,
                      Scholz, M.,
                      Galesloot, T. E.,
                      Bradfield, J. P.,
                      Ruotsalainen, S. E.,
                      Daw, E.,
                      Zmuda, J. M.,
                      Mitchell, J. S.,
                      Fuchsberger, C.,
                      Christensen, H.,
                      Brody, J. A.,
                      Vazquez-Moreno, M.,
                      Feitosa, M. F.,
                      Wojczynski, M. K.,
                      Wang, Z.,
                      Preuss, M. H.,
                      Mangino, M.,
                      Christofidou, P.,
                      Verweij, N.,
                      Benjamins, J. W.,
                      Engmann, J.,
                      Tsao, N. L.,
                      Verma, A.,
                      Slieker, R. C.,
                      Lo, K. S.,
                      Zilhao, N. R.,
                      Le, P.,
                      Kleber, M. E.,
                      Delgado, G. E.,
                      Huo, S.,
                      Ikeda, D. D.,
                      Iha, H.,
                      Yang, J.,
                      Liu, J.,
                      Demirkan, A.,
                      Leonard, H. L.,
                      Marten, J.,
                      Frank, M.,
                      Schmidt, B.,
                      Smyth, L. J.,
                      CaÃ±adas-Garre, M.,
                      Wang, C.,
                      Nakatochi, M.,
                      Wong, A.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Sim, X.,
                      Xia, R.,
                      Huerta-Chagoya, A.,
                      Fernandez-Lopez, J. C.,
                      Lyssenko, V.,
                      Nongmaithem, S. S.,
                      Bayyana, S.,
                      Stringham, H. M.,
                      Irvin, M. R.,
                      Oldmeadow, C.,
                      Kim, H.-N.,
                      Ryu, S.,
                      Timmers, P. R. H. J.,
                      Arbeeva, L.,
                      Dorajoo, R.,
                      Lange, L. A.,
                      Prasad, G.,
                      LorÃ©s-Motta, L.,
                      Pauper, M.,
                      Long, J.,
                      Li, X.,
                      Theusch, E.,
                      Takeuchi, F.,
                      Spracklen, C. N.,
                      Loukola, A.,
                      Bollepalli, S.,
                      Warner, S. C.,
                      Wang, Y. X.,
                      Wei, W. B.,
                      Nutile, T.,
                      Ruggiero, D.,
                      Sung, Y. J.,
                      Chen, S.,
                      Liu, F.,
                      Yang, J.,
                      Kentistou, K. A.,
                      Banas, B.,
                      Nardone, G. G.,
                      Meidtner, K.,
                      Bielak, L. F.,
                      Smith, J. A.,
                      Hebbar, P.,
                      Farmaki, A.-E.,
                      Hofer, E.,
                      Lin, M.,
                      Concas, M. P.,
                      Vaccargiu, S.,
                      Most, P. J.,
                      PitkÃ¤nen, N.,
                      Cade, B. E.,
                      Laan, S. W.,
                      Chitrala, K. N.,
                      Weiss, S.,
                      Bentley, A. R.,
                      Doumatey, A. P.,
                      Adeyemo, A. A.,
                      Lee, J. Y.,
                      Petersen, E. R. B.,
                      Nielsen, A. A.,
                      Choi, H. S.,
                      Nethander, M.,
                      Freitag-Wolf, S.,
                      Southam, L.,
                      Rayner, N. W.,
                      Wang, C. A.,
                      Lin, S.-Y.,
                      Wang, J.-S.,
                      Couture, C.,
                      LyytikÃ¤inen, L.-P.,
                      Nikus, K.,
                      Cuellar-Partida, G.,
                      Vestergaard, H.,
                      Hidalgo, B.,
                      Giannakopoulou, O.,
                      Cai, Q.,
                      Obura, M. O.,
                      Setten, J.,
                      Li, X.,
                      Liang, J.,
                      Tang, H.,
                      Terzikhan, N.,
                      Shin, J. H.,
                      Jackson, R. D.,
                      Reiner, A. P.,
                      Martin, L. W.,
                      Chen, Z.,
                      Li, L.,
                      Kawaguchi, T.,
                      Thiery, J.,
                      Bis, J. C.,
                      Launer, L. J.,
                      Li, H.,
                      Nalls, M. A.,
                      Raitakari, O. T.,
                      Ichihara, S.,
                      Wild, S. H.,
                      Nelson, C. P.,
                      Campbell, H.,
                      JÃ¤ger, S.,
                      Nabika, T.,
                      Al-Mulla, F.,
                      Niinikoski, H.,
                      Braund, P. S.,
                      Kolcic, I.,
                      Kovacs, P.,
                      Giardoglou, T.,
                      Katsuya, T.,
                      Kleijn, D.,
                      Borst, G. J.,
                      Kim, E. K.,
                      Adams, H. H. H.,
                      Ikram, M. A.,
                      Zhu, X.,
                      Asselbergs, F. W.,
                      Kraaijeveld, A. O.,
                      Beulens, J. W. J.,
                      Shu, X.-O.,
                      Rallidis, L. S.,
                      Pedersen, O.,
                      Hansen, T.,
                      Mitchell, P.,
                      Hewitt, A. W.,
                      KÃ¤hÃ¶nen, M.,
                      PÃ©russe, L.,
                      Bouchard, C.,
                      TÃ¶njes, A.,
                      Chen, Y.-D. I.,
                      Pennell, C. E.,
                      Mori, T. A.,
                      Lieb, W.,
                      Franke, A.,
                      Ohlsson, C.,
                      MellstrÃ¶m, D.,
                      Cho, Y. S.,
                      Lee, H.,
                      Yuan, J.-M.,
                      Koh, W.-P.,
                      Rhee, S. Y.,
                      Woo, J.-T.,
                      Heid, I. M.,
                      Stark, K. J.,
                      Zimmermann, M. E.,
                      VÃ¶lzke, H.,
                      Homuth, G.,
                      Evans, M. K.,
                      Zonderman, A. B.,
                      Polasek, O.,
                      Pasterkamp, G.,
                      Hoefer, I. E.,
                      Redline, S.,
                      Pahkala, K.,
                      Oldehinkel, A. J.,
                      Snieder, H.,
                      Biino, G.,
                      Schmidt, R.,
                      Schmidt, H.,
                      Bandinelli, S.,
                      Dedoussis, G.,
                      Thanaraj, T. A.,
                      Kardia, S. L. R.,
                      Peyser, P. A.,
                      Kato, N.,
                      Schulze, M. B.,
                      Girotto, G.,
                      BÃ¶ger, C. A.,
                      Jung, B.,
                      Joshi, P. K.,
                      Bennett, D. A.,
                      De Jager, P. L.,
                      Lu, X.,
                      Mamakou, V.,
                      Brown, M.,
                      Caulfield, M. J.,
                      Munroe, P. B.,
                      Guo, X.,
                      Ciullo, M.,
                      Jonas, J. B.,
                      Samani, N. J.,
                      Kaprio, J.,
                      Pajukanta, P.,
                      TusiÃ©-Luna, T.,
                      Aguilar-Salinas, C. A.,
                      Adair, L. S.,
                      Bechayda, S. A.,
                      Silva, H. J.,
                      Wickremasinghe, A. R.,
                      Krauss, R. M.,
                      Wu, J.-Y.,
                      Zheng, W.,
                      Hollander, A. I.,
                      Bharadwaj, D.,
                      Correa, A.,
                      Wilson, J. G.,
                      Lind, L.,
                      Heng, C.-K.,
                      Nelson, A. E.,
                      Golightly, Y. M.,
                      Wilson, J. F.,
                      Penninx, B.,
                      Kim, H.-L.,
                      Attia, J.,
                      Scott, R. J.,
                      Rao, D. C.,
                      Arnett, D. K.,
                      Hunt, S. C.,
                      Walker, M.,
                      Koistinen, H. A.,
                      Chandak, G. R.,
                      Mercader, J. M.,
                      Costanzo, M. C.,
                      Jang, D.,
                      Burtt, N. P.,
                      Villalpando, C. G.,
                      Orozco, L.,
                      Fornage, M.,
                      Tai, E.,
                      Dam, R. M.,
                      LehtimÃ¤ki, T.,
                      Chaturvedi, N.,
                      Yokota, M.,
                      Liu, J.,
                      Reilly, D. F.,
                      McKnight, A. J.,
                      Kee, F.,
                      JÃ¶ckel, K.-H.,
                      McCarthy, M. I.,
                      Palmer, C. N. A.,
                      Vitart, V.,
                      Hayward, C.,
                      Simonsick, E.,
                      Duijn, C. M.,
                      Jin, Z.-B.,
                      Qu, J.,
                      Hishigaki, H.,
                      Lin, X.,
                      MÃ¤rz, W.,
                      Gudnason, V.,
                      Tardif, J.-C.,
                      Lettre, G.,
                      Hart, L. M. â€™t.,
                      Elders, P. J. M.,
                      Damrauer, S. M.,
                      Kumari, M.,
                      Kivimaki, M.,
                      Harst, P.,
                      Spector, T. D.,
                      Loos, R. J. F.,
                      Province, M. A.,
                      Parra, E. J.,
                      Cruz, M.,
                      Psaty, B. M.,
                      Brandslund, I.,
                      Pramstaller, P. P.,
                      Rotimi, C. N.,
                      Christensen, K.,
                      Ripatti, S.,
                      WidÃ©n, E.,
                      Hakonarson, H.,
                      Grant, S. F. A.,
                      Kiemeney, L. A. L. M.,
                      Graaf, J.,
                      Loeffler, M.,
                      Kronenberg, F.,
                      Gu, D.,
                      Erdmann, J.,
                      Schunkert, H.,
                      Franks, P. W.,
                      Linneberg, A.,
                      Jukema, J. W.,
                      Khera, A. V.,
                      MÃ¤nnikkÃ¶, M.,
                      Jarvelin, M.-R.,
                      Kutalik, Z.,
                      Francesco, C.,
                      Mook-Kanamori, D. O.,
                      Dijk, K. W.,
                      Watkins, H.,
                      Strachan, D. P.,
                      Grarup, N.,
                      Sever, P.,
                      Poulter, N.,
                      Chuang, L.-M.,
                      Rotter, J. I.,
                      Dantoft, T. M.,
                      Karpe, F.,
                      Neville, M. J.,
                      Timpson, N. J.,
                      Cheng, C.-Y.,
                      Wong, T.-Y.,
                      Khor, C. C.,
                      Li, H.,
                      Sabanayagam, C.,
                      Peters, A.,
                      Gieger, C.,
                      Hattersley, A. T.,
                      Pedersen, N. L.,
                      Magnusson, P. K. E.,
                      Boomsma, D. I.,
                      Willemsen, A. H. M.,
                      Cupples, L.,
                      Meurs, J. B. J.,
                      Ghanbari, M.,
                      Gordon-Larsen, P.,
                      Huang, W.,
                      Kim, Y. J.,
                      Tabara, Y.,
                      Wareham, N. J.,
                      Langenberg, C.,
                      Zeggini, E.,
                      Kuusisto, J.,
                      Laakso, M.,
                      Ingelsson, E.,
                      Abecasis, G.,
                      Chambers, J. C.,
                      Kooner, J. S.,
                      Vries, P. S.,
                      Morrison, A. C.,
                      Hazelhurst, S.,
                      Ramsay, M.,
                      North, K. E.,
                      Daviglus, M.,
                      Kraft, P.,
                      Martin, N. G.,
                      Whitfield, J. B.,
                      Abbas, S.,
                      Saleheen, D.,
                      Walters, R. G.,
                      Holmes, M. V.,
                      Black, C.,
                      Smith, B. H.,
                      Baras, A.,
                      Justice, A. E.,
                      Buring, J. E.,
                      Ridker, P. M.,
                      Chasman, D. I.,
                      Kooperberg, C.,
                      Tamiya, G.,
                      Yamamoto, M.,
                      Heel, D. A.,
                      Trembath, R. C.,
                      Wei, W.-Q.,
                      Jarvik, G. P.,
                      Namjou, B.,
                      Hayes, M. G.,
                      Ritchie, M. D.,
                      Jousilahti, P.,
                      Salomaa, V.,
                      Hveem, K.,
                      Ã…svold, B. O.,
                      Kubo, M.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Murakami, Y.,
                      Kim, B.-J.,
                      Thorsteinsdottir, U.,
                      Stefansson, K.,
                      Zhang, J.,
                      Chen, Y.,
                      Ho, Y.-L.,
                      Lynch, J. A.,
                      Rader, D. J.,
                      Tsao, P. S.,
                      Chang, K.-M.,
                      Cho, K.,
                      Oâ€™Donnell, C. J.,
                      Gaziano, J. M.,
                      Wilson, P. W. F.,
                      Frayling, T. M.,
                      Hirschhorn, J. N.,
                      Kathiresan, S.,
                      Mohlke, K. L.,
                      Sun, Y. V.,
                      Morris, A. P.,
                      Boehnke, M.,
                      Brown, C. D.,
                      Natarajan, P.,
                      Deloukas, P.,
                      Willer, C. J.,
                      Assimes, T. L.,
                    </span>
                    and Peloso, G. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Genome Biology</em>
                    <b>23</b>,
                    268
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/c2qvy" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36575460" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1186/s13059-022-02837-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>BACKGROUND: Genetic variants within nearly 1000 loci are known to contribute to modulation of blood lipid levels. However, the biological pathways underlying these associations are frequently unknown, limiting understanding of these findings and hindering downstream translational efforts such as drug target discovery. RESULTS: To expand our understanding of the underlying biological pathways and mechanisms controlling blood lipid levels, we leverage a large multi-ancestry meta-analysis (N = 1,654,960) of blood lipids to prioritize putative causal genes for 2286 lipid associations using six gene prediction approaches. Using phenome-wide association (PheWAS) scans, we identify relationships of genetically predicted lipid levels to other diseases and conditions. We confirm known pleiotropic associations with cardiovascular phenotypes and determine novel associations, notably with cholelithiasis risk. We perform sex-stratified GWAS meta-analysis of lipid levels and show that 3-5% of autosomal lipid-associated loci demonstrate sex-biased effects. Finally, we report 21 novel lipid loci identified on the X chromosome. Many of the sex-biased autosomal and X chromosome lipid loci show pleiotropic associations with sex hormones, emphasizing the role of hormone regulation in lipid metabolism. CONCLUSIONS: Taken together, our findings provide insights into the biological mechanisms through which associated variants lead to altered lipid levels and potentially cardiovascular disease risk.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1186/s13059-022-02837-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Tsuo2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2022.100212" target="_blank" rel="noopener">Multi-ancestry meta-analysis of asthma identifies novel associations and highlights the value of increased power and diversity</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Tsuo, K.,
                    </span>
                    <span class="coauthor">
                      Zhou, W.,
                    </span>
                    <span class="coauthor">
                      Wang, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Namba, S.,
                      Gupta, R.,
                      Majara, L.,
                      Nkambule, L. L.,
                      Morisaki, T.,
                      Okada, Y.,
                      Neale, B. M.,
                      Global Biobank Meta-analysis Initiative.,
                      Daly, M. J.,
                    </span>
                    and Martin, A. R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>2</b>,
                    100212
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(22)00165-3" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36778051" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.11.30.21267108" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.globalbiobankmeta.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2022.100212" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Asthma is a complex disease that varies widely in prevalence across populations. The extent to which genetic variation contributes to these disparities is unclear, as the genetics underlying asthma have been investigated primarily in populations of European descent. As part of the Global Biobank Meta-analysis Initiative, we conducted a large-scale genome-wide association study of asthma (153,763 cases and 1,647,022 controls) via meta-analysis across 22 biobanks spanning multiple ancestries. We discovered 179 asthma-associated loci, 49 of which were not previously reported. Despite the wide range in asthma prevalence among biobanks, we found largely consistent genetic effects across biobanks and ancestries. The meta-analysis also improved polygenic risk prediction in non-European populations compared with previous studies. Additionally, we found considerable genetic overlap between age-of-onset subtypes and between asthma and comorbid diseases. Our work underscores the multi-factorial nature of asthma development and offers insight into its shared genetic architecture.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2022.100212" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2022.100210" target="_blank" rel="noopener">Meta-analysis fine-mapping is often miscalibrated at single-variant resolution</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Elzur, R.,
                    </span>
                    <span class="coauthor">
                      Zhou, W.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Global Biobank Meta-analysis Initiative.,
                      Daly, M. J.,
                    </span>
                    and Finucane, H. K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>2</b>,
                    100210
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(22)00163-X" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36643910" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.03.16.22272457" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/mkanai/slalom" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.globalbiobankmeta.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2022.100210" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Meta-analysis is pervasively used to combine multiple genome-wide association studies (GWASs). Fine-mapping of meta-analysis studies is typically performed as in a single-cohort study. Here, we first demonstrate that heterogeneity (e.g., of sample size, phenotyping, imputation) hurts calibration of meta-analysis fine-mapping. We propose a summary statistics-based quality-control (QC) method, suspicious loci analysis of meta-analysis summary statistics (SLALOM), that identifies suspicious loci for meta-analysis fine-mapping by detecting outliers in association statistics. We validate SLALOM in simulations and the GWAS Catalog. Applying SLALOM to 14 meta-analyses from the Global Biobank Meta-analysis Initiative (GBMI), we find that 67% of loci show suspicious patterns that call into question fine-mapping accuracy. These predicted suspicious loci are significantly depleted for having nonsynonymous variants as lead variant (2.7\times; Fisherâ€™s exact p = 7.3 \times 10âˆ’4). We find limited evidence of fine-mapping improvement in the GBMI meta-analyses compared with individual biobanks. We urge extreme caution when interpreting fine-mapping results from meta-analysis of heterogeneous cohorts.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2022.100210" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Zhou2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2022.100192" target="_blank" rel="noopener">Global Biobank Meta-analysis Initiative: Powering genetic discovery across human disease</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Zhou, W.,
                    </span>
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Wu, K.-H. H.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Rasheed, H.,
                      Tsuo, K.,
                      Hirbo, J. B.,
                      Wang, Y.,
                      Bhattacharya, A.,
                      Zhao, H.,
                      Namba, S.,
                      Surakka, I.,
                      Wolford, B. N.,
                      Lo Faro, V.,
                      Lopera-Maya, E. A.,
                      LÃ¤ll, K.,
                      FavÃ©, M.-J.,
                      Partanen, J. J.,
                      Chapman, S. B.,
                      Karjalainen, J.,
                      Kurki, M.,
                      Maasha, M.,
                      Brumpton, B. M.,
                      Chavan, S.,
                      Chen, T.-T.,
                      Daya, M.,
                      Ding, Y.,
                      Feng, Y.-C. A.,
                      Guare, L. A.,
                      Gignoux, C. R.,
                      Graham, S. E.,
                      Hornsby, W. E.,
                      Ingold, N.,
                      Ismail, S. I.,
                      Johnson, R.,
                      Laisk, T.,
                      Lin, K.,
                      Lv, J.,
                      Millwood, I. Y.,
                      Moreno-Grau, S.,
                      Nam, K.,
                      Palta, P.,
                      Pandit, A.,
                      Preuss, M. H.,
                      Saad, C.,
                      Setia-Verma, S.,
                      Thorsteinsdottir, U.,
                      Uzunovic, J.,
                      Verma, A.,
                      Zawistowski, M.,
                      Zhong, X.,
                      Afifi, N.,
                      Al-Dabhani, K. M.,
                      Al Thani, A.,
                      Bradford, Y.,
                      Campbell, A.,
                      Crooks, K.,
                      Bock, G. H.,
                      Damrauer, S. M.,
                      Douville, N. J.,
                      Finer, S.,
                      Fritsche, L. G.,
                      Fthenou, E.,
                      Gonzalez-Arroyo, G.,
                      Griffiths, C. J.,
                      Guo, Y.,
                      Hunt, K. A.,
                      Ioannidis, A.,
                      Jansonius, N. M.,
                      Konuma, T.,
                      Lee, M. T. M.,
                      Lopez-Pineda, A.,
                      Matsuda, Y.,
                      Marioni, R. E.,
                      Moatamed, B.,
                      Nava-Aguilar, M. A.,
                      Numakura, K.,
                      Patil, S.,
                      Rafaels, N.,
                      Richmond, A.,
                      Rojas-MuÃ±oz, A.,
                      Shortt, J. A.,
                      Straub, P.,
                      Tao, R.,
                      Vanderwerff, B.,
                      Vernekar, M.,
                      Veturi, Y.,
                      Barnes, K. C.,
                      Boezen, M.,
                      Chen, Z.,
                      Chen, C.-Y.,
                      Cho, J.,
                      Smith, G. D.,
                      Finucane, H. K.,
                      Franke, L.,
                      Gamazon, E. R.,
                      Ganna, A.,
                      Gaunt, T. R.,
                      Ge, T.,
                      Huang, H.,
                      Huffman, J.,
                      Katsanis, N.,
                      Koskela, J. T.,
                      Lajonchere, C.,
                      Law, M. H.,
                      Li, L.,
                      Lindgren, C. M.,
                      Loos, R. J. F.,
                      MacGregor, S.,
                      Matsuda, K.,
                      Olsen, C. M.,
                      Porteous, D. J.,
                      Shavit, J. A.,
                      Snieder, H.,
                      Takano, T.,
                      Trembath, R. C.,
                      Vonk, J. M.,
                      Whiteman, D. C.,
                      Wicks, S. J.,
                      Wijmenga, C.,
                      Wright, J.,
                      Zheng, J.,
                      Zhou, X.,
                      Awadalla, P.,
                      Boehnke, M.,
                      Bustamante, C. D.,
                      Cox, N. J.,
                      Fatumo, S.,
                      Geschwind, D. H.,
                      Hayward, C.,
                      Hveem, K.,
                      Kenny, E. E.,
                      Lee, S.,
                      Lin, Y.-F.,
                      Mbarek, H.,
                      MÃ¤gi, R.,
                      Martin, H. C.,
                      Medland, S. E.,
                      Okada, Y.,
                      Palotie, A. V.,
                      Pasaniuc, B.,
                      Rader, D. J.,
                      Ritchie, M. D.,
                      Sanna, S.,
                      Smoller, J. W.,
                      Stefansson, K.,
                      Heel, D. A.,
                      Walters, R. G.,
                      ZÃ¶llner, S.,
                      Martin, A. R.,
                      Willer, C. J.,
                      Daly, M. J.,
                    </span>
                    and Neale, B. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>2</b>,
                    100192
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(22)00141-0" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36777996" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.11.19.21266436" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.globalbiobankmeta.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <a href="https://www.broadinstitute.org/news/global-initiative-shows-how-bringing-biobanks-together-can-yield-new-genetic-insights" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2022.100192" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Biobanks facilitate genome-wide association studies (GWASs), which have mapped genomic loci across a range of human diseases and traits. However, most biobanks are primarily composed of individuals of European ancestry. We introduce the Global Biobank Meta-analysis Initiative (GBMI)â€”a collaborative network of 23 biobanks from 4 continents representing more than 2.2 million consented individuals with genetic data linked to electronic health records. GBMI meta-analyzes summary statistics from GWASs generated using harmonized genotypes and phenotypes from member biobanks for 14 exemplar diseases and endpoints. This strategy validates that GWASs conducted in diverse biobanks can be integrated despite heterogeneity in case definitions, recruitment strategies, and baseline characteristics. This collaborative effort improves GWAS power for diseases, benefits understudied diseases, and improves risk prediction while also enabling the nomination of disease genes and drug candidates by incorporating gene and protein expression data and providing insight into the underlying biology of human diseases and traits.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2022.100192" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Bhattacharya2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2022.100180" target="_blank" rel="noopener">Best practices for multi-ancestry, meta-analytic transcriptome-wide association studies: Lessons from the Global Biobank Meta-analysis Initiative</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Bhattacharya, A.,
                    </span>
                    <span class="coauthor">
                      Hirbo, J. B.,
                    </span>
                    <span class="coauthor">
                      Zhou, D.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, W.,
                      Zheng, J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      The Global Biobank Meta-analysis Initiative.,
                      Pasaniuc, B.,
                      Gamazon, E. R.,
                    </span>
                    and Cox, N. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>2</b>,
                    100180
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(22)00125-2" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36341024" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.11.24.21266825" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.globalbiobankmeta.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2022.100180" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The Global Biobank Meta-analysis Initiative (GBMI), through its diversity, provides a valuable opportunity to study population-wide and ancestry-specific genetic associations. However, with multiple ascertainment strategies and multi-ancestry study populations across biobanks, GBMI presents unique challenges in implementing statistical genetics methods. Transcriptome-wide association studies (TWASs) boost detection power for and provide biological context to genetic associations by integrating genetic variant-to-trait associations from genome-wide association studies (GWASs) with predictive models of gene expression. TWASs present unique challenges beyond GWASs, especially in a multi-biobank, meta-analytic setting. Here, we present the GBMI TWAS pipeline, outlining practical considerations for ancestry and tissue specificity, meta-analytic strategies, and open challenges at every step of the framework. We advise conducting ancestry-stratified TWASs using ancestry-specific expression models and meta-analyzing results using inverse-variance weighting, showing the least test statistic inflation. Our work provides a foundation for adding transcriptomic context to biobank-linked GWASs, allowing for ancestry-aware discovery to accelerate genomic medicine.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2022.100180" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Yengo2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-022-05275-y" target="_blank" rel="noopener">A saturated map of common genetic variants associated with human height</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Yengo LoÄ±Ìˆc.,
                    </span>
                    <span class="coauthor">
                      Vedantam, S.,
                    </span>
                    <span class="coauthor">
                      Marouli, E.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Sidorenko, J.,
                      Bartell, E.,
                      Sakaue, S.,
                      Graff, M.,
                      Eliasen, A. U.,
                      Jiang, Y.,
                      Raghavan, S.,
                      Miao, J.,
                      Arias, J. D.,
                      Graham, S. E.,
                      Mukamel, R. E.,
                      Spracklen, C. N.,
                      Yin, X.,
                      Chen, S.-H.,
                      Ferreira, T.,
                      Highland, H. H.,
                      Ji, Y.,
                      Karaderi, T.,
                      Lin, K.,
                      LÃ¼ll, K.,
                      Malden, D. E.,
                      Medina-Gomez, C.,
                      Machado, M.,
                      Moore, A.,
                      RÃ¼eger, S.,
                      Sim, X.,
                      Vrieze, S.,
                      Ahluwalia, T. S.,
                      Akiyama, M.,
                      Allison, M. A.,
                      Alvarez, M.,
                      Andersen, M. K.,
                      Ani, A.,
                      Appadurai, V.,
                      Arbeeva, L.,
                      Bhaskar, S.,
                      Bielak, L. F.,
                      Bollepalli, S.,
                      Bonnycastle, L. L.,
                      Bork-Jensen, J.,
                      Bradfield, J. P.,
                      Bradford, Y.,
                      Braund, P. S.,
                      Brody, J. A.,
                      Burgdorf, K. S.,
                      Cade, B. E.,
                      Cai, H.,
                      Cai, Q.,
                      Campbell, A.,
                      CaÃ±adas-Garre, M.,
                      Catamo, E.,
                      Chai, J.-F.,
                      Chai, X.,
                      Chang, L.-C.,
                      Chang, Y.-C.,
                      Chen, C.-H.,
                      Chesi, A.,
                      Choi, S. H.,
                      Chung, R.-H.,
                      Cocca, M.,
                      Concas, M. P.,
                      Couture, C.,
                      Cuellar-Partida, G.,
                      Danning, R.,
                      Daw, E. W.,
                      Degenhard, F.,
                      Delgado, G. E.,
                      Delitala, A.,
                      Demirkan, A.,
                      Deng, X.,
                      Devineni, P.,
                      Dietl, A.,
                      Dimitriou, M.,
                      Dimitrov, L.,
                      Dorajoo, R.,
                      Ekici, A. B.,
                      Engmann, J. E.,
                      Fairhurst-Hunter, Z.,
                      Farmaki, A.-E.,
                      Faul, J. D.,
                      Fernandez-Lopez, J.-C.,
                      Forer, L.,
                      Francescatto, M.,
                      Freitag-Wolf, S.,
                      Fuchsberger, C.,
                      Galesloot, T. E.,
                      Gao, Y.,
                      Gao, Z.,
                      Geller, F.,
                      Giannakopoulou, O.,
                      Giulianini, F.,
                      Gjesing, A. P.,
                      Goel, A.,
                      Gordon, S. D.,
                      Gorski, M.,
                      Grove, J.,
                      Guo, X.,
                      Gustafsson, S.,
                      Haessler, J.,
                      Hansen, T. F.,
                      Havulinna, A. S.,
                      Haworth, S. J.,
                      He, J.,
                      Heard-Costa, N.,
                      Hebbar, P.,
                      Hindy, G.,
                      Ho, Y.-L. A.,
                      Hofer, E.,
                      Holliday, E.,
                      Horn, K.,
                      Hornsby, W. E.,
                      Hottenga, J.-J.,
                      Huang, H.,
                      Huang, J.,
                      Huerta-Chagoya, A.,
                      Huffman, J. E.,
                      Hung, Y.-J.,
                      Huo, S.,
                      Hwang, M. Y.,
                      Iha, H.,
                      Ikeda, D. D.,
                      Isono, M.,
                      Jackson, A. U.,
                      JÃ¤ger, S.,
                      Jansen, I. E.,
                      Johansson, I.,
                      Jonas, J. B.,
                      Jonsson, A.,
                      JÃ¸rgensen, T.,
                      Kalafati, I.-P.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kanoni, S.,
                      KÃ¥rhus, L. L.,
                      Kasturiratne, A.,
                      Katsuya, T.,
                      Kawaguchi, T.,
                      Kember, R. L.,
                      Kentistou, K. A.,
                      Kim, H.-N.,
                      Kim, Y. J.,
                      Kleber, M. E.,
                      Knol, M. J.,
                      Kurbasic, A.,
                      Lauzon, M.,
                      Le, P.,
                      Lea, R.,
                      Lee, J.-Y.,
                      Leonard, H. L.,
                      Li, S. A.,
                      Li, X.,
                      Li, X.,
                      Liang, J.,
                      Lin, H.,
                      Lin, S.-Y.,
                      Liu, J.,
                      Liu, X.,
                      Lo, K. S.,
                      Long, J.,
                      Lores-Motta, L.,
                      Luan, J.,
                      Lyssenko, V.,
                      LyytikÃ¤inen, L.-P.,
                      Mahajan, A.,
                      Mamakou, V.,
                      Mangino, M.,
                      Manichaikul, A.,
                      Marten, J.,
                      Mattheisen, M.,
                      Mavarani, L.,
                      McDaid, A. F.,
                      Meidtner, K.,
                      Melendez, T. L.,
                      Mercader, J. M.,
                      Milaneschi, Y.,
                      Miller, J. E.,
                      Millwood, I. Y.,
                      Mishra, P. P.,
                      Mitchell, R. E.,
                      MÃ¸llehave, L. T.,
                      Morgan, A.,
                      Mucha, S.,
                      Munz, M.,
                      Nakatochi, M.,
                      Nelson, C. P.,
                      Nethander, M.,
                      Nho, C. W.,
                      Nielsen, A. A.,
                      Nolte, I. M.,
                      Nongmaithem, S. S.,
                      Noordam, R.,
                      Ntalla, I.,
                      Nutile, T.,
                      Pandit, A.,
                      Christofidou, P.,
                      PÃ¤rna, K.,
                      Pauper, M.,
                      Petersen, E. R. B.,
                      Petersen, L. V.,
                      PitkÃ¤nen, N.,
                      PolaÅ¡ek, O.,
                      Poveda, A.,
                      Preuss, M. H.,
                      Pyarajan, S.,
                      Raffield, L. M.,
                      Rakugi, H.,
                      Ramirez, J.,
                      Rasheed, A.,
                      Raven, D.,
                      Rayner, N. W.,
                      Riveros, C.,
                      Rohde, R.,
                      Ruggiero, D.,
                      Ruotsalainen, S. E.,
                      Ryan, K. A.,
                      Sabater-Lleal, M.,
                      Saxena, R.,
                      Scholz, M.,
                      Sendamarai, A.,
                      Shen, B.,
                      Shi, J.,
                      Shin, J. H.,
                      Sidore, C.,
                      Sitlani, C. M.,
                      Slieker, R. C.,
                      Smit, R. A. J.,
                      Smith, A. V.,
                      Smith, J. A.,
                      Smyth, L. J.,
                      Southam, L.,
                      Steinthorsdottir, V.,
                      Sun, L.,
                      Takeuchi, F.,
                      Tallapragada, D. S. P.,
                      Taylor, K. D.,
                      Tayo, B. O.,
                      Tcheandjieu, C.,
                      Terzikhan, N.,
                      Tesolin, P.,
                      Teumer, A.,
                      Theusch, E.,
                      Thompson, D. J.,
                      Thorleifsson, G.,
                      Timmers, P. R. H. J.,
                      Trompet, S.,
                      Turman, C.,
                      Vaccargiu, S.,
                      Laan, S. W.,
                      Most, P. J.,
                      Klinken, J. B.,
                      Setten, J.,
                      Verma, S. S.,
                      Verweij, N.,
                      Veturi, Y.,
                      Wang, C. A.,
                      Wang, C.,
                      Wang, L.,
                      Wang, Z.,
                      Warren, H. R.,
                      Bin Wei, W.,
                      Wickremasinghe, A. R.,
                      Wielscher, M.,
                      Wiggins, K. L.,
                      Winsvold, B. S.,
                      Wong, A.,
                      Wu, Y.,
                      Wuttke, M.,
                      Xia, R.,
                      Xie, T.,
                      Yamamoto, K.,
                      Yang, J.,
                      Yao, J.,
                      Young, H.,
                      Yousri, N. A.,
                      Yu, L.,
                      Zeng, L.,
                      Zhang, W.,
                      Zhang, X.,
                      Zhao, J.-H.,
                      Zhao, W.,
                      Zhou, W.,
                      Zimmermann, M. E.,
                      Zoledziewska, M.,
                      Adair, L. S.,
                      Adams, H. H. H.,
                      Aguilar-Salinas, C. A.,
                      Al-Mulla, F.,
                      Arnett, D. K.,
                      Asselbergs, F. W.,
                      Ã…svold, B. O.,
                      Attia, J.,
                      Banas, B.,
                      Bandinelli, S.,
                      Bennett, D. A.,
                      Bergler, T.,
                      Bharadwaj, D.,
                      Biino, G.,
                      Bisgaard, H.,
                      Boerwinkle, E.,
                      BÃ¶ger, C. A.,
                      BÃ¸nnelykke, K.,
                      Boomsma, D. I.,
                      BÃ¸rglum, A. D.,
                      Borja, J. B.,
                      Bouchard, C.,
                      Bowden, D. W.,
                      Brandslund, I.,
                      Brumpton, B.,
                      Buring, J. E.,
                      Caulfield, M. J.,
                      Chambers, J. C.,
                      Chandak, G. R.,
                      Chanock, S. J.,
                      Chaturvedi, N.,
                      Chen, Y.-D. I.,
                      Chen, Z.,
                      Cheng, C.-Y.,
                      Christophersen, I. E.,
                      Ciullo, M.,
                      Cole, J. W.,
                      Collins, F. S.,
                      Cooper, R. S.,
                      Cruz, M.,
                      Cucca, F.,
                      Cupples, L. A.,
                      Cutler, M. J.,
                      Damrauer, S. M.,
                      Dantoft, T. M.,
                      Borst, G. J.,
                      Groot, L. C. P. G. M.,
                      De Jager, P. L.,
                      Kleijn, D. P. V.,
                      Silva, H.,
                      Dedoussis, G. V.,
                      Hollander, A. I.,
                      Du, S.,
                      Easton, D. F.,
                      Elders, P. J. M.,
                      Eliassen, A. H.,
                      Ellinor, P. T.,
                      ElmstÃ¥hl, S.,
                      Erdmann, J.,
                      Evans, M. K.,
                      Fatkin, D.,
                      Feenstra, B.,
                      Feitosa, M. F.,
                      Ferrucci, L.,
                      Ford, I.,
                      Fornage, M.,
                      Franke, A.,
                      Franks, P. W.,
                      Freedman, B. I.,
                      Gasparini, P.,
                      Gieger, C.,
                      Girotto, G.,
                      Goddard, M. E.,
                      Golightly, Y. M.,
                      Gonzalez-Villalpando, C.,
                      Gordon-Larsen, P.,
                      Grallert, H.,
                      Grant, S. F. A.,
                      Grarup, N.,
                      Griffiths, L.,
                      Gudnason, V.,
                      Haiman, C.,
                      Hakonarson, H.,
                      Hansen, T.,
                      Hartman, C. A.,
                      Hattersley, A. T.,
                      Hayward, C.,
                      Heckbert, S. R.,
                      Heng, C.-K.,
                      Hengstenberg, C.,
                      Hewitt, A. W.,
                      Hishigaki, H.,
                      Hoyng, C. B.,
                      Huang, P. L.,
                      Huang, W.,
                      Hunt, S. C.,
                      Hveem, K.,
                      HyppÃ¶nen, E.,
                      Iacono, W. G.,
                      Ichihara, S.,
                      Ikram, M. A.,
                      Isasi, C. R.,
                      Jackson, R. D.,
                      Jarvelin, M.-R.,
                      Jin, Z.-B.,
                      JÃ¶ckel, K.-H.,
                      Joshi, P. K.,
                      Jousilahti, P.,
                      Jukema, J. W.,
                      KÃ¤hÃ¶nen, M.,
                      Kamatani, Y.,
                      Kang, K. D.,
                      Kaprio, J.,
                      Kardia, S. L. R.,
                      Karpe, F.,
                      Kato, N.,
                      Kee, F.,
                      Kessler, T.,
                      Khera, A. V.,
                      Khor, C. C.,
                      Kiemeney, L. A. L. M.,
                      Kim, B.-J.,
                      Kim, E. K.,
                      Kim, H.-L.,
                      Kirchhof, P.,
                      Kivimaki, M.,
                      Koh, W.-P.,
                      Koistinen, H. A.,
                      Kolovou, G. D.,
                      Kooner, J. S.,
                      Kooperberg, C.,
                      KÃ¶ttgen, A.,
                      Kovacs, P.,
                      Kraaijeveld, A.,
                      Kraft, P.,
                      Krauss, R. M.,
                      Kumari, M.,
                      Kutalik, Z.,
                      Laakso, M.,
                      Lange, L. A.,
                      Langenberg, C.,
                      Launer, L. J.,
                      Le Marchand, L.,
                      Lee, H.,
                      Lee, N. R.,
                      LehtimÃ¤ki, T.,
                      Li, H.,
                      Li, L.,
                      Lieb, W.,
                      Lin, X.,
                      Lind, L.,
                      Linneberg, A.,
                      Liu, C.-T.,
                      Liu, J.,
                      Loeffler, M.,
                      London, B.,
                      Lubitz, S. A.,
                      Lye, S. J.,
                      Mackey, D. A.,
                      MÃ¤gi, R.,
                      Magnusson, P. K. E.,
                      Marcus, G. M.,
                      Vidal, P. M.,
                      Martin, N. G.,
                      MÃ¤rz, W.,
                      Matsuda, F.,
                      McGarrah, R. W.,
                      McGue, M.,
                      McKnight, A. J.,
                      Medland, S. E.,
                      MellstrÃ¶m, D.,
                      Metspalu, A.,
                      Mitchell, B. D.,
                      Mitchell, P.,
                      Mook-Kanamori, D. O.,
                      Morris, A. D.,
                      Mucci, L. A.,
                      Munroe, P. B.,
                      Nalls, M. A.,
                      Nazarian, S.,
                      Nelson, A. E.,
                      Neville, M. J.,
                      Newton-Cheh, C.,
                      Nielsen, C. S.,
                      NÃ¶then, M. M.,
                      Ohlsson, C.,
                      Oldehinkel, A. J.,
                      Orozco, L.,
                      Pahkala, K.,
                      Pajukanta, P.,
                      Palmer, C. N. A.,
                      Parra, E. J.,
                      Pattaro, C.,
                      Pedersen, O.,
                      Pennell, C. E.,
                      Penninx, B. W. J. H.,
                      Perusse, L.,
                      Peters, A.,
                      Peyser, P. A.,
                      Porteous, D. J.,
                      Power, C.,
                      Pramstaller, P. P.,
                      Province, M. A.,
                      Qi, Q.,
                      Qu, J.,
                      Rader, D. J.,
                      Raitakari, O. T.,
                      Ralhan, S.,
                      Rallidis, L. S.,
                      Rao, D. C.,
                      Redline, S.,
                      Reilly, D. F.,
                      Reiner, A. P.,
                      Rhee, S. Y.,
                      Ridker, P. M.,
                      Rienstra, M.,
                      Ripatti, S.,
                      Ritchie, M. D.,
                      Roden, D. M.,
                      Rosendaal, F. R.,
                      Rotter, J. I.,
                      Rudan, I.,
                      Rutters, F.,
                      Sabanayagam, C.,
                      Saleheen, D.,
                      Salomaa, V.,
                      Samani, N. J.,
                      Sanghera, D. K.,
                      Sattar, N.,
                      Schmidt, B.,
                      Schmidt, H.,
                      Schmidt, R.,
                      Schulze, M. B.,
                      Schunkert, H.,
                      Scott, L. J.,
                      Scott, R. J.,
                      Sever, P.,
                      Shiroma, E. J.,
                      Shoemaker, M. B.,
                      Shu, X.-O.,
                      Simonsick, E. M.,
                      Sims, M.,
                      Singh, J. R.,
                      Singleton, A. B.,
                      Sinner, M. F.,
                      Smith, J. G.,
                      Snieder, H.,
                      Spector, T. D.,
                      Stampfer, M. J.,
                      Stark, K. J.,
                      Strachan, D. P.,
                      Hart, L. M.,
                      Tabara, Y.,
                      Tang, H.,
                      Tardif, J.-C.,
                      Thanaraj, T. A.,
                      Timpson, N. J.,
                      TÃ¶njes, A.,
                      Tremblay, A.,
                      Tuomi, T.,
                      Tuomilehto, J.,
                      TusiÃ©-Luna, M.-T.,
                      Uitterlinden, A. G.,
                      Dam, R. M.,
                      Harst, P.,
                      Velde, N.,
                      Duijn, C. M.,
                      Schoor, N. M.,
                      Vitart, V.,
                      VÃ¶lker, U.,
                      Vollenweider, P.,
                      VÃ¶lzke, H.,
                      Wacher-Rodarte, N. H.,
                      Walker, M.,
                      Wang, Y. X.,
                      Wareham, N. J.,
                      Watanabe, R. M.,
                      Watkins, H.,
                      Weir, D. R.,
                      Werge, T. M.,
                      Widen, E.,
                      Wilkens, L. R.,
                      Willemsen, G.,
                      Willett, W. C.,
                      Wilson, J. F.,
                      Wong, T.-Y.,
                      Woo, J.-T.,
                      Wright, A. F.,
                      Wu, J.-Y.,
                      Xu, H.,
                      Yajnik, C. S.,
                      Yokota, M.,
                      Yuan, J.-M.,
                      Zeggini, E.,
                      Zemel, B. S.,
                      Zheng, W.,
                      Zhu, X.,
                      Zmuda, J. M.,
                      Zonderman, A. B.,
                      Zwart, J.-A.,
                      Chasman, D. I.,
                      Cho, Y. S.,
                      Heid, I. M.,
                      McCarthy, M. I.,
                      Ng, M. C. Y.,
                      Oâ€™Donnell, C. J.,
                      Rivadeneira, F.,
                      Thorsteinsdottir, U.,
                      Sun, Y. V.,
                      Tai, E. S.,
                      Boehnke, M.,
                      Deloukas, P.,
                      Justice, A. E.,
                      Lindgren, C. M.,
                      Loos, R. J. F.,
                      Mohlke, K. L.,
                      North, K. E.,
                      Stefansson, K.,
                      Walters, R. G.,
                      Winkler, T. W.,
                      Young, K. L.,
                      Loh, P.-R.,
                      Yang, J.,
                      Esko, T.,
                      Assimes, T. L.,
                      Auton, A.,
                      Abecasis, G. R.,
                      Willer, C. J.,
                      Locke, A. E.,
                      Berndt, S. I.,
                      Lettre, G.,
                      Frayling, T. M.,
                      Okada, Y.,
                      Wood, A. R.,
                      Visscher, P. M.,
                    </span>
                    and Hirschhorn, J. N.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>610</b>,
                    704â€“712
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cXqeN" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/36224396" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2022.01.07.475305" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://www.broadinstitute.org/news/largest-genome-wide-association-study-ever-uncovers-nearly-all-genetic-variants-linked-height" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-022-05275-y" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Common single-nucleotide polymorphisms (SNPs) are predicted to collectively explain 40â€“50% of phenotypic variation in human height, but identifying the specific variants and associated regions requires huge sample sizes1. Here, using data from a genome-wide association study of 5.4 million individuals of diverse ancestries, we show that 12,111 independent SNPs that are significantly associated with height account for nearly all of the common SNP-based heritability. These SNPs are clustered within 7,209 non-overlapping genomic segments with a mean size of around 90 kb, covering about 21% of the genome. The density of independent associations varies across the genome and the regions of increased density are enriched for biologically relevant genes. In out-of-sample estimation and prediction, the 12,111 SNPs (or all SNPs in the HapMap 3 panel2) account for 40% (45%) of phenotypic variance in populations of European ancestry but only around 10â€“20% (14â€“24%) in populations of other ancestries. Effect sizes, associated regions and gene prioritization are similar across ancestries, indicating that reduced prediction accuracy is likely to be explained by linkage disequilibrium and differences in allele frequency within associated regions. Finally, we show that the relevant biological pathways are detectable with smaller sample sizes than are needed to implicate causal genes and variants. Overall, this study provides a comprehensive map of specific genomic regions that contain the vast majority of common height-associated variants. Although this map is saturated for populations of European ancestry, further research is needed to achieve equivalent saturation in other ancestries. A large genome-wide association study of more than 5 million individuals reveals that 12,111 single-nucleotide polymorphisms account for nearly all the heritability of height attributable to common genetic variants.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-022-05275-y" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Wang_QS2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-022-32276-2" target="_blank" rel="noopener">The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Wang, Q. S.,
                    </span>
                    <span class="coauthor">
                      Edahiro, R.,
                    </span>
                    <span class="coauthor">
                      Namkoong, H.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Hasegawa, T.,
                      Shirai, Y.,
                      Sonehara, K.,
                      Tanaka, H.,
                      Lee, H.,
                      Saiki, R.,
                      Hyugaji, T.,
                      Shimizu, E.,
                      Katayama, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Naito, T.,
                      Sasa, N.,
                      Yamamoto, K.,
                      Kato, Y.,
                      Morita, T.,
                      Takahashi, K.,
                      Harada, N.,
                      Naito, T.,
                      Hiki, M.,
                      Matsushita, Y.,
                      Takagi, H.,
                      Ichikawa, M.,
                      Nakamura, A.,
                      Harada, S.,
                      Sandhu, Y.,
                      Kabata, H.,
                      Masaki, K.,
                      Kamata, H.,
                      Ikemura, S.,
                      Chubachi, S.,
                      Okamori, S.,
                      Terai, H.,
                      Morita, A.,
                      Asakura, T.,
                      Sasaki, J.,
                      Morisaki, H.,
                      Uwamino, Y.,
                      Nanki, K.,
                      Uchida, S.,
                      Uno, S.,
                      Nishimura, T.,
                      Ishiguro, T.,
                      Isono, T.,
                      Shibata, S.,
                      Matsui, Y.,
                      Hosoda, C.,
                      Takano, K.,
                      Nishida, T.,
                      Kobayashi, Y.,
                      Takaku, Y.,
                      Takayanagi, N.,
                      Ueda, S.,
                      Tada, A.,
                      Miyawaki, M.,
                      Yamamoto, M.,
                      Yoshida, E.,
                      Hayashi, R.,
                      Nagasaka, T.,
                      Arai, S.,
                      Kaneko, Y.,
                      Sasaki, K.,
                      Tagaya, E.,
                      Kawana, M.,
                      Arimura, K.,
                      Takahashi, K.,
                      Anzai, T.,
                      Ito, S.,
                      Endo, A.,
                      Uchimura, Y.,
                      Miyazaki, Y.,
                      Honda, T.,
                      Tateishi, T.,
                      Tohda, S.,
                      Ichimura, N.,
                      Sonobe, K.,
                      Sassa, C. T.,
                      Nakajima, J.,
                      Nakano, Y.,
                      Nakajima, Y.,
                      Anan, R.,
                      Arai, R.,
                      Kurihara, Y.,
                      Harada, Y.,
                      Nishio, K.,
                      Ueda, T.,
                      Azuma, M.,
                      Saito, R.,
                      Sado, T.,
                      Miyazaki, Y.,
                      Sato, R.,
                      Haruta, Y.,
                      Nagasaki, T.,
                      Yasui, Y.,
                      Hasegawa, Y.,
                      Mutoh, Y.,
                      Kimura, T.,
                      Sato, T.,
                      Takei, R.,
                      Hagimoto, S.,
                      Noguchi, Y.,
                      Yamano, Y.,
                      Sasano, H.,
                      Ota, S.,
                      Nakamori, Y.,
                      Yoshiya, K.,
                      Saito, F.,
                      Yoshihara, T.,
                      Wada, D.,
                      Iwamura, H.,
                      Kanayama, S.,
                      Maruyama, S.,
                      Yoshiyama, T.,
                      Ohta, K.,
                      Kokuto, H.,
                      Ogata, H.,
                      Tanaka, Y.,
                      Arakawa, K.,
                      Shimoda, M.,
                      Osawa, T.,
                      Tateno, H.,
                      Hase, I.,
                      Yoshida, S.,
                      Suzuki, S.,
                      Kawada, M.,
                      Horinouchi, H.,
                      Saito, F.,
                      Mitamura, K.,
                      Hagihara, M.,
                      Ochi, J.,
                      Uchida, T.,
                      Baba, R.,
                      Arai, D.,
                      Ogura, T.,
                      Takahashi, H.,
                      Hagiwara, S.,
                      Nagao, G.,
                      Konishi, S.,
                      Nakachi, I.,
                      Murakami, K.,
                      Yamada, M.,
                      Sugiura, H.,
                      Sano, H.,
                      Matsumoto, S.,
                      Kimura, N.,
                      Ono, Y.,
                      Baba, H.,
                      Suzuki, Y.,
                      Nakayama, S.,
                      Masuzawa, K.,
                      Namba, S.,
                      Shiroyama, T.,
                      Noda, Y.,
                      Niitsu, T.,
                      Adachi, Y.,
                      Enomoto, T.,
                      Amiya, S.,
                      Hara, R.,
                      Yamaguchi, Y.,
                      Murakami, T.,
                      Kuge, T.,
                      Matsumoto, K.,
                      Yamamoto, Y.,
                      Yamamoto, M.,
                      Yoneda, M.,
                      Tomono, K.,
                      Kato, K.,
                      Hirata, H.,
                      Takeda, Y.,
                      Koh, H.,
                      Manabe, T.,
                      Funatsu, Y.,
                      Ito, F.,
                      Fukui, T.,
                      Shinozuka, K.,
                      Kohashi, S.,
                      Miyazaki, M.,
                      Shoko, T.,
                      Kojima, M.,
                      Adachi, T.,
                      Ishikawa, M.,
                      Takahashi, K.,
                      Inoue, T.,
                      Hirano, T.,
                      Kobayashi, K.,
                      Takaoka, H.,
                      Watanabe, K.,
                      Miyazawa, N.,
                      Kimura, Y.,
                      Sado, R.,
                      Sugimoto, H.,
                      Kamiya, A.,
                      Kuwahara, N.,
                      Fujiwara, A.,
                      Matsunaga, T.,
                      Sato, Y.,
                      Okada, T.,
                      Hirai, Y.,
                      Kawashima, H.,
                      Narita, A.,
                      Niwa, K.,
                      Sekikawa, Y.,
                      Nishi, K.,
                      Nishitsuji, M.,
                      Tani, M.,
                      Suzuki, J.,
                      Nakatsumi, H.,
                      Ogura, T.,
                      Kitamura, H.,
                      Hagiwara, E.,
                      Murohashi, K.,
                      Okabayashi, H.,
                      Mochimaru, T.,
                      Nukaga, S.,
                      Satomi, R.,
                      Oyamada, Y.,
                      Mori, N.,
                      Baba, T.,
                      Fukui, Y.,
                      Odate, M.,
                      Mashimo, S.,
                      Makino, Y.,
                      Yagi, K.,
                      Hashiguchi, M.,
                      Kagyo, J.,
                      Shiomi, T.,
                      Fuke, S.,
                      Saito, H.,
                      Tsuchida, T.,
                      Fujitani, S.,
                      Takita, M.,
                      Morikawa, D.,
                      Yoshida, T.,
                      Izumo, T.,
                      Inomata, M.,
                      Kuse, N.,
                      Awano, N.,
                      Tone, M.,
                      Ito, A.,
                      Nakamura, Y.,
                      Hoshino, K.,
                      Maruyama, J.,
                      Ishikura, H.,
                      Takata, T.,
                      Odani, T.,
                      Amishima, M.,
                      Hattori, T.,
                      Shichinohe, Y.,
                      Kagaya, T.,
                      Kita, T.,
                      Ohta, K.,
                      Sakagami, S.,
                      Koshida, K.,
                      Hayashi, K.,
                      Shimizu, T.,
                      Kozu, Y.,
                      Hiranuma, H.,
                      Gon, Y.,
                      Izumi, N.,
                      Nagata, K.,
                      Ueda, K.,
                      Taki, R.,
                      Hanada, S.,
                      Kawamura, K.,
                      Ichikado, K.,
                      Nishiyama, K.,
                      Muranaka, H.,
                      Nakamura, K.,
                      Hashimoto, N.,
                      Wakahara, K.,
                      Koji, S.,
                      Omote, N.,
                      Ando, A.,
                      Kodama, N.,
                      Kaneyama, Y.,
                      Maeda, S.,
                      Kuraki, T.,
                      Matsumoto, T.,
                      Yokote, K.,
                      Nakada, T.-A.,
                      Abe, R.,
                      Oshima, T.,
                      Shimada, T.,
                      Harada, M.,
                      Takahashi, T.,
                      Ono, H.,
                      Sakurai, T.,
                      Shibusawa, T.,
                      Kimizuka, Y.,
                      Kawana, A.,
                      Sano, T.,
                      Watanabe, C.,
                      Suematsu, R.,
                      Sageshima, H.,
                      Yoshifuji, A.,
                      Ito, K.,
                      Takahashi, S.,
                      Ishioka, K.,
                      Nakamura, M.,
                      Masuda, M.,
                      Wakabayashi, A.,
                      Watanabe, H.,
                      Ueda, S.,
                      Nishikawa, M.,
                      Chihara, Y.,
                      Takeuchi, M.,
                      Onoi, K.,
                      Shinozuka, J.,
                      Sueyoshi, A.,
                      Nagasaki, Y.,
                      Okamoto, M.,
                      Ishihara, S.,
                      Shimo, M.,
                      Tokunaga, Y.,
                      Kusaka, Y.,
                      Ohba, T.,
                      Isogai, S.,
                      Ogawa, A.,
                      Inoue, T.,
                      Fukuyama, S.,
                      Eriguchi, Y.,
                      Yonekawa, A.,
                      Kan-o, K.,
                      Matsumoto, K.,
                      Kanaoka, K.,
                      Ihara, S.,
                      Komuta, K.,
                      Inoue, Y.,
                      Chiba, S.,
                      Yamagata, K.,
                      Hiramatsu, Y.,
                      Kai, H.,
                      Asano, K.,
                      Oguma, T.,
                      Ito, Y.,
                      Hashimoto, S.,
                      Yamasaki, M.,
                      Kasamatsu, Y.,
                      Komase, Y.,
                      Hida, N.,
                      Tsuburai, T.,
                      Oyama, B.,
                      Takada, M.,
                      Kanda, H.,
                      Kitagawa, Y.,
                      Fukuta, T.,
                      Miyake, T.,
                      Yoshida, S.,
                      Ogura, S.,
                      Abe, S.,
                      Kono, Y.,
                      Togashi, Y.,
                      Takoi, H.,
                      Kikuchi, R.,
                      Ogawa, S.,
                      Ogata, T.,
                      Ishihara, S.,
                      Kanehiro, A.,
                      Ozaki, S.,
                      Fuchimoto, Y.,
                      Wada, S.,
                      Fujimoto, N.,
                      Nishiyama, K.,
                      Terashima, M.,
                      Beppu, S.,
                      Yoshida, K.,
                      Narumoto, O.,
                      Nagai, H.,
                      Ooshima, N.,
                      Motegi, M.,
                      Umeda, A.,
                      Miyagawa, K.,
                      Shimada, H.,
                      Endo, M.,
                      Ohira, Y.,
                      Watanabe, M.,
                      Inoue, S.,
                      Igarashi, A.,
                      Sato, M.,
                      Sagara, H.,
                      Tanaka, A.,
                      Ohta, S.,
                      Kimura, T.,
                      Shibata, Y.,
                      Tanino, Y.,
                      Nikaido, T.,
                      Minemura, H.,
                      Sato, Y.,
                      Yamada, Y.,
                      Hashino, T.,
                      Shinoki, M.,
                      Iwagoe, H.,
                      Takahashi, H.,
                      Fujii, K.,
                      Kishi, H.,
                      Kanai, M.,
                      Imamura, T.,
                      Yamashita, T.,
                      Yatomi, M.,
                      Maeno, T.,
                      Hayashi, S.,
                      Takahashi, M.,
                      Kuramochi, M.,
                      Kamimaki, I.,
                      Tominaga, Y.,
                      Ishii, T.,
                      Utsugi, M.,
                      Ono, A.,
                      Tanaka, T.,
                      Kashiwada, T.,
                      Fujita, K.,
                      Saito, Y.,
                      Seike, M.,
                      Watanabe, H.,
                      Matsuse, H.,
                      Kodaka, N.,
                      Nakano, C.,
                      Oshio, T.,
                      Hirouchi, T.,
                      Makino, S.,
                      Egi, M.,
                      Omae, Y.,
                      Nannya, Y.,
                      Ueno, T.,
                      Takano, T.,
                      Katayama, K.,
                      Ai, M.,
                      Kumanogoh, A.,
                      Sato, T.,
                      Hasegawa, N.,
                      Tokunaga, K.,
                      Ishii, M.,
                      Koike, R.,
                      Kitagawa, Y.,
                      Kimura, A.,
                      Imoto, S.,
                      Miyano, S.,
                      Ogawa, S.,
                      Kanai, T.,
                      Fukunaga, K.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>13</b>,
                    4830
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cT68e" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35995775" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-022-32276-2" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Coronavirus disease 2019 (COVID-19) is a recently-emerged infectious disease that has caused millions of deaths, where comprehensive understanding of disease mechanisms is still unestablished. In particular, studies of gene expression dynamics and regulation landscape in COVID-19 infected individuals are limited. Here, we report on a thorough analysis of whole blood RNA-seq data from 465 genotyped samples from the Japan COVID-19 Task Force, including 359 severe and 106 non-severe COVID-19 cases. We discover 1169 putative causal expression quantitative trait loci (eQTLs) including 34 possible colocalizations with biobank fine-mapping results of hematopoietic traits in a Japanese population, 1549 putative causal splice QTLs (sQTLs; e.g. two independent sQTLs at TOR1AIP1), as well as biologically interpretable trans-eQTL examples (e.g., REST and STING1), all fine-mapped at single variant resolution. We perform differential gene expression analysis to elucidate 198 genes with increased expression in severe COVID-19 cases and enriched for innate immune-related functions. Finally, we evaluate the limited but non-zero effect of COVID-19 phenotype on eQTL discovery, and highlight the presence of COVID-19 severity-interaction eQTLs (ieQTLs; e.g., CLEC4C and MYBL2). Our study provides a comprehensive catalog of whole blood regulatory variants in Japanese, as well as a reference for transcriptional landscapes in response to COVID-19 infection. Genetic mechanisms influencing COVID-19 susceptibility are not well understood. Here, the authors analyzed whole blood RNA-seq data of 465 Japanese individuals with COVID-19, highlighting thousands of fine-mapped variants affecting expression and splicing of genes, as well as the presence of COVID-19 severity-interaction eQTLs.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-022-32276-2" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Ruotsalainen2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s42003-022-03552-0" target="_blank" rel="noopener">Inframe insertion and splice site variants in MFGE8 associate with protection against coronary atherosclerosis</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Ruotsalainen, S. E.,
                    </span>
                    <span class="coauthor">
                      Surakka, I.,
                    </span>
                    <span class="coauthor">
                      Mars, N.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karjalainen, J.,
                      Kurki, M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Krebs, K.,
                      Graham, S.,
                      Mishra, P. P.,
                      Mishra, B. H.,
                      Sinisalo, J.,
                      Palta, P.,
                      LehtimÃ¤ki, T.,
                      Raitakari, O.,
                      Milani, L.,
                      Okada, Y.,
                      Palotie, A.,
                      Widen, E.,
                      Daly, M. J.,
                    </span>
                    and Ripatti, S.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Communications Biology</em>
                    <b>5</b>,
                    802
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cTPJW" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35978133" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.06.23.21259381" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.helsinki.fi/en/news/genes/new-gene-variant-protects-against-coronary-heart-disease-uncovered" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s42003-022-03552-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Cardiovascular diseases are the leading cause of premature death and disability worldwide, with both genetic and environmental determinants. While genome-wide association studies have identified multiple genetic loci associated with cardiovascular diseases, exact genes driving these associations remain mostly uncovered. Due to Finlandâ€™s population history, many deleterious and high-impact variants are enriched in the Finnish population giving a possibility to find genetic associations for protein-truncating variants that likely tie the association to a gene and that would not be detected elsewhere. In a large Finnish biobank study FinnGen, we identified an association between an inframe insertion rs534125149 in MFGE8 (encoding lactadherin) and protection against coronary atherosclerosis. This variant is highly enriched in Finland, and the protective association was replicated in meta-analysis of BioBank Japan and Estonian biobank. Additionally, we identified a protective association between splice acceptor variant rs201988637 in MFGE8 and coronary atherosclerosis, independent of the rs534125149, with no significant risk-increasing associations. This variant was also associated with lower pulse pressure, pointing towards a function of MFGE8 in arterial aging also in humans in addition to previous evidence in mice. In conclusion, our results suggest that inhibiting the production of lactadherin could lower the risk for coronary heart disease substantially. A genome-wide association study identifies MFGE8 as protective against coronary atherosclerosis in European and East Asian populations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s42003-022-03552-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Namkoong2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-022-05163-5" target="_blank" rel="noopener">DOCK2 is involved in the host genetics and biology of severe COVID-19</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Namkoong, H.,
                    </span>
                    <span class="coauthor">
                      Edahiro, R.,
                    </span>
                    <span class="coauthor">
                      Takano, T.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Nishihara, H.,
                      Shirai, Y.,
                      Sonehara, K.,
                      Tanaka, H.,
                      Azekawa, S.,
                      Mikami, Y.,
                      Lee, H.,
                      Hasegawa, T.,
                      Okudela, K.,
                      Okuzaki, D.,
                      Motooka, D.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Naito, T.,
                      Yamamoto, K.,
                      Wang, Q. S.,
                      Saiki, R.,
                      Ishihara, R.,
                      Matsubara, Y.,
                      Hamamoto, J.,
                      Hayashi, H.,
                      Yoshimura, Y.,
                      Tachikawa, N.,
                      Yanagita, E.,
                      Hyugaji, T.,
                      Shimizu, E.,
                      Katayama, K.,
                      Kato, Y.,
                      Morita, T.,
                      Takahashi, K.,
                      Harada, N.,
                      Naito, T.,
                      Hiki, M.,
                      Matsushita, Y.,
                      Takagi, H.,
                      Aoki, R.,
                      Nakamura, A.,
                      Harada, S.,
                      Sasano, H.,
                      Kabata, H.,
                      Masaki, K.,
                      Kamata, H.,
                      Ikemura, S.,
                      Chubachi, S.,
                      Okamori, S.,
                      Terai, H.,
                      Morita, A.,
                      Asakura, T.,
                      Sasaki, J.,
                      Morisaki, H.,
                      Uwamino, Y.,
                      Nanki, K.,
                      Uchida, S.,
                      Uno, S.,
                      Nishimura, T.,
                      Ishiguro, T.,
                      Isono, T.,
                      Shibata, S.,
                      Matsui, Y.,
                      Hosoda, C.,
                      Takano, K.,
                      Nishida, T.,
                      Kobayashi, Y.,
                      Takaku, Y.,
                      Takayanagi, N.,
                      Ueda, S.,
                      Tada, A.,
                      Miyawaki, M.,
                      Yamamoto, M.,
                      Yoshida, E.,
                      Hayashi, R.,
                      Nagasaka, T.,
                      Arai, S.,
                      Kaneko, Y.,
                      Sasaki, K.,
                      Tagaya, E.,
                      Kawana, M.,
                      Arimura, K.,
                      Takahashi, K.,
                      Anzai, T.,
                      Ito, S.,
                      Endo, A.,
                      Uchimura, Y.,
                      Miyazaki, Y.,
                      Honda, T.,
                      Tateishi, T.,
                      Tohda, S.,
                      Ichimura, N.,
                      Sonobe, K.,
                      Sassa, C. T.,
                      Nakajima, J.,
                      Nakano, Y.,
                      Nakajima, Y.,
                      Anan, R.,
                      Arai, R.,
                      Kurihara, Y.,
                      Harada, Y.,
                      Nishio, K.,
                      Ueda, T.,
                      Azuma, M.,
                      Saito, R.,
                      Sado, T.,
                      Miyazaki, Y.,
                      Sato, R.,
                      Haruta, Y.,
                      Nagasaki, T.,
                      Yasui, Y.,
                      Hasegawa, Y.,
                      Mutoh, Y.,
                      Kimura, T.,
                      Sato, T.,
                      Takei, R.,
                      Hagimoto, S.,
                      Noguchi, Y.,
                      Yamano, Y.,
                      Sasano, H.,
                      Ota, S.,
                      Nakamori, Y.,
                      Yoshiya, K.,
                      Saito, F.,
                      Yoshihara, T.,
                      Wada, D.,
                      Iwamura, H.,
                      Kanayama, S.,
                      Maruyama, S.,
                      Yoshiyama, T.,
                      Ohta, K.,
                      Kokuto, H.,
                      Ogata, H.,
                      Tanaka, Y.,
                      Arakawa, K.,
                      Shimoda, M.,
                      Osawa, T.,
                      Tateno, H.,
                      Hase, I.,
                      Yoshida, S.,
                      Suzuki, S.,
                      Kawada, M.,
                      Horinouchi, H.,
                      Saito, F.,
                      Mitamura, K.,
                      Hagihara, M.,
                      Ochi, J.,
                      Uchida, T.,
                      Baba, R.,
                      Arai, D.,
                      Ogura, T.,
                      Takahashi, H.,
                      Hagiwara, S.,
                      Nagao, G.,
                      Konishi, S.,
                      Nakachi, I.,
                      Murakami, K.,
                      Yamada, M.,
                      Sugiura, H.,
                      Sano, H.,
                      Matsumoto, S.,
                      Kimura, N.,
                      Ono, Y.,
                      Baba, H.,
                      Suzuki, Y.,
                      Nakayama, S.,
                      Masuzawa, K.,
                      Namba, S.,
                      Suzuki, K.,
                      Naito, Y.,
                      Liu, Y.-C.,
                      Takuwa, A.,
                      Sugihara, F.,
                      Wing, J. B.,
                      Sakakibara, S.,
                      Hizawa, N.,
                      Shiroyama, T.,
                      Miyawaki, S.,
                      Kawamura, Y.,
                      Nakayama, A.,
                      Matsuo, H.,
                      Maeda, Y.,
                      Nii, T.,
                      Noda, Y.,
                      Niitsu, T.,
                      Adachi, Y.,
                      Enomoto, T.,
                      Amiya, S.,
                      Hara, R.,
                      Yamaguchi, Y.,
                      Murakami, T.,
                      Kuge, T.,
                      Matsumoto, K.,
                      Yamamoto, Y.,
                      Yamamoto, M.,
                      Yoneda, M.,
                      Kishikawa, T.,
                      Yamada, S.,
                      Kawabata, S.,
                      Kijima, N.,
                      Takagaki, M.,
                      Sasa, N.,
                      Ueno, Y.,
                      Suzuki, M.,
                      Takemoto, N.,
                      Eguchi, H.,
                      Fukusumi, T.,
                      Imai, T.,
                      Fukushima, M.,
                      Kishima, H.,
                      Inohara, H.,
                      Tomono, K.,
                      Kato, K.,
                      Takahashi, M.,
                      Matsuda, F.,
                      Hirata, H.,
                      Takeda, Y.,
                      Koh, H.,
                      Manabe, T.,
                      Funatsu, Y.,
                      Ito, F.,
                      Fukui, T.,
                      Shinozuka, K.,
                      Kohashi, S.,
                      Miyazaki, M.,
                      Shoko, T.,
                      Kojima, M.,
                      Adachi, T.,
                      Ishikawa, M.,
                      Takahashi, K.,
                      Inoue, T.,
                      Hirano, T.,
                      Kobayashi, K.,
                      Takaoka, H.,
                      Watanabe, K.,
                      Miyazawa, N.,
                      Kimura, Y.,
                      Sado, R.,
                      Sugimoto, H.,
                      Kamiya, A.,
                      Kuwahara, N.,
                      Fujiwara, A.,
                      Matsunaga, T.,
                      Sato, Y.,
                      Okada, T.,
                      Hirai, Y.,
                      Kawashima, H.,
                      Narita, A.,
                      Niwa, K.,
                      Sekikawa, Y.,
                      Nishi, K.,
                      Nishitsuji, M.,
                      Tani, M.,
                      Suzuki, J.,
                      Nakatsumi, H.,
                      Ogura, T.,
                      Kitamura, H.,
                      Hagiwara, E.,
                      Murohashi, K.,
                      Okabayashi, H.,
                      Mochimaru, T.,
                      Nukaga, S.,
                      Satomi, R.,
                      Oyamada, Y.,
                      Mori, N.,
                      Baba, T.,
                      Fukui, Y.,
                      Odate, M.,
                      Mashimo, S.,
                      Makino, Y.,
                      Yagi, K.,
                      Hashiguchi, M.,
                      Kagyo, J.,
                      Shiomi, T.,
                      Fuke, S.,
                      Saito, H.,
                      Tsuchida, T.,
                      Fujitani, S.,
                      Takita, M.,
                      Morikawa, D.,
                      Yoshida, T.,
                      Izumo, T.,
                      Inomata, M.,
                      Kuse, N.,
                      Awano, N.,
                      Tone, M.,
                      Ito, A.,
                      Nakamura, Y.,
                      Hoshino, K.,
                      Maruyama, J.,
                      Ishikura, H.,
                      Takata, T.,
                      Odani, T.,
                      Amishima, M.,
                      Hattori, T.,
                      Shichinohe, Y.,
                      Kagaya, T.,
                      Kita, T.,
                      Ohta, K.,
                      Sakagami, S.,
                      Koshida, K.,
                      Hayashi, K.,
                      Shimizu, T.,
                      Kozu, Y.,
                      Hiranuma, H.,
                      Gon, Y.,
                      Izumi, N.,
                      Nagata, K.,
                      Ueda, K.,
                      Taki, R.,
                      Hanada, S.,
                      Kawamura, K.,
                      Ichikado, K.,
                      Nishiyama, K.,
                      Muranaka, H.,
                      Nakamura, K.,
                      Hashimoto, N.,
                      Wakahara, K.,
                      Koji, S.,
                      Omote, N.,
                      Ando, A.,
                      Kodama, N.,
                      Kaneyama, Y.,
                      Maeda, S.,
                      Kuraki, T.,
                      Matsumoto, T.,
                      Yokote, K.,
                      Nakada, T.-A.,
                      Abe, R.,
                      Oshima, T.,
                      Shimada, T.,
                      Harada, M.,
                      Takahashi, T.,
                      Ono, H.,
                      Sakurai, T.,
                      Shibusawa, T.,
                      Kimizuka, Y.,
                      Kawana, A.,
                      Sano, T.,
                      Watanabe, C.,
                      Suematsu, R.,
                      Sageshima, H.,
                      Yoshifuji, A.,
                      Ito, K.,
                      Takahashi, S.,
                      Ishioka, K.,
                      Nakamura, M.,
                      Masuda, M.,
                      Wakabayashi, A.,
                      Watanabe, H.,
                      Ueda, S.,
                      Nishikawa, M.,
                      Chihara, Y.,
                      Takeuchi, M.,
                      Onoi, K.,
                      Shinozuka, J.,
                      Sueyoshi, A.,
                      Nagasaki, Y.,
                      Okamoto, M.,
                      Ishihara, S.,
                      Shimo, M.,
                      Tokunaga, Y.,
                      Kusaka, Y.,
                      Ohba, T.,
                      Isogai, S.,
                      Ogawa, A.,
                      Inoue, T.,
                      Fukuyama, S.,
                      Eriguchi, Y.,
                      Yonekawa, A.,
                      Kan-o, K.,
                      Matsumoto, K.,
                      Kanaoka, K.,
                      Ihara, S.,
                      Komuta, K.,
                      Inoue, Y.,
                      Chiba, S.,
                      Yamagata, K.,
                      Hiramatsu, Y.,
                      Kai, H.,
                      Asano, K.,
                      Oguma, T.,
                      Ito, Y.,
                      Hashimoto, S.,
                      Yamasaki, M.,
                      Kasamatsu, Y.,
                      Komase, Y.,
                      Hida, N.,
                      Tsuburai, T.,
                      Oyama, B.,
                      Takada, M.,
                      Kanda, H.,
                      Kitagawa, Y.,
                      Fukuta, T.,
                      Miyake, T.,
                      Yoshida, S.,
                      Ogura, S.,
                      Abe, S.,
                      Kono, Y.,
                      Togashi, Y.,
                      Takoi, H.,
                      Kikuchi, R.,
                      Ogawa, S.,
                      Ogata, T.,
                      Ishihara, S.,
                      Kanehiro, A.,
                      Ozaki, S.,
                      Fuchimoto, Y.,
                      Wada, S.,
                      Fujimoto, N.,
                      Nishiyama, K.,
                      Terashima, M.,
                      Beppu, S.,
                      Yoshida, K.,
                      Narumoto, O.,
                      Nagai, H.,
                      Ooshima, N.,
                      Motegi, M.,
                      Umeda, A.,
                      Miyagawa, K.,
                      Shimada, H.,
                      Endo, M.,
                      Ohira, Y.,
                      Watanabe, M.,
                      Inoue, S.,
                      Igarashi, A.,
                      Sato, M.,
                      Sagara, H.,
                      Tanaka, A.,
                      Ohta, S.,
                      Kimura, T.,
                      Shibata, Y.,
                      Tanino, Y.,
                      Nikaido, T.,
                      Minemura, H.,
                      Sato, Y.,
                      Yamada, Y.,
                      Hashino, T.,
                      Shinoki, M.,
                      Iwagoe, H.,
                      Takahashi, H.,
                      Fujii, K.,
                      Kishi, H.,
                      Kanai, M.,
                      Imamura, T.,
                      Yamashita, T.,
                      Yatomi, M.,
                      Maeno, T.,
                      Hayashi, S.,
                      Takahashi, M.,
                      Kuramochi, M.,
                      Kamimaki, I.,
                      Tominaga, Y.,
                      Ishii, T.,
                      Utsugi, M.,
                      Ono, A.,
                      Tanaka, T.,
                      Kashiwada, T.,
                      Fujita, K.,
                      Saito, Y.,
                      Seike, M.,
                      Watanabe, H.,
                      Matsuse, H.,
                      Kodaka, N.,
                      Nakano, C.,
                      Oshio, T.,
                      Hirouchi, T.,
                      Makino, S.,
                      Egi, M.,
                      Omae, Y.,
                      Nannya, Y.,
                      Ueno, T.,
                      Katayama, K.,
                      Ai, M.,
                      Fukui, Y.,
                      Kumanogoh, A.,
                      Sato, T.,
                      Hasegawa, N.,
                      Tokunaga, K.,
                      Ishii, M.,
                      Koike, R.,
                      Kitagawa, Y.,
                      Kimura, A.,
                      Imoto, S.,
                      Miyano, S.,
                      Ogawa, S.,
                      Kanai, T.,
                      Fukunaga, K.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>609</b>,
                    754â€“760
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cTfEy" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35940203" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.05.17.21256513" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-022-05163-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Identifying the factors underlying severe COVID-19 in the host genetics is an emerging issue. We conducted a genome-wide association study (GWAS) involving 2,393 Japanese COVID-19 cases collected in initial pandemic waves with 3,289 controls, which identified a variant on 5q35 (rs60200309-A) near DOCK2 associated with severe COVID-19 in younger (&lt;65 ages) patients (nCase=440, odds ratio=2.01, P=1.2\times10-8). This risk allele was prevalent in East Asians but rare in Europeans, showing a value of non-European GWAS. RNA-seq of 473 bulk peripheral blood identified decreasing effect of the risk allele on DOCK2 expression in younger patients. DOCK2 expression was suppressed in severe forms of COVID-19. Single cell RNA-seq analysis (n=61) identified cell type-specific downregulation of DOCK2 and COVID-19-specific decreasing effects of the risk allele on DOCK2 in non-classical monocytes. Immunohistochemistry of lung specimens from severe COVID-19 pneumonia showed suppressed DOCK2. Moreover, inhibition of DOCK2 function using CPYPP induced much more severe pneumonia in a Syrian hamster model of SARS-CoV-2 infection characterized as weight loss, lung edema, enhanced viral loads, impaired macrophage recruitment and dysregulated type I interferon responses. We conclude that DOCK2 plays an important role in the host immune response to SARS-CoV-2 infection and development of severe COVID-19, and could be further explored as a potential biomarker and/or therapeutic target.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-022-05163-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Ramdas2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.ajhg.2022.06.012" target="_blank" rel="noopener">A multi-layer functional genomic analysis to understand noncoding genetic variation in lipids</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Ramdas, S.,
                    </span>
                    <span class="coauthor">
                      Judd, J.,
                    </span>
                    <span class="coauthor">
                      Graham, S. E.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kanoni, S.,
                      Wang, Y.,
                      Surakka, I.,
                      Wenz, B.,
                      Clarke, S. L.,
                      Chesi, A.,
                      Wells, A.,
                      Bhatti, K. F.,
                      Vedantam, S.,
                      Winkler, T. W.,
                      Locke, A. E.,
                      Marouli, E.,
                      Zajac, G. J. M.,
                      Wu, K.-H. H.,
                      Ntalla, I.,
                      Hui, Q.,
                      Klarin, D.,
                      Hilliard, A. T.,
                      Wang, Z.,
                      Xue, C.,
                      Thorleifsson, G.,
                      Helgadottir, A.,
                      Gudbjartsson, D. F.,
                      Holm, H.,
                      Olafsson, I.,
                      Hwang, M. Y.,
                      Han, S.,
                      Akiyama, M.,
                      Sakaue, S.,
                      Terao, C.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, W.,
                      Brumpton, B. M.,
                      Rasheed, H.,
                      Havulinna, A. S.,
                      Veturi, Y.,
                      Pacheco, J. A.,
                      Rosenthal, E. A.,
                      Lingren, T.,
                      Feng, Q.,
                      Kullo, I. J.,
                      Narita, A.,
                      Takayama, J.,
                      Martin, H. C.,
                      Hunt, K. A.,
                      Trivedi, B.,
                      Haessler, J.,
                      Giulianini, F.,
                      Bradford, Y.,
                      Miller, J. E.,
                      Campbell, A.,
                      Lin, K.,
                      Millwood, I. Y.,
                      Rasheed, A.,
                      Hindy, G.,
                      Faul, J. D.,
                      Zhao, W.,
                      Weir, D. R.,
                      Turman, C.,
                      Huang, H.,
                      Graff, M.,
                      Choudhury, A.,
                      Sengupta, D.,
                      Mahajan, A.,
                      Brown, M. R.,
                      Zhang, W.,
                      Yu, K.,
                      Schmidt, E. M.,
                      Pandit, A.,
                      Gustafsson, S.,
                      Yin, X.,
                      Luan, J.,
                      Zhao, J.-H.,
                      Matsuda, F.,
                      Jang, H.-M.,
                      Yoon, K.,
                      Medina-Gomez, C.,
                      Pitsillides, A.,
                      Hottenga, J. J.,
                      Wood, A. R.,
                      Ji, Y.,
                      Gao, Z.,
                      Haworth, S.,
                      Mitchell, R. E.,
                      Chai, J. F.,
                      Aadahl, M.,
                      Bjerregaard, A. A.,
                      Yao, J.,
                      Manichaikul, A.,
                      Lee, W.-J.,
                      Hsiung, C. A.,
                      Warren, H. R.,
                      Ramirez, J.,
                      Bork-Jensen, J.,
                      KÃ¥rhus, L. L.,
                      Goel, A.,
                      Sabater-Lleal, M.,
                      Noordam, R.,
                      Mauro, P.,
                      Matteo, F.,
                      McDaid, A. F.,
                      Marques-Vidal, P.,
                      Wielscher, M.,
                      Trompet, S.,
                      Sattar, N.,
                      MÃ¸llehave, L. T.,
                      Munz, M.,
                      Zeng, L.,
                      Huang, J.,
                      Yang, B.,
                      Poveda, A.,
                      Kurbasic, A.,
                      SchÃ¶nherr, S.,
                      Forer, L.,
                      Scholz, M.,
                      Galesloot, T. E.,
                      Bradfield, J. P.,
                      Ruotsalainen, S. E.,
                      Daw, E. W.,
                      Zmuda, J. M.,
                      Mitchell, J. S.,
                      Fuchsberger, C.,
                      Christensen, H.,
                      Brody, J. A.,
                      Le, P.,
                      Feitosa, M. F.,
                      Wojczynski, M. K.,
                      Hemerich, D.,
                      Preuss, M.,
                      Mangino, M.,
                      Christofidou, P.,
                      Verweij, N.,
                      Benjamins, J. W.,
                      Engmann, J.,
                      Noah, T. L.,
                      Verma, A.,
                      Slieker, R. C.,
                      Lo, K. S.,
                      Zilhao, N. R.,
                      Kleber, M. E.,
                      Delgado, G. E.,
                      Huo, S.,
                      Ikeda, D. D.,
                      Iha, H.,
                      Yang, J.,
                      Liu, J.,
                      Demirkan, A.,
                      Leonard, H. L.,
                      Marten, J.,
                      Emmel, C.,
                      Schmidt, B.,
                      Smyth, L. J.,
                      CaÃ±adas-Garre, M.,
                      Wang, C.,
                      Nakatochi, M.,
                      Wong, A.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Sim, X.,
                      Xia, R.,
                      Huerta-Chagoya, A.,
                      Fernandez-Lopez, J. C.,
                      Lyssenko, V.,
                      Nongmaithem, S. S.,
                      Sankareswaran, A.,
                      Irvin, M. R.,
                      Oldmeadow, C.,
                      Kim, H.-N.,
                      Ryu, S.,
                      Timmers, P. R. H. J.,
                      Arbeeva, L.,
                      Dorajoo, R.,
                      Lange, L. A.,
                      Prasad, G.,
                      LorÃ©s-Motta, L.,
                      Pauper, M.,
                      Long, J.,
                      Li, X.,
                      Theusch, E.,
                      Takeuchi, F.,
                      Spracklen, C. N.,
                      Loukola, A.,
                      Bollepalli, S.,
                      Warner, S. C.,
                      Wang, Y. X.,
                      Wei, W. B.,
                      Nutile, T.,
                      Ruggiero, D.,
                      Sung, Y. J.,
                      Chen, S.,
                      Liu, F.,
                      Yang, J.,
                      Kentistou, K. A.,
                      Banas, B.,
                      Morgan, A.,
                      Meidtner, K.,
                      Bielak, L. F.,
                      Smith, J. A.,
                      Hebbar, P.,
                      Farmaki, A.-E.,
                      Hofer, E.,
                      Lin, M.,
                      Concas, M. P.,
                      Vaccargiu, S.,
                      Most, P. J.,
                      PitkÃ¤nen, N.,
                      Cade, B. E.,
                      Laan, S. W.,
                      Chitrala, K. N.,
                      Weiss, S.,
                      Bentley, A. R.,
                      Doumatey, A. P.,
                      Adeyemo, A. A.,
                      Lee, J. Y.,
                      Petersen, E. R. B.,
                      Nielsen, A. A.,
                      Choi, H. S.,
                      Nethander, M.,
                      Freitag-Wolf, S.,
                      Southam, L.,
                      Rayner, N. W.,
                      Wang, C. A.,
                      Lin, S.-Y.,
                      Wang, J.-S.,
                      Couture, C.,
                      LyytikÃ¤inen, L.-P.,
                      Nikus, K.,
                      Cuellar-Partida, G.,
                      Vestergaard, H.,
                      Hidalgo, B.,
                      Giannakopoulou, O.,
                      Cai, Q.,
                      Obura, M. O.,
                      Setten, J.,
                      He, K. Y.,
                      Tang, H.,
                      Terzikhan, N.,
                      Shin, J. H.,
                      Jackson, R. D.,
                      Reiner, A. P.,
                      Martin, L. W.,
                      Chen, Z.,
                      Li, L.,
                      Kawaguchi, T.,
                      Thiery, J.,
                      Bis, J. C.,
                      Launer, L. J.,
                      Li, H.,
                      Nalls, M. A.,
                      Raitakari, O. T.,
                      Ichihara, S.,
                      Wild, S. H.,
                      Nelson, C. P.,
                      Campbell, H.,
                      JÃ¤ger, S.,
                      Nabika, T.,
                      Al-Mulla, F.,
                      Niinikoski, H.,
                      Braund, P. S.,
                      Kolcic, I.,
                      Kovacs, P.,
                      Giardoglou, T.,
                      Katsuya, T.,
                      Kleijn, D.,
                      Borst, G. J.,
                      Kim, E. K.,
                      Adams, H. H. H.,
                      Ikram, M. A.,
                      Zhu, X.,
                      Asselbergs, F. W.,
                      Kraaijeveld, A. O.,
                      Beulens, J. W. J.,
                      Shu, X.-O.,
                      Rallidis, L. S.,
                      Pedersen, O.,
                      Hansen, T.,
                      Mitchell, P.,
                      Hewitt, A. W.,
                      KÃ¤hÃ¶nen, M.,
                      PÃ©russe, L.,
                      Bouchard, C.,
                      TÃ¶njes, A.,
                      Ida Chen, Y.-D.,
                      Pennell, C. E.,
                      Mori, T. A.,
                      Lieb, W.,
                      Franke, A.,
                      Ohlsson, C.,
                      MellstrÃ¶m, D.,
                      Cho, Y. S.,
                      Lee, H.,
                      Yuan, J.-M.,
                      Koh, W.-P.,
                      Rhee, S. Y.,
                      Woo, J.-T.,
                      Heid, I. M.,
                      Stark, K. J.,
                      Zimmermann, M. E.,
                      VÃ¶lzke, H.,
                      Homuth, G.,
                      Evans, M. K.,
                      Zonderman, A. B.,
                      Polasek, O.,
                      Pasterkamp, G.,
                      Hoefer, I. E.,
                      Redline, S.,
                      Pahkala, K.,
                      Oldehinkel, A. J.,
                      Snieder, H.,
                      Biino, G.,
                      Schmidt, R.,
                      Schmidt, H.,
                      Bandinelli, S.,
                      Dedoussis, G.,
                      Thanaraj, T. A.,
                      Peyser, P. A.,
                      Kato, N.,
                      Schulze, M. B.,
                      Girotto, G.,
                      BÃ¶ger, C. A.,
                      Jung, B.,
                      Joshi, P. K.,
                      Bennett, D. A.,
                      De Jager, P. L.,
                      Lu, X.,
                      Mamakou, V.,
                      Brown, M.,
                      Caulfield, M. J.,
                      Munroe, P. B.,
                      Guo, X.,
                      Ciullo, M.,
                      Jonas, J. B.,
                      Samani, N. J.,
                      Kaprio, J.,
                      Pajukanta, P.,
                      TusiÃ©-Luna, T.,
                      Aguilar-Salinas, C. A.,
                      Adair, L. S.,
                      Bechayda, S. A.,
                      Silva, H. J.,
                      Wickremasinghe, A. R.,
                      Krauss, R. M.,
                      Wu, J.-Y.,
                      Zheng, W.,
                      Hollander, A. I.,
                      Bharadwaj, D.,
                      Correa, A.,
                      Wilson, J. G.,
                      Lind, L.,
                      Heng, C.-K.,
                      Nelson, A. E.,
                      Golightly, Y. M.,
                      Wilson, J. F.,
                      Penninx, B.,
                      Kim, H.-L.,
                      Attia, J.,
                      Scott, R. J.,
                      Rao, D. C.,
                      Arnett, D. K.,
                      Walker, M.,
                      Scott, L. J.,
                      Koistinen, H. A.,
                      Chandak, G. R.,
                      Mercader, J. M.,
                      Villalpando, C. G.,
                      Orozco, L.,
                      Fornage, M.,
                      Tai, E. S.,
                      Dam, R. M.,
                      LehtimÃ¤ki, T.,
                      Chaturvedi, N.,
                      Yokota, M.,
                      Liu, J.,
                      Reilly, D. F.,
                      McKnight, A. J.,
                      Kee, F.,
                      JÃ¶ckel, K.-H.,
                      McCarthy, M. I.,
                      Palmer, C. N. A.,
                      Vitart, V.,
                      Hayward, C.,
                      Simonsick, E.,
                      Duijn, C. M.,
                      Jin, Z.-B.,
                      Lu, F.,
                      Hishigaki, H.,
                      Lin, X.,
                      MÃ¤rz, W.,
                      Gudnason, V.,
                      Tardif, J.-C.,
                      Lettre, G.,
                      T Hart, L. M.,
                      Elders, P. J. M.,
                      Rader, D. J.,
                      Damrauer, S. M.,
                      Kumari, M.,
                      Kivimaki, M.,
                      Harst, P.,
                      Spector, T. D.,
                      Loos, R. J. F.,
                      Province, M. A.,
                      Parra, E. J.,
                      Cruz, M.,
                      Psaty, B. M.,
                      Brandslund, I.,
                      Pramstaller, P. P.,
                      Rotimi, C. N.,
                      Christensen, K.,
                      Ripatti, S.,
                      WidÃ©n, E.,
                      Hakonarson, H.,
                      Grant, S. F. A.,
                      Kiemeney, L.,
                      Graaf, J.,
                      Loeffler, M.,
                      Kronenberg, F.,
                      Gu, D.,
                      Erdmann, J.,
                      Schunkert, H.,
                      Franks, P. W.,
                      Linneberg, A.,
                      Jukema, J. W.,
                      Khera, A. V.,
                      MÃ¤nnikkÃ¶, M.,
                      Jarvelin, M.-R.,
                      Kutalik, Z.,
                      Francesco, C.,
                      Mook-Kanamori, D. O.,
                      Dijk, K.,
                      Watkins, H.,
                      Strachan, D. P.,
                      Grarup, N.,
                      Sever, P.,
                      Poulter, N.,
                      Huey-Herng Sheu, W.,
                      Rotter, J. I.,
                      Dantoft, T. M.,
                      Karpe, F.,
                      Neville, M. J.,
                      Timpson, N. J.,
                      Cheng, C.-Y.,
                      Wong, T.-Y.,
                      Khor, C. C.,
                      Li, H.,
                      Sabanayagam, C.,
                      Peters, A.,
                      Gieger, C.,
                      Hattersley, A. T.,
                      Pedersen, N. L.,
                      Magnusson, P. K. E.,
                      Boomsma, D. I.,
                      Geus, E. J. C.,
                      Cupples, L. A.,
                      Meurs, J. B. J.,
                      Ikram, A.,
                      Ghanbari, M.,
                      Gordon-Larsen, P.,
                      Huang, W.,
                      Kim, Y. J.,
                      Tabara, Y.,
                      Wareham, N. J.,
                      Langenberg, C.,
                      Zeggini, E.,
                      Tuomilehto, J.,
                      Kuusisto, J.,
                      Laakso, M.,
                      Ingelsson, E.,
                      Abecasis, G.,
                      Chambers, J. C.,
                      Kooner, J. S.,
                      Vries, P. S.,
                      Morrison, A. C.,
                      Hazelhurst, S.,
                      Ramsay, M.,
                      North, K. E.,
                      Daviglus, M.,
                      Kraft, P.,
                      Martin, N. G.,
                      Whitfield, J. B.,
                      Abbas, S.,
                      Saleheen, D.,
                      Walters, R. G.,
                      Holmes, M. V.,
                      Black, C.,
                      Smith, B. H.,
                      Baras, A.,
                      Justice, A. E.,
                      Buring, J. E.,
                      Ridker, P. M.,
                      Chasman, D. I.,
                      Kooperberg, C.,
                      Tamiya, G.,
                      Yamamoto, M.,
                      Heel, D. A.,
                      Trembath, R. C.,
                      Wei, W.-Q.,
                      Jarvik, G. P.,
                      Namjou, B.,
                      Hayes, M. G.,
                      Ritchie, M. D.,
                      Jousilahti, P.,
                      Salomaa, V.,
                      Hveem, K.,
                      Ã…svold, B. O.,
                      Kubo, M.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Murakami, Y.,
                      Kim, B.-J.,
                      Thorsteinsdottir, U.,
                      Stefansson, K.,
                      Zhang, J.,
                      Chen, Y. E.,
                      Ho, Y.-L.,
                      Lynch, J. A.,
                      Tsao, P. S.,
                      Chang, K.-M.,
                      Cho, K.,
                      Oâ€™Donnell, C. J.,
                      Gaziano, J. M.,
                      Wilson, P.,
                      Mohlke, K. L.,
                      Frayling, T. M.,
                      Hirschhorn, J. N.,
                      Kathiresan, S.,
                      Boehnke, M.,
                      Million Veterans Program.,
                      Global Lipids Genetics Consortium.,
                      Grant, S.,
                      Natarajan, P.,
                      Sun, Y. V.,
                      Morris, A. P.,
                      Deloukas, P.,
                      Peloso, G.,
                      Assimes, T. L.,
                      Willer, C. J.,
                      Zhu, X.,
                    </span>
                    and Brown, C. D.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>The American Journal of Human Genetics</em>
                    <b>109</b>,
                    1366â€“1387
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35931049" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.12.07.470215" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.ajhg.2022.06.012" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>A major challenge of genome-wide association studies (GWASs) is to translate phenotypic associations into biological insights. Here, we integrate a large GWAS on blood lipids involving 1.6 million individuals from five ancestries with a wide array of functional genomic datasets to discover regulatory mechanisms underlying lipid associations. We first prioritize lipid-associated genes with expression quantitative trait locus (eQTL) colocalizations and then add chromatin interaction data to narrow the search for functional genes. Polygenic enrichment analysis across 697 annotations from a host of tissues and cell types confirms the central role of the liver in lipid levels and highlights the selective enrichment of adipose-specific chromatin marks in high-density lipoprotein cholesterol and triglycerides. Overlapping transcription factor (TF) binding sites with lipid-associated loci identifies TFs relevant in lipid biology. In addition, we present an integrative framework to prioritize causal variants at GWAS loci, producing a comprehensive list of candidate causal genes and variants with multiple layers of functional evidence. We highlight two of the prioritized genes, CREBRF and RRBP1, which show convergent evidence across functional datasets supporting their roles in lipid biology.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.ajhg.2022.06.012" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="The_COVID-19_Host_Genetics_Initiative2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-022-04826-7" target="_blank" rel="noopener">A first update on mapping the human genetic architecture of COVID-19</a></div>
                  <div class="author">
                    The COVID-19 Host Genetics Initiative.
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>608</b>,
                    E1â€“E10
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cSY49" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35922517" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.11.08.21265944" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.covid19hg.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-022-04826-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The COVID-19 pandemic continues to pose a major public health threat, especially in countries with low vaccination rates. To better understand the biological underpinnings of SARS-CoV-2 infection and COVID-19 severity, we formed the COVID-19 Host Genetics Initiative. Here we present a genome-wide association study meta-analysis of up to 125,584 cases and over 2.5â€‰million control individuals across 60 studies from 25 countries, adding 11 genome-wide significant loci compared with those previously identified. Genes at new loci, including SFTPD, MUC5B and ACE2, reveal compelling insights regarding disease susceptibility and severity.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-022-04826-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Winkler2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s42003-022-03448-z" target="_blank" rel="noopener">Differential and shared genetic effects on kidney function between diabetic and non-diabetic individuals</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Winkler, T. W.,
                    </span>
                    <span class="coauthor">
                      Rasheed, H.,
                    </span>
                    <span class="coauthor">
                      Teumer, A.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Gorski, M.,
                      Rowan, B. X.,
                      Stanzick, K. J.,
                      Thomas, L. F.,
                      Tin, A.,
                      Hoppmann, A.,
                      Chu, A. Y.,
                      Tayo, B.,
                      Thio, C. H. L.,
                      Cusi, D.,
                      Chai, J.-F.,
                      Sieber, K. B.,
                      Horn, K.,
                      Li, M.,
                      Scholz, M.,
                      Cocca, M.,
                      Wuttke, M.,
                      Most, P. J.,
                      Yang, Q.,
                      Ghasemi, S.,
                      Nutile, T.,
                      Li, Y.,
                      Pontali, G.,
                      GÃ¼nther, F.,
                      Dehghan, A.,
                      Correa, A.,
                      Parsa, A.,
                      Feresin, A.,
                      Vries, A. P. J.,
                      Zonderman, A. B.,
                      Smith, A. V.,
                      Oldehinkel, A. J.,
                      De Grandi, A.,
                      Rosenkranz, A. R.,
                      Franke, A.,
                      Teren, A.,
                      Metspalu, A.,
                      Hicks, A. A.,
                      Morris, A. P.,
                      TÃ¶njes, A.,
                      Morgan, A.,
                      Podgornaia, A. I.,
                      Peters, A.,
                      KÃ¶rner, A.,
                      Mahajan, A.,
                      Campbell, A.,
                      Freedman, B. I.,
                      Spedicati, B.,
                      Ponte, B.,
                      SchÃ¶ttker, B.,
                      Brumpton, B.,
                      Banas, B.,
                      KrÃ¤mer, B. K.,
                      Jung, B.,
                      Ã…svold, B. O.,
                      Smith, B. H.,
                      Ning, B.,
                      Penninx, B. W. J. H.,
                      Vanderwerff, B. R.,
                      Psaty, B. M.,
                      Kammerer, C. M.,
                      Langefeld, C. D.,
                      Hayward, C.,
                      Spracklen, C. N.,
                      Robinson-Cohen, C.,
                      Hartman, C. A.,
                      Lindgren, C. M.,
                      Wang, C.,
                      Sabanayagam, C.,
                      Heng, C.-K.,
                      Lanzani, C.,
                      Khor, C.-C.,
                      Cheng, C.-Y.,
                      Fuchsberger, C.,
                      Gieger, C.,
                      Shaffer, C. M.,
                      Schulz, C.-A.,
                      Willer, C. J.,
                      Chasman, D. I.,
                      Gudbjartsson, D. F.,
                      Ruggiero, D.,
                      Toniolo, D.,
                      Czamara, D.,
                      Porteous, D. J.,
                      Waterworth, D. M.,
                      Mascalzoni, D.,
                      Mook-Kanamori, D. O.,
                      Reilly, D. F.,
                      Daw, E. W.,
                      Hofer, E.,
                      Boerwinkle, E.,
                      Salvi, E.,
                      Bottinger, E. P.,
                      Tai, E.-S.,
                      Catamo, E.,
                      Rizzi, F.,
                      Guo, F.,
                      Rivadeneira, F.,
                      Guilianini, F.,
                      Sveinbjornsson, G.,
                      Ehret, G.,
                      Waeber, G.,
                      Biino, G.,
                      Girotto, G.,
                      Pistis, G.,
                      Nadkarni, G. N.,
                      Delgado, G. E.,
                      Montgomery, G. W.,
                      Snieder, H.,
                      Campbell, H.,
                      White, H. D.,
                      Gao, H.,
                      Stringham, H. M.,
                      Schmidt, H.,
                      Li, H.,
                      Brenner, H.,
                      Holm, H.,
                      Kirsten, H.,
                      Kramer, H.,
                      Rudan, I.,
                      Nolte, I. M.,
                      Tzoulaki, I.,
                      Olafsson, I.,
                      Martins, J.,
                      Cook, J. P.,
                      Wilson, J. F.,
                      Halbritter, J.,
                      Felix, J. F.,
                      Divers, J.,
                      Kooner, J. S.,
                      Lee, J. J.-M.,
                      Oâ€™Connell, J.,
                      Rotter, J. I.,
                      Liu, J.,
                      Xu, J.,
                      Thiery, J.,
                      Ã„rnlÃ¶v, J.,
                      Kuusisto, J.,
                      Jakobsdottir, J.,
                      Tremblay, J.,
                      Chambers, J. C.,
                      Whitfield, J. B.,
                      Gaziano, J. M.,
                      Marten, J.,
                      Coresh, J.,
                      Jonas, J. B.,
                      Mychaleckyj, J. C.,
                      Christensen, K.,
                      Eckardt, K.-U.,
                      Mohlke, K. L.,
                      Endlich, K.,
                      Dittrich, K.,
                      Ryan, K. A.,
                      Rice, K. M.,
                      Taylor, K. D.,
                      Ho, K.,
                      Nikus, K.,
                      Matsuda, K.,
                      Strauch, K.,
                      Miliku, K.,
                      Hveem, K.,
                      Lind, L.,
                      Wallentin, L.,
                      Yerges-Armstrong, L. M.,
                      Raffield, L. M.,
                      Phillips, L. S.,
                      Launer, L. J.,
                      LyytikÃ¤inen, L.-P.,
                      Lange, L. A.,
                      Citterio, L.,
                      Klaric, L.,
                      Ikram, M. A.,
                      Ising, M.,
                      Kleber, M. E.,
                      Francescatto, M.,
                      Concas, M. P.,
                      Ciullo, M.,
                      Piratsu, M.,
                      Orho-Melander, M.,
                      Laakso, M.,
                      Loeffler, M.,
                      Perola, M.,
                      Borst, M. H.,
                      GÃ¶gele, M.,
                      Bianca, M. L.,
                      Lukas, M. A.,
                      Feitosa, M. F.,
                      Biggs, M. L.,
                      Wojczynski, M. K.,
                      Kavousi, M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Akiyama, M.,
                      Yasuda, M.,
                      Nauck, M.,
                      Waldenberger, M.,
                      Chee, M.-L.,
                      Chee, M.-L.,
                      Boehnke, M.,
                      Preuss, M. H.,
                      Stumvoll, M.,
                      Province, M. A.,
                      Evans, M. K.,
                      Oâ€™Donoghue, M. L.,
                      Kubo, M.,
                      KÃ¤hÃ¶nen, M.,
                      Kastarinen, M.,
                      Nalls, M. A.,
                      Kuokkanen, M.,
                      Ghanbari, M.,
                      Bochud, M.,
                      Josyula, N. S.,
                      Martin, N. G.,
                      Tan, N. Y. Q.,
                      Palmer, N. D.,
                      Pirastu, N.,
                      Schupf, N.,
                      Verweij, N.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Mononen, N.,
                      Bansal, N.,
                      Devuyst, O.,
                      Melander, O.,
                      Raitakari, O. T.,
                      Polasek, O.,
                      Manunta, P.,
                      Gasparini, P.,
                      Mishra, P. P.,
                      Sulem, P.,
                      Magnusson, P. K. E.,
                      Elliott, P.,
                      Ridker, P. M.,
                      Hamet, P.,
                      Svensson, P. O.,
                      Joshi, P. K.,
                      Kovacs, P.,
                      Pramstaller, P. P.,
                      Rossing, P.,
                      Vollenweider, P.,
                      Harst, P.,
                      Dorajoo, R.,
                      Sim, R. Z. H.,
                      Burkhardt, R.,
                      Tao, R.,
                      Noordam, R.,
                      MÃ¤gi, R.,
                      Schmidt, R.,
                      Mutsert, R.,
                      Rueedi, R.,
                      Dam, R. M.,
                      Carroll, R. J.,
                      Gansevoort, R. T.,
                      Loos, R. J. F.,
                      Felicita, S. C.,
                      Sedaghat, S.,
                      Padmanabhan, S.,
                      Freitag-Wolf, S.,
                      Pendergrass, S. A.,
                      Graham, S. E.,
                      Gordon, S. D.,
                      Hwang, S.-J.,
                      Kerr, S. M.,
                      Vaccargiu, S.,
                      Patil, S. B.,
                      Hallan, S.,
                      Bakker, S. J. L.,
                      Lim, S.-C.,
                      Lucae, S.,
                      Vogelezang, S.,
                      Bergmann, S.,
                      Corre, T.,
                      Ahluwalia, T. S.,
                      LehtimÃ¤ki, T.,
                      Boutin, T. S.,
                      Meitinger, T.,
                      Wong, T.-Y.,
                      Bergler, T.,
                      Rabelink, T. J.,
                      Esko, T.,
                      Haller, T.,
                      Thorsteinsdottir, U.,
                      VÃ¶lker, U.,
                      Foo, V. H. X.,
                      Salomaa, V.,
                      Vitart, V.,
                      Giedraitis, V.,
                      Gudnason, V.,
                      Jaddoe, V. W. V.,
                      Huang, W.,
                      Zhang, W.,
                      Wei, W. B.,
                      Kiess, W.,
                      MÃ¤rz, W.,
                      Koenig, W.,
                      Lieb, W.,
                      Gao, X.,
                      Sim, X.,
                      Wang, Y. X.,
                      Friedlander, Y.,
                      Tham, Y.-C.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Milaneschi, Y.,
                      Yu, Z.,
                      Stark, K. J.,
                      Stefansson, K.,
                      BÃ¶ger, C. A.,
                      Hung, A. M.,
                      Kronenberg, F.,
                      KÃ¶ttgen, A.,
                      Pattaro, C.,
                    </span>
                    and Heid, I. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Communications Biology</em>
                    <b>5</b>,
                    580
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cPxCF" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35697829" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s42003-022-03448-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Reduced glomerular filtration rate (GFR) can progress to kidney failure. Risk factors include genetics and diabetes mellitus (DM), but little is known about their interaction. We conducted genome-wide association meta-analyses for estimated GFR based on serum creatinine (eGFR), separately for individuals with or without DM (nDM = 178,691, nnoDM = 1,296,113). Our genome-wide searches identified (i) seven eGFR loci with significant DM/noDM-difference, (ii) four additional novel loci with suggestive difference and (iii) 28 further novel loci (including CUBN) by allowing for potential difference. GWAS on eGFR among DM individuals identified 2 known and 27 potentially responsible loci for diabetic kidney disease. Gene prioritization highlighted 18 genes that may inform reno-protective drug development. We highlight the existence of DM-only and noDM-only effects, which can inform about the target group, if respective genes are advanced as drug targets. Largely shared effects suggest that most drug interventions to alter eGFR should be effective in DM and noDM. A large-scale GWAS provides insight on diabetes-dependent genetic effects on the glomerular filtration rate, a common metric to monitor kidney health in disease.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s42003-022-03448-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Wang_Y2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1146/annurev-biodatasci-111721-074830" target="_blank" rel="noopener">Challenges and Opportunities for Developing More Generalizable Polygenic Risk Scores</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Wang, Y.,
                    </span>
                    <span class="coauthor">
                      Tsuo, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Neale, B. M.,
                    </span>
                    and Martin, A. R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Annual Review of Biomedical Data Science</em>
                    <b>5</b>,
                    293â€“320
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="http://www.annualreviews.org/eprint/2QHJNISUBYMY7XR6RPNZ/full/10.1146/annurev-biodatasci-111721-074830" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35576555" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1146/annurev-biodatasci-111721-074830" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores (PRS) estimate an individualâ€™s genetic likelihood of complex traits and diseases by aggregating information across multiple genetic variants identified from genome-wide association studies. PRS can predict a broad spectrum of diseases and have therefore been widely used in research settings. Some work has investigated their potential applications as biomarkers in preventative medicine, but significant work is still needed to definitively establish and communicate absolute risk to patients for genetic and modifiable risk factors across demographic groups. However, the biggest limitation of PRS currently is that they show poor generalizability across diverse ancestries and cohorts. Major efforts are underway through methodological development and data generation initiatives to improve their generalizability. This review aims to comprehensively discuss current progress on the development of PRS, the factors that affect their generalizability, and promising areas for improving their accuracy, portability, and implementation.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1146/annurev-biodatasci-111721-074830" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Mars2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.xgen.2022.100118" target="_blank" rel="noopener">Genome-wide risk prediction of common diseases across ancestries in one million people</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Mars, N.,
                    </span>
                    <span class="coauthor">
                      Kerminen, S.,
                    </span>
                    <span class="coauthor">
                      Feng, Y.-C. A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      LÃ¤ll, K.,
                      Thomas, L. F.,
                      Skogholt, A. H.,
                      Briotta Parolo, P.,
                      Neale, B. M.,
                      Smoller, J. W.,
                      Gabrielsen, M. E.,
                      Hveem, K.,
                      MÃ¤gi, R.,
                      Matsuda, K.,
                      Okada, Y.,
                      Pirinen, M.,
                      Palotie, A.,
                      Ganna, A.,
                      Martin, A. R.,
                    </span>
                    and Ripatti, S.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell Genomics</em>
                    <b>2</b>,
                    100118
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/cell-genomics/pdfExtended/S2666-979X(22)00042-8" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35591975" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://www2.helsinki.fi/en/news/health-news/polygenic-risk-scores-identify-high-risk-individuals-in-european-and-asian-ancestry-but-less-so-in-african-ancestry" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.xgen.2022.100118" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores (PRS) measure genetic disease susceptibility by combining risk effects across the genome. For coronary artery disease (CAD), type 2 diabetes (T2D), and breast and prostate cancer, we performed cross-ancestry evaluation of genome-wide PRSs in six biobanks in Europe, the United States, and Asia. We studied transferability of these highly polygenic, genome-wide PRSs across global ancestries, within European populations with different health-care systems, and local population substructures in a population isolate. All four PRSs had similar accuracy across European and Asian populations, with poorer transferability in the smaller group of individuals of African ancestry. The PRSs had highly similar effect sizes in different populations of European ancestry, and in early- and late-settlement regions with different recent population bottlenecks in Finland. Comparing genome-wide PRSs to PRSs containing a smaller number of variants, the highly polygenic, genome-wide PRSs generally displayed higher effect sizes and better transferability across global ancestries. Our findings indicate that in the populations investigated, the current genome-wide polygenic scores for common diseases have potential for clinical utility within different health-care settings for individuals of European ancestry, but that the utility in individuals of African ancestry is currently much lower.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.xgen.2022.100118" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Weissbrod2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-022-01036-9" target="_blank" rel="noopener">Leveraging fine-mapping and multipopulation training data to improve cross-population polygenic risk scores</a></div>
                  <div class="author">
                    <span class="coauthor">
                      *Weissbrod, O.,
                    </span>
                    <em>*Kanai, M.</em>,
                    <span class="coauthor">
                      *Shi, H.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Gazal, S.,
                      Peyrot, W. J.,
                      Khera, A. V.,
                      Okada, Y.,
                      The Biobank Japan Project.,
                      Martin, A. R.,
                      Finucane, H. K.,
                    </span>
                    and Price, A. L.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>54</b>,
                    450â€“458
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cKSAw" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/35393596" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.01.19.21249483" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/omerwe/polyfun" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="http://data.broadinstitute.org/alkesgroup/polypred_results" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-022-01036-9" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores suffer reduced accuracy in non-European populations, exacerbating health disparities. We propose PolyPred, a method that improves cross-population polygenic risk scores by combining two predictors: a new predictor that leverages functionally informed fine-mapping to estimate causal effects (instead of tagging effects), addressing linkage disequilibrium differences, and BOLT-LMM, a published predictor. When a large training sample is available in the non-European target population, we propose PolyPred+, which further incorporates the non-European training data. We applied PolyPred to 49 diseases/traits in four UK Biobank populations using UK Biobank British training data, and observed relative improvements versus BOLT-LMM ranging from +7% in south Asians to +32% in Africans, consistent with simulations. We applied PolyPred+ to 23 diseases/traits in UK Biobank east Asians using both UK Biobank British and Biobank Japan training data, and observed improvements of +24% versus BOLT-LMM and +12% versus PolyPred. Summary statistics-based analogs of PolyPred and PolyPred+ attained similar improvements. PolyPred and PolyPred+ methods that leverage fine-mapping and non-European training data significantly improve cross-population polygenic prediction accuracy when applied to diseases and complex traits in UK Biobank populations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-022-01036-9" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Zheng2022" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1093/ije/dyab203" target="_blank" rel="noopener">Trans-ethnic Mendelian-randomization study reveals causal relationships between cardiometabolic factors and chronic kidney disease</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Zheng, J.,
                    </span>
                    <span class="coauthor">
                      Zhang, Y.,
                    </span>
                    <span class="coauthor">
                      Rasheed, H.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Walker, V.,
                      Sugawara, Y.,
                      Li, J.,
                      Leng, Y.,
                      Elsworth, B.,
                      Wootton, R. E.,
                      Fang, S.,
                      Yang, Q.,
                      Burgess, S.,
                      Haycock, P. C.,
                      Borges, M. C.,
                      Cho, Y.,
                      Carnegie, R.,
                      Howell, A.,
                      Robinson, J.,
                      Thomas, L. F.,
                      Brumpton, B. M.,
                      Hveem, K.,
                      Hallan, S.,
                      Franceschini, N.,
                      Morris, A. P.,
                      KÃ¶ttgen, A.,
                      Pattaro, C.,
                      Wuttke, M.,
                      Yamamoto, M.,
                      Kashihara, N.,
                      Akiyama, M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Matsuda, K.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Walters, R.,
                      Millwood, I. Y.,
                      Chen, Z.,
                      Davey Smith, G.,
                      Barbour, S.,
                      Yu, C.,
                      Ã…svold, B. O.,
                      Zhang, H.,
                    </span>
                    and Gaunt, T. R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>International Journal of Epidemiology</em>
                    <b>50</b>,
                    1995â€“2010
                    (2022)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://academic.oup.com/ije/article-pdf/50/6/1995/42092302/dyab203.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34999880" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.09.04.20188284" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1093/ije/dyab203" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>BACKGROUND: This study was to systematically test whether previously reported risk factors for chronic kidney disease (CKD) are causally related to CKD in European and East Asian ancestries using Mendelian randomization. METHODS: A total of 45 risk factors with genetic data in European ancestry and 17 risk factors in East Asian participants were identified as exposures from PubMed. We defined the CKD by clinical diagnosis or by estimated glomerular filtration rate of 25 kg/m2. CONCLUSIONS: Eight cardiometabolic risk factors showed causal effects on CKD in Europeans and three of them showed causality in East Asians, providing insights into the design of future interventions to reduce the burden of CKD.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1093/ije/dyab203" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2021">2021</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Graham2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-021-04064-3" target="_blank" rel="noopener">The power of genetic diversity in genome-wide association studies of lipids</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Graham, S. E.,
                    </span>
                    <span class="coauthor">
                      Clarke, S. L.,
                    </span>
                    <span class="coauthor">
                      Wu, K.-H. H.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kanoni, S.,
                      Zajac, G. J. M.,
                      Ramdas, S.,
                      Surakka, I.,
                      Ntalla, I.,
                      Vedantam, S.,
                      Winkler, T. W.,
                      Locke, A. E.,
                      Marouli, E.,
                      Hwang, M. Y.,
                      Han, S.,
                      Narita, A.,
                      Choudhury, A.,
                      Bentley, A. R.,
                      Ekoru, K.,
                      Verma, A.,
                      Trivedi, B.,
                      Martin, H. C.,
                      Hunt, K. A.,
                      Hui, Q.,
                      Klarin, D.,
                      Zhu, X.,
                      Thorleifsson, G.,
                      Helgadottir, A.,
                      Gudbjartsson, D. F.,
                      Holm, H.,
                      Olafsson, I.,
                      Akiyama, M.,
                      Sakaue, S.,
                      Terao, C.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Zhou, W.,
                      Brumpton, B. M.,
                      Rasheed, H.,
                      Ruotsalainen, S. E.,
                      Havulinna, A. S.,
                      Veturi, Y.,
                      Feng, Q.,
                      Rosenthal, E. A.,
                      Lingren, T.,
                      Pacheco, J. A.,
                      Pendergrass, S. A.,
                      Haessler, J.,
                      Giulianini, F.,
                      Bradford, Y.,
                      Miller, J. E.,
                      Campbell, A.,
                      Lin, K.,
                      Millwood, I. Y.,
                      Hindy, G.,
                      Rasheed, A.,
                      Faul, J. D.,
                      Zhao, W.,
                      Weir, D. R.,
                      Turman, C.,
                      Huang, H.,
                      Graff, M.,
                      Mahajan, A.,
                      Brown, M. R.,
                      Zhang, W.,
                      Yu, K.,
                      Schmidt, E. M.,
                      Pandit, A.,
                      Gustafsson, S.,
                      Yin, X.,
                      Luan, J.,
                      Zhao, J.-H.,
                      Matsuda, F.,
                      Jang, H.-M.,
                      Yoon, K.,
                      Medina-Gomez, C.,
                      Pitsillides, A.,
                      Hottenga, J. J.,
                      Willemsen, G.,
                      Wood, A. R.,
                      Ji, Y.,
                      Gao, Z.,
                      Haworth, S.,
                      Mitchell, R. E.,
                      Chai, J. F.,
                      Aadahl, M.,
                      Yao, J.,
                      Manichaikul, A.,
                      Warren, H. R.,
                      Ramirez, J.,
                      Bork-Jensen, J.,
                      KÃ¥rhus, L. L.,
                      Goel, A.,
                      Sabater-Lleal, M.,
                      Noordam, R.,
                      Sidore, C.,
                      Fiorillo, E.,
                      McDaid, A. F.,
                      Marques-Vidal, P.,
                      Wielscher, M.,
                      Trompet, S.,
                      Sattar, N.,
                      MÃ¸llehave, L. T.,
                      Thuesen, B. H.,
                      Munz, M.,
                      Zeng, L.,
                      Huang, J.,
                      Yang, B.,
                      Poveda, A.,
                      Kurbasic, A.,
                      Lamina, C.,
                      Forer, L.,
                      Scholz, M.,
                      Galesloot, T. E.,
                      Bradfield, J. P.,
                      Daw, E. W.,
                      Zmuda, J. M.,
                      Mitchell, J. S.,
                      Fuchsberger, C.,
                      Christensen, H.,
                      Brody, J. A.,
                      Feitosa, M. F.,
                      Wojczynski, M. K.,
                      Preuss, M.,
                      Mangino, M.,
                      Christofidou, P.,
                      Verweij, N.,
                      Benjamins, J. W.,
                      Engmann, J.,
                      Kember, R. L.,
                      Slieker, R. C.,
                      Lo, K. S.,
                      Zilhao, N. R.,
                      Le, P.,
                      Kleber, M. E.,
                      Delgado, G. E.,
                      Huo, S.,
                      Ikeda, D. D.,
                      Iha, H.,
                      Yang, J.,
                      Liu, J.,
                      Leonard, H. L.,
                      Marten, J.,
                      Schmidt, B.,
                      Arendt, M.,
                      Smyth, L. J.,
                      CaÃ±adas-Garre, M.,
                      Wang, C.,
                      Nakatochi, M.,
                      Wong, A.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Sim, X.,
                      Xia, R.,
                      Huerta-Chagoya, A.,
                      Fernandez-Lopez, J. C.,
                      Lyssenko, V.,
                      Ahmed, M.,
                      Jackson, A. U.,
                      Irvin, M. R.,
                      Oldmeadow, C.,
                      Kim, H.-N.,
                      Ryu, S.,
                      Timmers, P. R. H. J.,
                      Arbeeva, L.,
                      Dorajoo, R.,
                      Lange, L. A.,
                      Chai, X.,
                      Prasad, G.,
                      LorÃ©s-Motta, L.,
                      Pauper, M.,
                      Long, J.,
                      Li, X.,
                      Theusch, E.,
                      Takeuchi, F.,
                      Spracklen, C. N.,
                      Loukola, A.,
                      Bollepalli, S.,
                      Warner, S. C.,
                      Wang, Y. X.,
                      Wei, W. B.,
                      Nutile, T.,
                      Ruggiero, D.,
                      Sung, Y. J.,
                      Hung, Y.-J.,
                      Chen, S.,
                      Liu, F.,
                      Yang, J.,
                      Kentistou, K. A.,
                      Gorski, M.,
                      Brumat, M.,
                      Meidtner, K.,
                      Bielak, L. F.,
                      Smith, J. A.,
                      Hebbar, P.,
                      Farmaki, A.-E.,
                      Hofer, E.,
                      Lin, M.,
                      Xue, C.,
                      Zhang, J.,
                      Concas, M. P.,
                      Vaccargiu, S.,
                      Most, P. J.,
                      PitkÃ¤nen, N.,
                      Cade, B. E.,
                      Lee, J.,
                      Laan, S. W.,
                      Chitrala, K. N.,
                      Weiss, S.,
                      Zimmermann, M. E.,
                      Lee, J. Y.,
                      Choi, H. S.,
                      Nethander, M.,
                      Freitag-Wolf, S.,
                      Southam, L.,
                      Rayner, N. W.,
                      Wang, C. A.,
                      Lin, S.-Y.,
                      Wang, J.-S.,
                      Couture, C.,
                      LyytikÃ¤inen, L.-P.,
                      Nikus, K.,
                      Cuellar-Partida, G.,
                      Vestergaard, H.,
                      Hildalgo, B.,
                      Giannakopoulou, O.,
                      Cai, Q.,
                      Obura, M. O.,
                      Setten, J.,
                      Li, X.,
                      Schwander, K.,
                      Terzikhan, N.,
                      Shin, J. H.,
                      Jackson, R. D.,
                      Reiner, A. P.,
                      Martin, L. W.,
                      Chen, Z.,
                      Li, L.,
                      Highland, H. M.,
                      Young, K. L.,
                      Kawaguchi, T.,
                      Thiery, J.,
                      Bis, J. C.,
                      Nadkarni, G. N.,
                      Launer, L. J.,
                      Li, H.,
                      Nalls, M. A.,
                      Raitakari, O. T.,
                      Ichihara, S.,
                      Wild, S. H.,
                      Nelson, C. P.,
                      Campbell, H.,
                      JÃ¤ger, S.,
                      Nabika, T.,
                      Al-Mulla, F.,
                      Niinikoski, H.,
                      Braund, P. S.,
                      Kolcic, I.,
                      Kovacs, P.,
                      Giardoglou, T.,
                      Katsuya, T.,
                      Bhatti, K. F.,
                      Kleijn, D.,
                      Borst, G. J.,
                      Kim, E. K.,
                      Adams, H. H. H.,
                      Ikram, M. A.,
                      Zhu, X.,
                      Asselbergs, F. W.,
                      Kraaijeveld, A. O.,
                      Beulens, J. W. J.,
                      Shu, X.-O.,
                      Rallidis, L. S.,
                      Pedersen, O.,
                      Hansen, T.,
                      Mitchell, P.,
                      Hewitt, A. W.,
                      KÃ¤hÃ¶nen, M.,
                      PÃ©russe, L.,
                      Bouchard, C.,
                      TÃ¶njes, A.,
                      Chen, Y.-D. I.,
                      Pennell, C. E.,
                      Mori, T. A.,
                      Lieb, W.,
                      Franke, A.,
                      Ohlsson, C.,
                      MellstrÃ¶m, D.,
                      Cho, Y. S.,
                      Lee, H.,
                      Yuan, J.-M.,
                      Koh, W.-P.,
                      Rhee, S. Y.,
                      Woo, J.-T.,
                      Heid, I. M.,
                      Stark, K. J.,
                      VÃ¶lzke, H.,
                      Homuth, G.,
                      Evans, M. K.,
                      Zonderman, A. B.,
                      Polasek, O.,
                      Pasterkamp, G.,
                      Hoefer, I. E.,
                      Redline, S.,
                      Pahkala, K.,
                      Oldehinkel, A. J.,
                      Snieder, H.,
                      Biino, G.,
                      Schmidt, R.,
                      Schmidt, H.,
                      Chen, Y. E.,
                      Bandinelli, S.,
                      Dedoussis, G.,
                      Thanaraj, T. A.,
                      Kardia, S. L. R.,
                      Kato, N.,
                      Schulze, M. B.,
                      Girotto, G.,
                      Jung, B.,
                      BÃ¶ger, C. A.,
                      Joshi, P. K.,
                      Bennett, D. A.,
                      De Jager, P. L.,
                      Lu, X.,
                      Mamakou, V.,
                      Brown, M.,
                      Caulfield, M. J.,
                      Munroe, P. B.,
                      Guo, X.,
                      Ciullo, M.,
                      Jonas, J. B.,
                      Samani, N. J.,
                      Kaprio, J.,
                      Pajukanta, P.,
                      Adair, L. S.,
                      Bechayda, S. A.,
                      Silva, H. J.,
                      Wickremasinghe, A. R.,
                      Krauss, R. M.,
                      Wu, J.-Y.,
                      Zheng, W.,
                      Hollander, A. I.,
                      Bharadwaj, D.,
                      Correa, A.,
                      Wilson, J. G.,
                      Lind, L.,
                      Heng, C.-K.,
                      Nelson, A. E.,
                      Golightly, Y. M.,
                      Wilson, J. F.,
                      Penninx, B.,
                      Kim, H.-L.,
                      Attia, J.,
                      Scott, R. J.,
                      Rao, D. C.,
                      Arnett, D. K.,
                      Walker, M.,
                      Koistinen, H. A.,
                      Chandak, G. R.,
                      Yajnik, C. S.,
                      Mercader, J. M.,
                      TusiÃ©-Luna, T.,
                      Aguilar-Salinas, C. A.,
                      Villalpando, C. G.,
                      Orozco, L.,
                      Fornage, M.,
                      Tai, E. S.,
                      Dam, R. M.,
                      LehtimÃ¤ki, T.,
                      Chaturvedi, N.,
                      Yokota, M.,
                      Liu, J.,
                      Reilly, D. F.,
                      McKnight, A. J.,
                      Kee, F.,
                      JÃ¶ckel, K.-H.,
                      McCarthy, M. I.,
                      Palmer, C. N. A.,
                      Vitart, V.,
                      Hayward, C.,
                      Simonsick, E.,
                      Duijn, C. M.,
                      Lu, F.,
                      Qu, J.,
                      Hishigaki, H.,
                      Lin, X.,
                      MÃ¤rz, W.,
                      Parra, E. J.,
                      Cruz, M.,
                      Gudnason, V.,
                      Tardif, J.-C.,
                      Lettre, G.,
                      Hart, L. M.,
                      Elders, P. J. M.,
                      Damrauer, S. M.,
                      Kumari, M.,
                      Kivimaki, M.,
                      Harst, P.,
                      Spector, T. D.,
                      Loos, R. J. F.,
                      Province, M. A.,
                      Psaty, B. M.,
                      Brandslund, I.,
                      Pramstaller, P. P.,
                      Christensen, K.,
                      Ripatti, S.,
                      WidÃ©n, E.,
                      Hakonarson, H.,
                      Grant, S. F. A.,
                      Kiemeney, L. A. L. M.,
                      Graaf, J.,
                      Loeffler, M.,
                      Kronenberg, F.,
                      Gu, D.,
                      Erdmann, J.,
                      Schunkert, H.,
                      Franks, P. W.,
                      Linneberg, A.,
                      Jukema, J. W.,
                      Khera, A. V.,
                      MÃ¤nnikkÃ¶, M.,
                      Jarvelin, M.-R.,
                      Kutalik, Z.,
                      Cucca, F.,
                      Mook-Kanamori, D. O.,
                      Dijk, K. W.,
                      Watkins, H.,
                      Strachan, D. P.,
                      Grarup, N.,
                      Sever, P.,
                      Poulter, N.,
                      Rotter, J. I.,
                      Dantoft, T. M.,
                      Karpe, F.,
                      Neville, M. J.,
                      Timpson, N. J.,
                      Cheng, C.-Y.,
                      Wong, T.-Y.,
                      Khor, C. C.,
                      Sabanayagam, C.,
                      Peters, A.,
                      Gieger, C.,
                      Hattersley, A. T.,
                      Pedersen, N. L.,
                      Magnusson, P. K. E.,
                      Boomsma, D. I.,
                      Geus, E. J. C.,
                      Cupples, L. A.,
                      Meurs, J. B. J.,
                      Ghanbari, M.,
                      Gordon-Larsen, P.,
                      Huang, W.,
                      Kim, Y. J.,
                      Tabara, Y.,
                      Wareham, N. J.,
                      Langenberg, C.,
                      Zeggini, E.,
                      Kuusisto, J.,
                      Laakso, M.,
                      Ingelsson, E.,
                      Abecasis, G.,
                      Chambers, J. C.,
                      Kooner, J. S.,
                      Vries, P. S.,
                      Morrison, A. C.,
                      North, K. E.,
                      Daviglus, M.,
                      Kraft, P.,
                      Martin, N. G.,
                      Whitfield, J. B.,
                      Abbas, S.,
                      Saleheen, D.,
                      Walters, R. G.,
                      Holmes, M. V.,
                      Black, C.,
                      Smith, B. H.,
                      Justice, A. E.,
                      Baras, A.,
                      Buring, J. E.,
                      Ridker, P. M.,
                      Chasman, D. I.,
                      Kooperberg, C.,
                      Wei, W.-Q.,
                      Jarvik, G. P.,
                      Namjou, B.,
                      Hayes, M. G.,
                      Ritchie, M. D.,
                      Jousilahti, P.,
                      Salomaa, V.,
                      Hveem, K.,
                      Ã…svold, B. O.,
                      Kubo, M.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Murakami, Y.,
                      Thorsteinsdottir, U.,
                      Stefansson, K.,
                      Ho, Y.-L.,
                      Lynch, J. A.,
                      Rader, D. J.,
                      Tsao, P. S.,
                      Chang, K.-M.,
                      Cho, K.,
                      Oâ€™Donnell, C. J.,
                      Gaziano, J. M.,
                      Wilson, P.,
                      Rotimi, C. N.,
                      Hazelhurst, S.,
                      Ramsay, M.,
                      Trembath, R. C.,
                      Heel, D. A.,
                      Tamiya, G.,
                      Yamamoto, M.,
                      Kim, B.-J.,
                      Mohlke, K. L.,
                      Frayling, T. M.,
                      Hirschhorn, J. N.,
                      Kathiresan, S.,
                      Boehnke, M.,
                      Natarajan, P.,
                      Peloso, G. M.,
                      Brown, C. D.,
                      Morris, A. P.,
                      Assimes, T. L.,
                      Deloukas, P.,
                      Sun, Y. V.,
                    </span>
                    and Willer, C. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>600</b>,
                    675â€“679
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cCWym" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34887591" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://csg.sph.umich.edu/willer/public/glgc-lipids2021/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-021-04064-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Increased blood lipid levels are heritable risk factors of cardiovascular disease with varied prevalence worldwide owing to different dietary patterns and medication use1. Despite advances in prevention and treatment, in particular through reducing low-density lipoprotein cholesterol levels2, heart disease remains the leading cause of death worldwide3. Genome-wideassociation studies (GWAS) of blood lipid levels have led to important biological and clinical insights, as well as new drug targets, for cardiovascular disease. However, most previous GWAS4â€“23 have been conducted in European ancestry populations and may have missed genetic variants that contribute to lipid-level variation in other ancestry groups. These include differences in allele frequencies, effect sizes and linkage-disequilibrium patterns24. Here we conduct a multi-ancestry, genome-wide genetic discovery meta-analysis of lipid levels in approximately 1.65 million individuals, including 350,000 of non-European ancestries. We quantify the gain in studying non-European ancestries and provide evidence to support the expansion of recruitment of additional ancestries, even with relatively small sample sizes. We find that increasing diversity rather than studying additional individuals of European ancestry results in substantial improvements in fine-mapping functional variants and portability of polygenic prediction (evaluated in approximately 295,000 individuals from 7 ancestry groupings). Modest gains in the number of discovered loci and ancestry-specific variants were also achieved. As GWAS expand emphasis beyond the identification of genes and fundamental biology towards the use of genetic variants for preventive and precision medicine25, we anticipate that increased diversity of participants will lead to more accurate and equitable26 application of polygenic scores in clinical practice. A genome-wide association meta-analysis study of blood lipid levels in roughly 1.6 million individuals demonstrates the gain of power attained when diverse ancestries are included to improve fine-mapping and polygenic score generation, with gains in locus discovery related to sample size.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-021-04064-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Polygenic_Risk_Score_Task_Force_of_the_International_Common_Disease_Alliance_2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41591-021-01549-6" target="_blank" rel="noopener">Responsible use of polygenic risk scores in the clinic: potential benefits, risks and gaps</a></div>
                  <div class="author">
                    Polygenic Risk Score Task Force of the International Common Disease Alliance.
                  </div>
                  <div class="periodical">
                    <em>Nature Medicine</em>
                    <b>27</b>,
                    1876â€“1884
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cBoE2" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34782789" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41591-021-01549-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores (PRSs) aggregate the many small effects of alleles across the human genome to estimate the risk of a disease or disease-related trait for an individual. The potential benefits of PRSs include cost-effective enhancement of primary disease prevention, more refined diagnoses and improved precision when prescribing medicines. However, these must be weighed against the potential risks, such as uncertainties and biases in PRS performance, as well as potential misunderstanding and misuse of these within medical practice and in wider society. By addressing key issues including gaps in best practices, risk communication and regulatory frameworks, PRSs can be used responsibly to improve human health. Here, the International Common Disease Allianceâ€™s PRS Task Force, a multidisciplinary group comprising expertise in genetics, law, ethics, behavioral science and more, highlights recent research to provide a comprehensive summary of the state of polygenic score research, as well as the needs and challenges as PRSs move closer to widespread use in the clinic. As polygenic risk scores move closer to widespread clinical use, this Perspective summarizes the benefits, risks and challenges to be overcome.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41591-021-01549-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Luo2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-021-00935-7" target="_blank" rel="noopener">A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Luo, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Choi, W.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Li, X.,
                      Sakaue, S.,
                      Yamamoto, K.,
                      Ogawa, K.,
                      Gutierrez-Arcelus, M.,
                      Gregersen, P. K.,
                      Stuart, P. E.,
                      Elder, J. T.,
                      Forer, L.,
                      SchÃ¶nherr, S.,
                      Fuchsberger, C.,
                      Smith, A. V.,
                      Fellay, J.,
                      Carrington, M.,
                      Haas, D. W.,
                      Guo, X.,
                      Palmer, N. D.,
                      Chen, Y.-D. I.,
                      Rotter, J. I.,
                      Taylor, K. D.,
                      Rich, S. S.,
                      Correa, A.,
                      Wilson, J. G.,
                      Kathiresan, S.,
                      Cho, M. H.,
                      Metspalu, A.,
                      Esko, T.,
                      Okada, Y.,
                      Han, B.,
                      McLaren, P. J.,
                    </span>
                    and Raychaudhuri, S.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>53</b>,
                    1504â€“1516
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cyVza" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34611364" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.07.16.20155606" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://github.com/immunogenomics/HLA-TAPAS" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-021-00935-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Fine-mapping to plausible causal variation may be more effective in multi-ancestry cohorts, particularly in the MHC, which has population-specific structure. To enable such studies, we constructed a large (n = 21,546) HLA reference panel spanning five global populations based on whole-genome sequences. Despite population-specific long-range haplotypes, we demonstrated accurate imputation at G-group resolution (94.2%, 93.7%, 97.8% and 93.7% in admixed African (AA), East Asian (EAS), European (EUR) and Latino (LAT) populations). Applying HLA imputation to genome-wide association study data for HIV-1 viral load in three populations (EUR, AA and LAT), we obviated effects of previously reported associations from population-specific HIV studies and discovered a novel association at position 156 in HLA-B. We pinpointed the MHC association to three amino acid positions (97, 67 and 156) marking three consecutive pockets (C, B and D) within the HLA-B peptide-binding groove, explaining 12.9% of trait variance. A high-resolution reference panel based on whole-genome sequencing data enables accurate imputation of HLA alleles across diverse populations and fine-mapping of HLA association signals for HIV-1 host response.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-021-00935-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sakaue2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-021-00931-x" target="_blank" rel="noopener">A cross-population atlas of genetic associations for 220 human phenotypes</a></div>
                  <div class="author">
                    <span class="coauthor">
                      *Sakaue, S.,
                    </span>
                    <em>*Kanai, M.</em>,
                    <span class="coauthor">
                      Tanigawa, Y.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karjalainen, J.,
                      Kurki, M.,
                      Koshiba, S.,
                      Narita, A.,
                      Konuma, T.,
                      Yamamoto, K.,
                      Akiyama, M.,
                      Ishigaki, K.,
                      Suzuki, A.,
                      Suzuki, K.,
                      Obara, W.,
                      Yamaji, K.,
                      Takahashi, K.,
                      Asai, S.,
                      Takahashi, Y.,
                      Suzuki, T.,
                      Shinozaki, N.,
                      Yamaguchi, H.,
                      Minami, S.,
                      Murayama, S.,
                      Yoshimori, K.,
                      Nagayama, S.,
                      Obata, D.,
                      Higashiyama, M.,
                      Masumoto, A.,
                      Koretsune, Y.,
                      Ito, K.,
                      Terao, C.,
                      Yamauchi, T.,
                      Komuro, I.,
                      Kadowaki, T.,
                      Tamiya, G.,
                      Yamamoto, M.,
                      Nakamura, Y.,
                      Kubo, M.,
                      Murakami, Y.,
                      Yamamoto, K.,
                      Kamatani, Y.,
                      Palotie, A.,
                      Rivas, M. A.,
                      Daly, M. J.,
                      Matsuda, K.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>53</b>,
                    1415â€“1424
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cyG6p" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34594039" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.10.23.20213652" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://pheweb.jp/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <a href="https://www.med.osaka-u.ac.jp/eng/activities/results/2021year/okada2021-10-01" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-021-00931-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Current genome-wide association studies do not yet capture sufficient diversity in populations and scope of phenotypes. To expand an atlas of genetic associations in non-European populations, we conducted 220 deep-phenotype genome-wide association studies (diseases, biomarkers and medication usage) in BioBank Japan (n = 179,000), by incorporating past medical history and text-mining of electronic medical records. Meta-analyses with the UK Biobank and FinnGen (ntotal = 628,000) identified Â 5,000 new loci, which improved the resolution of the genomic map of human traits. This atlas elucidated the landscape of pleiotropy as represented by the major histocompatibility complex locus, where we conducted HLA fine-mapping. Finally, we performed statistical decomposition of matrices of phenome-wide summary statistics, and identified latent genetic components, which pinpointed responsible variants and biological mechanisms underlying current disease classifications across populations. The decomposed components enabled genetically informed subtyping of similar diseases (for example, allergic diseases). Our study suggests a potential avenue for hypothesis-free re-investigation of human diseases through genetics. Genome-wide analyses in BioBank Japan, UK Biobank and FinnGen identify Â 5,000 new loci associated with 220 human traits. Statistical decomposition of matrices of phenome-wide summary statistics further highlights variants underpinning diseases across populations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-021-00931-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Griesemer2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.cell.2021.08.025" target="_blank" rel="noopener">Genome-wide functional screen of 3â€™UTR variants uncovers causal variants for human disease and evolution</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Griesemer, D.,
                    </span>
                    <span class="coauthor">
                      Xue, J. R.,
                    </span>
                    <span class="coauthor">
                      Reilly, S. K.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ulirsch, J. C.,
                      Kukreja, K.,
                      Davis, J. R.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Yang, D. K.,
                      Butts, J. C.,
                      Guney, M. H.,
                      Luban, J.,
                      Montgomery, S. B.,
                      Finucane, H. K.,
                      Novina, C. D.,
                      Tewhey, R.,
                    </span>
                    and Sabeti, P. C.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell</em>
                    <b>184</b>,
                    5247â€“5260.e19
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34534445" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.01.13.424697" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.cell.2021.08.025" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>3â€™ untranslated region (3â€™UTR) variants are strongly associated with human traits and diseases, yet few have been causally identified. We developed the massively parallel reporter assay for 3â€™UTRs (MPRAu) to sensitively assay 12,173 3â€™UTR variants. We applied MPRAu to six human cell lines, focusing on genetic variants associated with genome-wide association studies (GWAS) and human evolutionary adaptation. MPRAu expands our understanding of 3â€™UTR function, suggesting that simple sequences predominately explain 3â€™UTR regulatory activity. We adapt MPRAu to uncover diverse molecular mechanisms at base pair resolution, including an adenylate-uridylate (AU)-rich element of LEPR linked to potential metabolic evolutionary adaptations in East Asians. We nominate hundreds of 3â€™UTR causal variants with genetically fine-mapped phenotype associations. Using endogenous allelic replacements, we characterize one variant that disrupts a miRNA site regulating the viral defense gene TRIM14 and one that alters PILRB abundance, nominating a causal variant underlying transcriptional changes in age-related macular degeneration.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.cell.2021.08.025" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Reilly2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-021-00900-4" target="_blank" rel="noopener">Direct characterization of cis-regulatory elements and functional dissection of complex genetic associations using HCRâ€“FlowFISH</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Reilly, S. K.,
                    </span>
                    <span class="coauthor">
                      Gosai, S. J.,
                    </span>
                    <span class="coauthor">
                      Guiterrez, A.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ulirsch, J. C.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Berenzy, D.,
                      Kales, S.,
                      Butler, G. B.,
                      Gladden-Young, A.,
                      Finucane, H. K.,
                      Sabeti, P. C.,
                    </span>
                    and Tewhey, R.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>53</b>,
                    1166â€“1176
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cqSX2" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34326544" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.05.11.078675" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://www.jax.org/news-and-insights/2021/july/determining-the-function-of-genomic-regulatory-elements" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-021-00900-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Effective interpretation of genome function and genetic variation requires a shift from epigenetic mapping of cis-regulatory elements (CREs) to characterization of endogenous function. We developed hybridization chain reaction fluorescence in situ hybridization coupled with flow cytometry (HCRâ€“FlowFISH), a broadly applicable approach to characterize CRISPR-perturbed CREs via accurate quantification of native transcripts, alongside CRISPR activity screen analysis (CASA), a hierarchical Bayesian model to quantify CRE activity. Across &gt;325,000 perturbations, we provide evidence that CREs can regulate multiple genes, skip over the nearest gene and display activating and/or silencing effects. At the cholesterol-level-associated FADS locus, we combine endogenous screens with reporter assays to exhaustively characterize multiple genome-wide association signals, functionally nominate causal variants and, importantly, identify their target genes. HCRâ€“FlowFISH is a new approach to characterize CRISPR-perturbed cis-regulatory elements (CREs) via accurate quantification of native transcripts, alongside CRISPR activity screen analysis (CASA), a hierarchical Bayesian model to quantify CRE activity.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-021-00900-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="The_COVID-19_Host_Genetics_Initiative2021a" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41586-021-03767-x" target="_blank" rel="noopener">Mapping the human genetic architecture of COVID-19</a></div>
                  <div class="author">
                    The COVID-19 Host Genetics Initiative.
                  </div>
                  <div class="periodical">
                    <em>Nature</em>
                    <b>600</b>,
                    472â€“477
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cnZTx" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34237774" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2021.03.10.21252820" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.covid19hg.org/" class="btn btn-sm z-depth-0" role="button" target="_blank">Website</a>
                    <a href="https://www.broadinstitute.org/news/large-genomic-analysis-highlights-covid-19-risk-factors" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41586-021-03767-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The genetic makeup of an individual contributes to susceptibility and response to viral infection. While environmental, clinical and social factors play a role in exposure to SARS-CoV-2 and COVID-19 disease severity1,2, host genetics may also be important. Identifying host-specific genetic factors may reveal biological mechanisms of therapeutic relevance and clarify causal relationships of modifiable environmental risk factors for SARS-CoV-2 infection and outcomes. We formed a global network of researchers to investigate the role of human genetics in SARS-CoV-2 infection and COVID-19 severity. We describe the results of three genome-wide association meta-analyses comprised of up to 49,562 COVID-19 patients from 46 studies across 19 countries. We reported 13 genome-wide significant loci that are associated with SARS-CoV-2 infection or severe manifestations of COVID-19. Several of these loci correspond to previously documented associations to lung or autoimmune and inflammatory diseases3â€“7. They also represent potentially actionable mechanisms in response to infection. Mendelian Randomization analyses support a causal role for smoking and body mass index for severe COVID-19 although not for type II diabetes. The identification of novel host genetic factors associated with COVID-19, with unprecedented speed, was made possible by the community of human genetic researchers coming together to prioritize sharing of data, results, resources and analytical frameworks. This working model of international collaboration underscores what is possible for future genetic discoveries in emerging pandemics, or indeed for any complex human disease.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41586-021-03767-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Wang_QS2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-021-23134-8" target="_blank" rel="noopener">Leveraging supervised learning for functionally informed fine-mapping of cis-eQTLs identifies an additional 20,913 putative causal eQTLs</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Wang, Q. S.,
                    </span>
                    <span class="coauthor">
                      Kelley, D. R.,
                    </span>
                    <span class="coauthor">
                      Ulirsch, J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Sadhuka, S.,
                      Cui, R.,
                      Albors, C.,
                      Cheng, N.,
                      Okada, Y.,
                      Aguet, F.,
                      Ardlie, K. G.,
                      MacArthur, D. G.,
                    </span>
                    and Finucane, H. K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>12</b>,
                    3394
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cl4Pw" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/34099641" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.10.20.347294" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://github.com/FinucaneLab/Expression_Modifier_Score/" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.finucanelab.org/data" class="btn btn-sm z-depth-0" role="button" target="_blank">Data</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-021-23134-8" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The large majority of variants identified by GWAS are non-coding, motivating detailed characterization of the function of non-coding variants. Experimental methods to assess variantsâ€™ effect on gene expressions in native chromatin context via direct perturbation are low-throughput. Existing high-throughput computational predictors thus have lacked large gold standard sets of regulatory variants for training and validation. Here, we leverage a set of 14,807 putative causal eQTLs in humans obtained through statistical fine-mapping, and we use 6121 features to directly train a predictor of whether a variant modifies nearby gene expression. We call the resulting prediction the expression modifier score (EMS). We validate EMS by comparing its ability to prioritize functional variants with other major scores. We then use EMS as a prior for statistical fine-mapping of eQTLs to identify an additional 20,913 putatively causal eQTLs, and we incorporate EMS into co-localization analysis to identify 310 additional candidate genes across UK Biobank phenotypes. Finding causal variants and genes from GWAS loci results remains a challenge. Here, the authors train a model to predict if a variant affects nearby gene expression, and apply the method to identify new possible causal eQTLs and mechanisms of GWAS loci.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-021-23134-8" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Nakatochi2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1093/rheumatology/keab404" target="_blank" rel="noopener">An X chromosome-wide meta-analysis based on Japanese cohorts revealed that non-autosomal variations are associated with serum urate</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Nakatochi, M.,
                    </span>
                    <span class="coauthor">
                      Toyoda, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Nakayama, A.,
                      Kawamura, Y.,
                      Hishida, A.,
                      Mikami, H.,
                      Matsuo, K.,
                      Takezaki, T.,
                      Momozawa, Y.,
                      Biobank Japan Project.,
                      Kamatani, Y.,
                      Ichihara, S.,
                      Shinomiya, N.,
                      Yokota, M.,
                      Wakai, K.,
                      Okada, Y.,
                      Matsuo, H.,
                    </span>
                    and Japan Uric Acid Genomics Consortium (Japan Urate).
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Rheumatology</em>
                    <b>60</b>,
                    4430â€“4432
                    (2021)
                  </div>
                  <div class="links">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/33944914" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1093/rheumatology/keab404" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1093/rheumatology/keab404" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Pirastu2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-021-00846-7" target="_blank" rel="noopener">Genetic analyses identify widespread sex-differential participation bias</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Pirastu, N.,
                    </span>
                    <span class="coauthor">
                      Cordioli, M.,
                    </span>
                    <span class="coauthor">
                      Nandakumar, P.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Mignogna, G.,
                      Abdellaoui, A.,
                      Hollis, B.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Rajagopal, V. M.,
                      Della Briotta Parolo, P.,
                      Baya, N.,
                      Carey, C. E.,
                      Karjalainen, J.,
                      Als, T. D.,
                      Zee, M. D.,
                      Day, F. R.,
                      Ong, K. K.,
                      Morisaki, T.,
                      Geus, E.,
                      Bellocco, R.,
                      Okada, Y.,
                      BÃ¸rglum, A. D.,
                      Joshi, P.,
                      Auton, A.,
                      Hinds, D.,
                      Neale, B. M.,
                      Walters, R. K.,
                      Nivard, M. G.,
                      Perry, J. R. B.,
                    </span>
                    and Ganna, A.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>53</b>,
                    663â€“671
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/clzCa" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/33888908" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.03.22.001453" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-021-00846-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Genetic association results are often interpreted with the assumption that study participation does not affect downstream analyses. Understanding the genetic basis of participation bias is challenging since it requires the genotypes of unseen individuals. Here we demonstrate that it is possible to estimate comparative biases by performing a genome-wide association study contrasting one subgroup versus another. For example, we showed that sex exhibits artifactual autosomal heritability in the presence of sex-differential participation bias. By performing a genome-wide association study of sex in approximately 3.3 million males and females, we identified over 158 autosomal loci spuriously associated with sex and highlighted complex traits underpinning differences in study participation between the sexes. For example, the body mass index-increasing allele at FTO was observed at higher frequency in males compared to females (odds ratio = 1.02, P = 4.4 \times 10-36). Finally, we demonstrated how these biases can potentially lead to incorrect inferences in downstream analyses and propose a conceptual framework for addressing such biases. Our findings highlight a new challenge that genetic studies may face as sample sizes continue to grow.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-021-00846-7" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Shi2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-021-21286-1" target="_blank" rel="noopener">Population-specific causal disease effect sizes in functionally important regions impacted by selection</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Shi, H.,
                    </span>
                    <span class="coauthor">
                      Gazal, S.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Koch, E. M.,
                      Schoech, A. P.,
                      Siewert, K. M.,
                      Kim, S. S.,
                      Luo, Y.,
                      Amariuta, T.,
                      Huang, H.,
                      Okada, Y.,
                      Raychaudhuri, S.,
                      Sunyaev, S. R.,
                    </span>
                    and Price, A. L.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>12</b>,
                    1098
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/clOBn" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/33597505" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/803452" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://github.com/huwenboshi/s-ldxr" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-021-21286-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Many diseases exhibit population-specific causal effect sizes with trans-ethnic genetic correlations significantly less than 1, limiting trans-ethnic polygenic risk prediction. We develop a new method, S-LDXR, for stratifying squared trans-ethnic genetic correlation across genomic annotations, and apply S-LDXR to genome-wide summary statistics for 31 diseases and complex traits in East Asians (average N = 90K) and Europeans (average N = 267K) with an average trans-ethnic genetic correlation of 0.85. We determine that squared trans-ethnic genetic correlation is 0.82\times (s.e. 0.01) depleted in the top quintile of background selection statistic, implying more population-specific causal effect sizes. Accordingly, causal effect sizes are more population-specific in functionally important regions, including conserved and regulatory regions. In regions surrounding specifically expressed genes, causal effect sizes are most population-specific for skin and immune genes, and least population-specific for brain genes. Our results could potentially be explained by stronger gene-environment interaction at loci impacted by selection, particularly positive selection.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-021-21286-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Atkinson2021" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-020-00766-y" target="_blank" rel="noopener">Tractor uses local ancestry to enable the inclusion of admixed individuals in GWAS and to boost power</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Atkinson, E. G.,
                    </span>
                    <span class="coauthor">
                      Maihofer, A. X.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Martin, A. R.,
                      Karczewski, K. J.,
                      Santoro, M. L.,
                      Ulirsch, J. C.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Finucane, H. K.,
                      Koenen, K. C.,
                      Nievergelt, C. M.,
                      Daly, M. J.,
                    </span>
                    and Neale, B. M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>53</b>,
                    195â€“204
                    (2021)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/clZ2f" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/33462486" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.05.17.100727" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://github.com/eatkinson/Tractor" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.massgeneral.org/news/press-release/Novel-software-tool-facilitates-inclusion-of-individuals-of-diverse-ancestry-in-large-scale-genetics-studies" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-020-00766-y" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Admixed populations are routinely excluded from genomic studies due to concerns over population structure. Here, we present a statistical framework and software package, Tractor, to facilitate the inclusion of admixed individuals in association studies by leveraging local ancestry. We test Tractor with simulated and empirical two-way admixed African-European cohorts. Tractor generates accurate ancestry-specific effect-size estimates and P values, can boost genome-wide association study (GWAS) power and improves the resolution of association signals. Using a local ancestry-aware regression model, we replicate known hits for blood lipids, discover novel hits missed by standard GWAS and localize signals closer to putative causal variants.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-020-00766-y" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2020">2020</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Chen2020" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.cell.2020.06.045" target="_blank" rel="noopener">Trans-ethnic and Ancestry-Specific Blood-Cell Genetics in 746,667 Individuals from 5 Global Populations</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Chen, M.-H.,
                    </span>
                    <span class="coauthor">
                      Raffield, L. M.,
                    </span>
                    <span class="coauthor">
                      Mousas, A.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Sakaue, S.,
                      Huffman, J. E.,
                      Moscati, A.,
                      Trivedi, B.,
                      Jiang, T.,
                      Akbari, P.,
                      Vuckovic, D.,
                      Bao, E. L.,
                      Zhong, X.,
                      Manansala, R.,
                      Laplante, V.,
                      Chen, M.,
                      Lo, K. S.,
                      Qian, H.,
                      Lareau, C. A.,
                      Beaudoin, M.,
                      Hunt, K. A.,
                      Akiyama, M.,
                      Bartz, T. M.,
                      Ben-Shlomo, Y.,
                      Beswick, A.,
                      Bork-Jensen, J.,
                      Bottinger, E. P.,
                      Brody, J. A.,
                      Rooij, F. J. A.,
                      Chitrala, K.,
                      Cho, K.,
                      Choquet, H.,
                      Correa, A.,
                      Danesh, J.,
                      Di Angelantonio, E.,
                      Dimou, N.,
                      Ding, J.,
                      Elliott, P.,
                      Esko, T.,
                      Evans, M. K.,
                      Floyd, J. S.,
                      Broer, L.,
                      Grarup, N.,
                      Guo, M. H.,
                      Greinacher, A.,
                      Haessler, J.,
                      Hansen, T.,
                      Howson, J. M. M.,
                      Huang, Q. Q.,
                      Huang, W.,
                      Jorgenson, E.,
                      Kacprowski, T.,
                      KÃ¤hÃ¶nen, M.,
                      Kamatani, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karthikeyan, S.,
                      Koskeridis, F.,
                      Lange, L. A.,
                      LehtimÃ¤ki, T.,
                      Lerch, M. M.,
                      Linneberg, A.,
                      Liu, Y.,
                      LyytikÃ¤inen, L.-P.,
                      Manichaikul, A.,
                      Martin, H. C.,
                      Matsuda, K.,
                      Mohlke, K. L.,
                      Mononen, N.,
                      Murakami, Y.,
                      Nadkarni, G. N.,
                      Nauck, M.,
                      Nikus, K.,
                      Ouwehand, W. H.,
                      Pankratz, N.,
                      Pedersen, O.,
                      Preuss, M.,
                      Psaty, B. M.,
                      Raitakari, O. T.,
                      Roberts, D. J.,
                      Rich, S. S.,
                      Rodriguez, B. A. T.,
                      Rosen, J. D.,
                      Rotter, J. I.,
                      Schubert, P.,
                      Spracklen, C. N.,
                      Surendran, P.,
                      Tang, H.,
                      Tardif, J.-C.,
                      Trembath, R. C.,
                      Ghanbari, M.,
                      VÃ¶lker, U.,
                      VÃ¶lzke, H.,
                      Watkins, N. A.,
                      Zonderman, A. B.,
                      Wilson, P. W. F.,
                      Li, Y.,
                      Butterworth, A. S.,
                      Gauchat, J.-F.,
                      Chiang, C. W. K.,
                      Li, B.,
                      Loos, R. J. F.,
                      Astle, W. J.,
                      Evangelou, E.,
                      Heel, D. A.,
                      Sankaran, V. G.,
                      Okada, Y.,
                      Soranzo, N.,
                      Johnson, A. D.,
                      Reiner, A. P.,
                      Auer, P. L.,
                    </span>
                    and Lettre, G.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell</em>
                    <b>182</b>,
                    1198â€“1213.e14
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32888493" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.01.17.910497" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://www.broadinstitute.org/news/genetic-information-can-predict-predisposition-rare-and-common-blood-diseases" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.cell.2020.06.045" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Most loci identified by GWASs have been found in populations of European ancestry (EUR). In trans-ethnic meta-analyses for 15 hematological traits in 746,667 participants, including 184,535 non-EUR individuals, we identified 5,552 trait-variant associations at p &lt; 5 \times 10-9, including 71 novel associations not found in EUR populations. We also identified 28 additional novel variants in ancestry-specific, non-EUR meta-analyses, including an IL7 missense variant in South Asians associated with lymphocyte count in vivo and IL-7 secretion levels in vitro. Fine-mapping prioritized variants annotated as functional and generated 95% credible sets that were 30% smaller when using the trans-ethnic as opposed to the EUR-only results. We explored the clinical significance and predictive value of trans-ethnic variants in multiple populations and compared genetic architecture and the effect of natural selection on these blood phenotypes between populations. Altogether, our results for hematological traits highlight the value of a more global representation of populations in genetic studies.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.cell.2020.06.045" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Vuckovic2020" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.cell.2020.08.008" target="_blank" rel="noopener">The Polygenic and Monogenic Basis of Blood Traits and Diseases</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Vuckovic, D.,
                    </span>
                    <span class="coauthor">
                      Bao, E. L.,
                    </span>
                    <span class="coauthor">
                      Akbari, P.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Lareau, C. A.,
                      Mousas, A.,
                      Jiang, T.,
                      Chen, M.-H.,
                      Raffield, L. M.,
                      Tardaguila, M.,
                      Huffman, J. E.,
                      Ritchie, S. C.,
                      Megy, K.,
                      Ponstingl, H.,
                      Penkett, C. J.,
                      Albers, P. K.,
                      Wigdor, E. M.,
                      Sakaue, S.,
                      Moscati, A.,
                      Manansala, R.,
                      Lo, K. S.,
                      Qian, H.,
                      Akiyama, M.,
                      Bartz, T. M.,
                      Ben-Shlomo, Y.,
                      Beswick, A.,
                      Bork-Jensen, J.,
                      Bottinger, E. P.,
                      Brody, J. A.,
                      Rooij, F. J. A.,
                      Chitrala, K. N.,
                      Wilson, P. W. F.,
                      Choquet, H.,
                      Danesh, J.,
                      Di Angelantonio, E.,
                      Dimou, N.,
                      Ding, J.,
                      Elliott, P.,
                      Esko, T.,
                      Evans, M. K.,
                      Felix, S. B.,
                      Floyd, J. S.,
                      Broer, L.,
                      Grarup, N.,
                      Guo, M. H.,
                      Guo, Q.,
                      Greinacher, A.,
                      Haessler, J.,
                      Hansen, T.,
                      Howson, J. M. M.,
                      Huang, W.,
                      Jorgenson, E.,
                      Kacprowski, T.,
                      KÃ¤hÃ¶nen, M.,
                      Kamatani, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Karthikeyan, S.,
                      Koskeridis, F.,
                      Lange, L. A.,
                      LehtimÃ¤ki, T.,
                      Linneberg, A.,
                      Liu, Y.,
                      LyytikÃ¤inen, L.-P.,
                      Manichaikul, A.,
                      Matsuda, K.,
                      Mohlke, K. L.,
                      Mononen, N.,
                      Murakami, Y.,
                      Nadkarni, G. N.,
                      Nikus, K.,
                      Pankratz, N.,
                      Pedersen, O.,
                      Preuss, M.,
                      Psaty, B. M.,
                      Raitakari, O. T.,
                      Rich, S. S.,
                      Rodriguez, B. A. T.,
                      Rosen, J. D.,
                      Rotter, J. I.,
                      Schubert, P.,
                      Spracklen, C. N.,
                      Surendran, P.,
                      Tang, H.,
                      Tardif, J.-C.,
                      Ghanbari, M.,
                      VÃ¶lker, U.,
                      VÃ¶lzke, H.,
                      Watkins, N. A.,
                      Weiss, S.,
                      Cai, N.,
                      Kundu, K.,
                      Watt, S. B.,
                      Walter, K.,
                      Zonderman, A. B.,
                      Cho, K.,
                      Li, Y.,
                      Loos, R. J. F.,
                      Knight, J. C.,
                      Georges, M.,
                      Stegle, O.,
                      Evangelou, E.,
                      Okada, Y.,
                      Roberts, D. J.,
                      Inouye, M.,
                      Johnson, A. D.,
                      Auer, P. L.,
                      Astle, W. J.,
                      Reiner, A. P.,
                      Butterworth, A. S.,
                      Ouwehand, W. H.,
                      Lettre, G.,
                      Sankaran, V. G.,
                    </span>
                    and Soranzo, N.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Cell</em>
                    <b>182</b>,
                    1214â€“1231.e11
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/action/showPdf?pii=S0092-8674%2820%2930999-5" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32888494" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/2020.02.02.20020065" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">medRxiv</a>
                    <a href="https://www.broadinstitute.org/news/genetic-information-can-predict-predisposition-rare-and-common-blood-diseases" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.cell.2020.08.008" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Blood cells play essential roles in human health, underpinning physiological processes such as immunity, oxygen transport, and clotting, which when perturbed cause a significant global health burden. Here we integrate data from UK Biobank and a large-scale international collaborative effort, including data for 563,085 European ancestry participants, and discover 5,106 new genetic variants independently associated with 29 blood cell phenotypes covering a range of variation impacting hematopoiesis. We holistically characterize the genetic architecture of hematopoiesis, assess the relevance of the omnigenic model to blood cell phenotypes, delineate relevant hematopoietic cell states influenced by regulatory genetic variants and gene networks, identify novel splice-altering variants mediating the associations, and assess the polygenic prediction potential for blood traits and clinical disorders at the interface of complex and Mendelian genetics. These results show the power of large-scale blood cell trait GWAS to interrogate clinically meaningful variants across a wide allelic spectrum of human variation.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.cell.2020.08.008" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Shirai2020" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1136/annrheumdis-2020-217256" target="_blank" rel="noopener">Association of the RPA3-UMAD1 locus with interstitial lung diseases complicated with rheumatoid arthritis in Japanese</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Shirai, Y.,
                    </span>
                    <span class="coauthor">
                      Honda, S.,
                    </span>
                    <span class="coauthor">
                      Ikari, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Takeda, Y.,
                      Kamatani, Y.,
                      Morisaki, T.,
                      Tanaka, E.,
                      Kumanogoh, A.,
                      Harigai, M.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Annals of the Rheumatic Diseases</em>
                    <b>79</b>,
                    1305â€“1309
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://ard.bmj.com/content/annrheumdis/79/10/1305.full.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32737115" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://www.amed.go.jp/news/release_20200801.html" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1136/annrheumdis-2020-217256" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>OBJECTIVES: The genetic background of rheumatoid arthritis-interstitial lung disease (RA-ILD) has been evaluated in Europeans, but little knowledge has been obtained in non-Europeans. This study aimed to elucidate genome-wide risk of RA-ILD in non-Europeans. METHODS: We performed an initial genome-wide association study (GWAS) of RA-ILD in the Japanese population. By conducting the meta-analysis of the three GWAS datasets of the RA cohorts and biobank of Japanese, our study included 358 RA-ILD cases and 4550 RA subjects without ILD. We then conducted the stratified analysis of the effect of the GWAS risk allele in each CT image pattern. RESULTS: We identified one novel RA-ILD risk locus at 7p21 that satisfied the genome-wide significance threshold (rs12702634 at RPA3-UMAD1, OR=2.04, 95% CI 1.59 to 2.60, p=1.5\times10-8). Subsequent stratified analysis based on the CT image patterns demonstrated that the effect size of the RA-ILD risk allele (rs12702634-C) was large with the UIP pattern (OR=1.86, 95% CI 0.97 to 3.58, p=0.062) and the probable UIP pattern (OR=2.26, 95% CI 1.36 to 3.73, p=0.0015). CONCLUSION: We revealed one novel genetic association with RA-ILD in Japanese. The RA-ILD risk of the identified variant at RPA3-UMAD1 was relatively high in the CT image patterns related to fibrosis. Our study should contribute to elucidation of the complicated aetiology of RA-ILD.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1136/annrheumdis-2020-217256" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Ishigaki2020" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-020-0640-3" target="_blank" rel="noopener">Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Ishigaki, K.,
                    </span>
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Takahashi, A.,
                      Kawakami, E.,
                      Sugishita, H.,
                      Sakaue, S.,
                      Matoba, N.,
                      Low, S.-K.,
                      Okada, Y.,
                      Terao, C.,
                      Amariuta, T.,
                      Gazal, S.,
                      Kochi, Y.,
                      Horikoshi, M.,
                      Suzuki, K.,
                      Ito, K.,
                      Koyama, S.,
                      Ozaki, K.,
                      Niida, S.,
                      Sakata, Y.,
                      Sakata, Y.,
                      Kohno, T.,
                      Shiraishi, K.,
                      Momozawa, Y.,
                      Hirata, M.,
                      Matsuda, K.,
                      Ikeda, M.,
                      Iwata, N.,
                      Ikegawa, S.,
                      Kou, I.,
                      Tanaka, T.,
                      Nakagawa, H.,
                      Suzuki, A.,
                      Hirota, T.,
                      Tamari, M.,
                      Chayama, K.,
                      Miki, D.,
                      Mori, M.,
                      Nagayama, S.,
                      Daigo, Y.,
                      Miki, Y.,
                      Katagiri, T.,
                      Ogawa, O.,
                      Obara, W.,
                      Ito, H.,
                      Yoshida, T.,
                      Imoto, I.,
                      Takahashi, T.,
                      Tanikawa, C.,
                      Suzuki, T.,
                      Sinozaki, N.,
                      Minami, S.,
                      Yamaguchi, H.,
                      Asai, S.,
                      Takahashi, Y.,
                      Yamaji, K.,
                      Takahashi, K.,
                      Fujioka, T.,
                      Takata, R.,
                      Yanai, H.,
                      Masumoto, A.,
                      Koretsune, Y.,
                      Kutsumi, H.,
                      Higashiyama, M.,
                      Murayama, S.,
                      Minegishi, N.,
                      Suzuki, K.,
                      Tanno, K.,
                      Shimizu, A.,
                      Yamaji, T.,
                      Iwasaki, M.,
                      Sawada, N.,
                      Uemura, H.,
                      Tanaka, K.,
                      Naito, M.,
                      Sasaki, M.,
                      Wakai, K.,
                      Tsugane, S.,
                      Yamamoto, M.,
                      Yamamoto, K.,
                      Murakami, Y.,
                      Nakamura, Y.,
                      Raychaudhuri, S.,
                      Inazawa, J.,
                      Yamauchi, T.,
                      Kadowaki, T.,
                      Kubo, M.,
                    </span>
                    and Kamatani, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>52</b>,
                    669â€“679
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmhb1" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32514122" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/795948" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://www.riken.jp/press/2020/20200609_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-020-0640-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The overwhelming majority of participants in current genetic studies are of European ancestry. To elucidate disease biology in the East Asian population, we conducted a genome-wide association study (GWAS) with 212,453 Japanese individuals across 42 diseases. We detected 320 independent signals in 276 loci for 27 diseases, with 25 novel loci (P &lt; 9.58 \times 10-9). East Asian-specific missense variants were identified as candidate causal variants for three novel loci, and we successfully replicated two of them by analyzing independent Japanese cohorts; p.R220W of ATG16L2 (associated with coronary artery disease) and p.V326A of POT1 (associated with lung cancer). We further investigated enrichment of heritability within 2,868 annotations of genome-wide transcription factor occupancy, and identified 378 significant enrichments across nine diseases (false discovery rate &lt; 0.05) (for example, NKX3-1 for prostate cancer). This large-scale GWAS in a Japanese population provides insights into the etiology of complex diseases and highlights the importance of performing GWAS in non-European populations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-020-0640-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sakaue2020b" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-020-15194-z" target="_blank" rel="noopener">Dimensionality reduction reveals fine-scale structure in the Japanese population with consequences for polygenic risk prediction</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Sakaue, S.,
                    </span>
                    <span class="coauthor">
                      Hirata, J.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Suzuki, K.,
                      Akiyama, M.,
                      Lai Too, C.,
                      Arayssi, T.,
                      Hammoudeh, M.,
                      Al Emadi, S.,
                      Masri, B. K.,
                      Halabi, H.,
                      Badsha, H.,
                      Uthman, I. W.,
                      Saxena, R.,
                      Padyukov, L.,
                      Hirata, M.,
                      Matsuda, K.,
                      Murakami, Y.,
                      Kamatani, Y.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>11</b>,
                    1569
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmhcB" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32218440" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://github.com/saorisakaue/Genotype-dimensionality-reduction" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="http://www.med.osaka-u.ac.jp/eng/activities/results/2020year/okadasakaue20200327" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-020-15194-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The diversity in our genome is crucial to understanding the demographic history of worldwide populations. However, we have yet to know whether subtle genetic differences within a population can be disentangled, or whether they have an impact on complex traits. Here we apply dimensionality reduction methods (PCA, t-SNE, PCA-t-SNE, UMAP, and PCA-UMAP) to biobank-derived genomic data of a Japanese population (n = 169,719). Dimensionality reduction reveals fine-scale population structure, conspicuously differentiating adjacent insular subpopulations. We further enluciate the demographic landscape of these Japanese subpopulations using population genetics analyses. Finally, we perform phenome-wide polygenic risk score (PRS) analyses on 67 complex traits. Differences in PRS between the deconvoluted subpopulations are not always concordant with those in the observed phenotypes, suggesting that the PRS differences might reflect biases from the uncorrected structure, in a trait-dependent manner. This study suggests that such an uncorrected structure can be a potential pitfall in the clinical application of PRS.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-020-15194-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sakaue2020a" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41591-020-0785-8" target="_blank" rel="noopener">Trans-biobank analysis with 676,000 individuals elucidates the association of polygenic risk scores of complex traits with human lifespan</a></div>
                  <div class="author">
                    <span class="coauthor">
                      *Sakaue, S.,
                    </span>
                    <em>*Kanai, M.</em>,
                    <span class="coauthor">
                      Karjalainen, J.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Akiyama, M.,
                      Kurki, M.,
                      Matoba, N.,
                      Takahashi, A.,
                      Hirata, M.,
                      Kubo, M.,
                      Matsuda, K.,
                      Murakami, Y.,
                      Daly, M. J.,
                      Kamatani, Y.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Medicine</em>
                    <b>26</b>,
                    542â€“548
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmhcK" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32251405" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/856351" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="http://www.med.osaka-u.ac.jp/eng/activities/results/2020year/okada20200324" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41591-020-0785-8" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>While polygenic risk scores (PRSs) are poised to be translated into clinical practice through prediction of inborn health risks1, a strategy to utilize genetics to prioritize modifiable risk factors driving heath outcome is warranted2. To this end, we investigated the association of the genetic susceptibility to complex traits with human lifespan in collaboration with three worldwide biobanks (ntotal = 675,898; BioBank Japan (n = 179,066), UK Biobank (n = 361,194) and FinnGen (n = 135,638)). In contrast to observational studies, in which discerning the cause-and-effect can be difficult, PRSs could help to identify the driver biomarkers affecting human lifespan. A high systolic blood pressure PRS was trans-ethnically associated with a shorter lifespan (hazard ratio = 1.03[1.02-1.04], Pmeta = 3.9 \times 10-13) and parental lifespan (hazard ratio = 1.06[1.06-1.07], P = 2.0 \times 10-86). The obesity PRS showed distinct effects on lifespan in Japanese and European individuals (Pheterogeneity = 9.5 \times 10-8 for BMI). The causal effect of blood pressure and obesity on lifespan was further supported by Mendelian randomization studies. Beyond genotype-phenotype associations, our trans-biobank study offers a new value of PRSs in prioritization of risk factors that could be potential targets of medical treatment to improve population health.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41591-020-0785-8" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Ray2020" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-020-15022-4" target="_blank" rel="noopener">Prioritizing disease and trait causal variants at the TNFAIP3 locus using functional and genomic features</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Ray, J. P.,
                    </span>
                    <span class="coauthor">
                      Boer, C. G.,
                    </span>
                    <span class="coauthor">
                      Fulco, C. P.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Lareau, C. A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ulirsch, J. C.,
                      Tewhey, R.,
                      Ludwig, L. S.,
                      Reilly, S. K.,
                      Bergman, D. T.,
                      Engreitz, J. M.,
                      Issner, R.,
                      Finucane, H. K.,
                      Lander, E. S.,
                      Regev, A.,
                    </span>
                    and Hacohen, N.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>11</b>,
                    1237
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmhcT" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/32144282" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-020-15022-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Genome-wide association studies have associated thousands of genetic variants with complex traits and diseases, but pinpointing the causal variant(s) among those in tight linkage disequilibrium with each associated variant remains a major challenge. Here, we use seven experimental assays to characterize all common variants at the multiple disease-associated TNFAIP3 locus in five disease-relevant immune cell lines, based on a set of features related to regulatory potential. Trait/disease-associated variants are enriched among SNPs prioritized based on either: (1) residing within CRISPRi-sensitive regulatory regions, or (2) localizing in a chromatin accessible region while displaying allele-specific reporter activity. Of the 15 trait/disease-associated haplotypes at TNFAIP3, 9 have at least one variant meeting one or both of these criteria, 5 of which are further supported by genetic fine-mapping. Our work provides a comprehensive strategy to characterize genetic variation at important disease-associated loci, and aids in the effort to identify trait causal genetic variants.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-020-15022-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Matoba2020" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41562-019-0805-1" target="_blank" rel="noopener">GWAS of 165,084 Japanese individuals identified nine loci associated with dietary habits</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Matoba, N.,
                    </span>
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <span class="coauthor">
                      Ishigaki, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Takahashi, A.,
                      Momozawa, Y.,
                      Ikegawa, S.,
                      Ikeda, M.,
                      Iwata, N.,
                      Hirata, M.,
                      Matsuda, K.,
                      Murakami, Y.,
                      Kubo, M.,
                      Kamatani, Y.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Human Behaviour</em>
                    <b>4</b>,
                    308â€“316
                    (2020)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmhds" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31959922" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://www.riken.jp/press/2020/20200130_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41562-019-0805-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Dietary habits are important factors in our lifestyle, and confer both susceptibility to and protection from a variety of human diseases. We performed genome-wide association studies for 13 dietary habits including consumption of alcohol (ever versus never drinkers and drinks per week), beverages (coffee, green tea and milk) and foods (yoghurt, cheese, natto, tofu, fish, small whole fish, vegetables and meat) in Japanese individuals (n = 58,610-165,084) collected by BioBank Japan, the nationwide hospital-based genome cohort. Significant associations were found in nine genetic loci (MCL1-ENSA, GCKR, AGR3-AHR, ADH1B, ALDH1B1, ALDH1A1, ALDH2, CYP1A2-CSK and ADORA2A-AS1) for 13 dietary traits (P &lt; 3.8 \times 10-9). Of these, ten associations between five loci and eight traits were new findings. Furthermore, a phenome-wide association study revealed that five of the dietary trait-associated loci have pleiotropic effects on multiple human complex diseases and clinical measurements. Our findings provide new insight into the genetics of habitual consumption.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41562-019-0805-1" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2019">2019</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Clark2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-019-12283-6" target="_blank" rel="noopener">Associations of autozygosity with a broad range of human phenotypes</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Clark, D. W.,
                    </span>
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <span class="coauthor">
                      Moore, K. H. S.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Mason, D.,
                      Pirastu, N.,
                      Gandin, I.,
                      Mattsson, H.,
                      Barnes, C. L. K.,
                      Lin, K.,
                      Zhao, J. H.,
                      Deelen, P.,
                      Rohde, R.,
                      Schurmann, C.,
                      Guo, X.,
                      Giulianini, F.,
                      Zhang, W.,
                      Medina-Gomez, C.,
                      Karlsson, R.,
                      Bao, Y.,
                      Bartz, T. M.,
                      Baumbach, C.,
                      Biino, G.,
                      Bixley, M. J.,
                      Brumat, M.,
                      Chai, J.-F.,
                      Corre, T.,
                      Cousminer, D. L.,
                      Dekker, A. M.,
                      Eccles, D. A.,
                      Eijk, K. R.,
                      Fuchsberger, C.,
                      Gao, H.,
                      Germain, M.,
                      Gordon, S. D.,
                      Haan, H. G.,
                      Harris, S. E.,
                      Hofer, E.,
                      Huerta-Chagoya, A.,
                      Igartua, C.,
                      Jansen, I. E.,
                      Jia, Y.,
                      Kacprowski, T.,
                      Karlsson, T.,
                      Kleber, M. E.,
                      Li, S. A.,
                      Li-Gao, R.,
                      Mahajan, A.,
                      Matsuda, K.,
                      Meidtner, K.,
                      Meng, W.,
                      Montasser, M. E.,
                      Most, P. J.,
                      Munz, M.,
                      Nutile, T.,
                      Palviainen, T.,
                      Prasad, G.,
                      Prasad, R. B.,
                      Priyanka, T. D. S.,
                      Rizzi, F.,
                      Salvi, E.,
                      Sapkota, B. R.,
                      Shriner, D.,
                      Skotte, L.,
                      Smart, M. C.,
                      Smith, A. V.,
                      Spek, A.,
                      Spracklen, C. N.,
                      Strawbridge, R. J.,
                      Tajuddin, S. M.,
                      Trompet, S.,
                      Turman, C.,
                      Verweij, N.,
                      Viberti, C.,
                      Wang, L.,
                      Warren, H. R.,
                      Wootton, R. E.,
                      Yanek, L. R.,
                      Yao, J.,
                      Yousri, N. A.,
                      Zhao, W.,
                      Adeyemo, A. A.,
                      Afaq, S.,
                      Aguilar-Salinas, C. A.,
                      Akiyama, M.,
                      Albert, M. L.,
                      Allison, M. A.,
                      Alver, M.,
                      Aung, T.,
                      Azizi, F.,
                      Bentley, A. R.,
                      Boeing, H.,
                      Boerwinkle, E.,
                      Borja, J. B.,
                      Borst, G. J.,
                      Bottinger, E. P.,
                      Broer, L.,
                      Campbell, H.,
                      Chanock, S.,
                      Chee, M.-L.,
                      Chen, G.,
                      Chen, Y.-D. I.,
                      Chen, Z.,
                      Chiu, Y.-F.,
                      Cocca, M.,
                      Collins, F. S.,
                      Concas, M. P.,
                      Corley, J.,
                      Cugliari, G.,
                      Dam, R. M.,
                      Damulina, A.,
                      Daneshpour, M. S.,
                      Day, F. R.,
                      Delgado, G. E.,
                      Dhana, K.,
                      Doney, A. S. F.,
                      DÃ¶rr, M.,
                      Doumatey, A. P.,
                      Dzimiri, N.,
                      EbenesersdÃ³ttir, S. S.,
                      Elliott, J.,
                      Elliott, P.,
                      Ewert, R.,
                      Felix, J. F.,
                      Fischer, K.,
                      Freedman, B. I.,
                      Girotto, G.,
                      Goel, A.,
                      GÃ¶gele, M.,
                      Goodarzi, M. O.,
                      Graff, M.,
                      Granot-Hershkovitz, E.,
                      Grodstein, F.,
                      Guarrera, S.,
                      Gudbjartsson, D. F.,
                      Guity, K.,
                      Gunnarsson, B.,
                      Guo, Y.,
                      Hagenaars, S. P.,
                      Haiman, C. A.,
                      Halevy, A.,
                      Harris, T. B.,
                      Hedayati, M.,
                      Heel, D. A.,
                      Hirata, M.,
                      HÃ¶fer, I.,
                      Hsiung, C. A.,
                      Huang, J.,
                      Hung, Y.-J.,
                      Ikram, M. A.,
                      Jagadeesan, A.,
                      Jousilahti, P.,
                      Kamatani, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kerrison, N. D.,
                      Kessler, T.,
                      Khaw, K.-T.,
                      Khor, C. C.,
                      Kleijn, D. P. V.,
                      Koh, W.-P.,
                      Kolcic, I.,
                      Kraft, P.,
                      KrÃ¤mer, B. K.,
                      Kutalik, Z.,
                      Kuusisto, J.,
                      Langenberg, C.,
                      Launer, L. J.,
                      Lawlor, D. A.,
                      Lee, I.-T.,
                      Lee, W.-J.,
                      Lerch, M. M.,
                      Li, L.,
                      Liu, J.,
                      Loh, M.,
                      London, S. J.,
                      Loomis, S.,
                      Lu, Y.,
                      Luan, J.,
                      MÃ¤gi, R.,
                      Manichaikul, A. W.,
                      Manunta, P.,
                      MÃ¡sson, G.,
                      Matoba, N.,
                      Mei, X. W.,
                      Meisinger, C.,
                      Meitinger, T.,
                      Mezzavilla, M.,
                      Milani, L.,
                      Millwood, I. Y.,
                      Momozawa, Y.,
                      Moore, A.,
                      Morange, P.-E.,
                      Moreno-MacÃ­as, H.,
                      Mori, T. A.,
                      Morrison, A. C.,
                      Muka, T.,
                      Murakami, Y.,
                      Murray, A. D.,
                      Mutsert, R.,
                      Mychaleckyj, J. C.,
                      Nalls, M. A.,
                      Nauck, M.,
                      Neville, M. J.,
                      Nolte, I. M.,
                      Ong, K. K.,
                      Orozco, L.,
                      Padmanabhan, S.,
                      PÃ¡lsson, G.,
                      Pankow, J. S.,
                      Pattaro, C.,
                      Pattie, A.,
                      Polasek, O.,
                      Poulter, N.,
                      Pramstaller, P. P.,
                      Quintana-Murci, L.,
                      RÃ¤ikkÃ¶nen, K.,
                      Ralhan, S.,
                      Rao, D. C.,
                      Rheenen, W.,
                      Rich, S. S.,
                      Ridker, P. M.,
                      Rietveld, C. A.,
                      Robino, A.,
                      Rooij, F. J. A.,
                      Ruggiero, D.,
                      Saba, Y.,
                      Sabanayagam, C.,
                      Sabater-Lleal, M.,
                      Sala, C. F.,
                      Salomaa, V.,
                      Sandow, K.,
                      Schmidt, H.,
                      Scott, L. J.,
                      Scott, W. R.,
                      Sedaghati-Khayat, B.,
                      Sennblad, B.,
                      Setten, J.,
                      Sever, P. J.,
                      Sheu, W. H.-H.,
                      Shi, Y.,
                      Shrestha, S.,
                      Shukla, S. R.,
                      Sigurdsson, J. K.,
                      Sikka, T. T.,
                      Singh, J. R.,
                      Smith, B. H.,
                      StanÄÃ¡kovÃ¡, A.,
                      Stanton, A.,
                      Starr, J. M.,
                      Stefansdottir, L.,
                      Straker, L.,
                      Sulem, P.,
                      Sveinbjornsson, G.,
                      Swertz, M. A.,
                      Taylor, A. M.,
                      Taylor, K. D.,
                      Terzikhan, N.,
                      Tham, Y.-C.,
                      Thorleifsson, G.,
                      Thorsteinsdottir, U.,
                      Tillander, A.,
                      Tracy, R. P.,
                      TusiÃ©-Luna, T.,
                      Tzoulaki, I.,
                      Vaccargiu, S.,
                      Vangipurapu, J.,
                      Veldink, J. H.,
                      Vitart, V.,
                      VÃ¶lker, U.,
                      Vuoksimaa, E.,
                      Wakil, S. M.,
                      Waldenberger, M.,
                      Wander, G. S.,
                      Wang, Y. X.,
                      Wareham, N. J.,
                      Wild, S.,
                      Yajnik, C. S.,
                      Yuan, J.-M.,
                      Zeng, L.,
                      Zhang, L.,
                      Zhou, J.,
                      Amin, N.,
                      Asselbergs, F. W.,
                      Bakker, S. J. L.,
                      Becker, D. M.,
                      Lehne, B.,
                      Bennett, D. A.,
                      Berg, L. H.,
                      Berndt, S. I.,
                      Bharadwaj, D.,
                      Bielak, L. F.,
                      Bochud, M.,
                      Boehnke, M.,
                      Bouchard, C.,
                      Bradfield, J. P.,
                      Brody, J. A.,
                      Campbell, A.,
                      Carmi, S.,
                      Caulfield, M. J.,
                      Cesarini, D.,
                      Chambers, J. C.,
                      Chandak, G. R.,
                      Cheng, C.-Y.,
                      Ciullo, M.,
                      Cornelis, M.,
                      Cusi, D.,
                      Smith, G. D.,
                      Deary, I. J.,
                      Dorajoo, R.,
                      Duijn, C. M.,
                      Ellinghaus, D.,
                      Erdmann, J.,
                      Eriksson, J. G.,
                      Evangelou, E.,
                      Evans, M. K.,
                      Faul, J. D.,
                      Feenstra, B.,
                      Feitosa, M.,
                      Foisy, S.,
                      Franke, A.,
                      Friedlander, Y.,
                      Gasparini, P.,
                      Gieger, C.,
                      Gonzalez, C.,
                      Goyette, P.,
                      Grant, S. F. A.,
                      Griffiths, L. R.,
                      Groop, L.,
                      Gudnason, V.,
                      Gyllensten, U.,
                      Hakonarson, H.,
                      Hamsten, A.,
                      Harst, P.,
                      Heng, C.-K.,
                      Hicks, A. A.,
                      Hochner, H.,
                      Huikuri, H.,
                      Hunt, S. C.,
                      Jaddoe, V. W. V.,
                      De Jager, P. L.,
                      Johannesson, M.,
                      Johansson, Ã….,
                      Jonas, J. B.,
                      Jukema, J. W.,
                      Junttila, J.,
                      Kaprio, J.,
                      Kardia, S. L. R.,
                      Karpe, F.,
                      Kumari, M.,
                      Laakso, M.,
                      Laan, S. W.,
                      Lahti, J.,
                      Laudes, M.,
                      Lea, R. A.,
                      Lieb, W.,
                      Lumley, T.,
                      Martin, N. G.,
                      MÃ¤rz, W.,
                      Matullo, G.,
                      McCarthy, M. I.,
                      Medland, S. E.,
                      Merriman, T. R.,
                      Metspalu, A.,
                      Meyer, B. F.,
                      Mohlke, K. L.,
                      Montgomery, G. W.,
                      Mook-Kanamori, D.,
                      Munroe, P. B.,
                      North, K. E.,
                      Nyholt, D. R.,
                      Oâ€™connell, J. R.,
                      Ober, C.,
                      Oldehinkel, A. J.,
                      Palmas, W.,
                      Palmer, C.,
                      Pasterkamp, G. G.,
                      Patin, E.,
                      Pennell, C. E.,
                      Perusse, L.,
                      Peyser, P. A.,
                      Pirastu, M.,
                      Polderman, T. J. C.,
                      Porteous, D. J.,
                      Posthuma, D.,
                      Psaty, B. M.,
                      Rioux, J. D.,
                      Rivadeneira, F.,
                      Rotimi, C.,
                      Rotter, J. I.,
                      Rudan, I.,
                      Den Ruijter, H. M.,
                      Sanghera, D. K.,
                      Sattar, N.,
                      Schmidt, R.,
                      Schulze, M. B.,
                      Schunkert, H.,
                      Scott, R. A.,
                      Shuldiner, A. R.,
                      Sim, X.,
                      Small, N.,
                      Smith, J. A.,
                      Sotoodehnia, N.,
                      Tai, E.-S.,
                      Teumer, A.,
                      Timpson, N. J.,
                      Toniolo, D.,
                      Tregouet, D.-A.,
                      Tuomi, T.,
                      Vollenweider, P.,
                      Wang, C. A.,
                      Weir, D. R.,
                      Whitfield, J. B.,
                      Wijmenga, C.,
                      Wong, T.-Y.,
                      Wright, J.,
                      Yang, J.,
                      Yu, L.,
                      Zemel, B. S.,
                      Zonderman, A. B.,
                      Perola, M.,
                      Magnusson, P. K. E.,
                      Uitterlinden, A. G.,
                      Kooner, J. S.,
                      Chasman, D. I.,
                      Loos, R. J. F.,
                      Franceschini, N.,
                      Franke, L.,
                      Haley, C. S.,
                      Hayward, C.,
                      Walters, R. G.,
                      Perry, J. R. B.,
                      Esko, T.,
                      Helgason, A.,
                      Stefansson, K.,
                      Joshi, P. K.,
                      Kubo, M.,
                    </span>
                    and Wilson, J. F.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>10</b>,
                    4957
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqTL" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31673082" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-019-12283-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>In many species, the offspring of related parents suffer reduced reproductive success, a phenomenon known as inbreeding depression. In humans, the importance of this effect has remained unclear, partly because reproduction between close relatives is both rare and frequently associated with confounding social factors. Here, using genomic inbreeding coefficients (FROH) for &gt;1.4 million individuals, we show that FROH is significantly associated (p &lt; 0.0005) with apparently deleterious changes in 32 out of 100 traits analysed. These changes are associated with runs of homozygosity (ROH), but not with common variant homozygosity, suggesting that genetic variants associated with inbreeding depression are predominantly rare. The effect on fertility is striking: FROH equivalent to the offspring of first cousins is associated with a 55% decrease [95% CI 44-66%] in the odds of having children. Finally, the effects of FROH are confirmed within full-sibling pairs, where the variation in FROH is independent of all environmental confounding.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-019-12283-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Leu2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1093/brain/awz292" target="_blank" rel="noopener">Polygenic burden in focal and generalized epilepsies</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Leu, C.,
                    </span>
                    <span class="coauthor">
                      Stevelink, R.,
                    </span>
                    <span class="coauthor">
                      Smith, A. W.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Goleva, S. B.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ferguson, L.,
                      Campbell, C.,
                      Kamatani, Y.,
                      Okada, Y.,
                      Sisodiya, S. M.,
                      Cavalleri, G. L.,
                      Koeleman, B. P. C.,
                      Lerche, H.,
                      Jehi, L.,
                      Davis, L. K.,
                      Najm, I. M.,
                      Palotie, A.,
                      Daly, M. J.,
                      Busch, R. M.,
                    </span>
                    and Lal, D.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Brain</em>
                    <b>142</b>,
                    3473â€“3481
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://academic.oup.com/brain/article-pdf/142/11/3473/35377751/awz292.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31608925" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1093/brain/awz292" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Rare genetic variants can cause epilepsy, and genetic testing has been widely adopted for severe, paediatric-onset epilepsies. The phenotypic consequences of common genetic risk burden for epilepsies and their potential future clinical applications have not yet been determined. Using polygenic risk scores (PRS) from a European-ancestry genome-wide association study in generalized and focal epilepsy, we quantified common genetic burden in patients with generalized epilepsy (GE-PRS) or focal epilepsy (FE-PRS) from two independent non-Finnish European cohorts (Epi25 Consortium, n = 5705; Cleveland Clinic Epilepsy Center, n = 620; both compared to 20 435 controls). One Finnish-ancestry population isolate (Finnish-ancestry Epi25, n = 449; compared to 1559 controls), two European-ancestry biobanks (UK Biobank, n = 383 656; Vanderbilt biorepository, n = 49 494), and one Japanese-ancestry biobank (BioBank Japan, n = 168 680) were used for additional replications. Across 8386 patients with epilepsy and 622 212 population controls, we found and replicated significantly higher GE-PRS in patients with generalized epilepsy of European-ancestry compared to patients with focal epilepsy (Epi25: P = 1.64\times10-15; Cleveland: P = 2.85\times10-4; Finnish-ancestry Epi25: P = 1.80\times10-4) or population controls (Epi25: P = 2.35\times10-70; Cleveland: P = 1.43\times10-7; Finnish-ancestry Epi25: P = 3.11\times10-4; UK Biobank and Vanderbilt biorepository meta-analysis: P = 7.99\times10-4). FE-PRS were significantly higher in patients with focal epilepsy compared to controls in the non-Finnish, non-biobank cohorts (Epi25: P = 5.74\times10-19; Cleveland: P = 1.69\times10-6). European ancestry-derived PRS did not predict generalized epilepsy or focal epilepsy in Japanese-ancestry individuals. Finally, we observed a significant 4.6-fold and a 4.5-fold enrichment of patients with generalized epilepsy compared to controls in the top 0.5% highest GE-PRS of the two non-Finnish European cohorts (Epi25: P = 2.60\times10-15; Cleveland: P = 1.39\times10-2). We conclude that common variant risk associated with epilepsy is significantly enriched in multiple cohorts of patients with epilepsy compared to controls-in particular for generalized epilepsy. As sample sizes and PRS accuracy continue to increase with further common variant discovery, PRS could complement established clinical biomarkers and augment genetic testing for patient classification, comorbidity research, and potentially targeted treatment.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1093/brain/awz292" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Tin2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-019-0504-x" target="_blank" rel="noopener">Target genes, variants, tissues and transcriptional pathways influencing human serum urate levels</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Tin, A.,
                    </span>
                    <span class="coauthor">
                      Marten, J.,
                    </span>
                    <span class="coauthor">
                      Halperin Kuhns, V. L.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Li, Y.,
                      Wuttke, M.,
                      Kirsten, H.,
                      Sieber, K. B.,
                      Qiu, C.,
                      Gorski, M.,
                      Yu, Z.,
                      Giri, A.,
                      Sveinbjornsson, G.,
                      Li, M.,
                      Chu, A. Y.,
                      Hoppmann, A.,
                      Oâ€™Connor, L. J.,
                      Prins, B.,
                      Nutile, T.,
                      Noce, D.,
                      Akiyama, M.,
                      Cocca, M.,
                      Ghasemi, S.,
                      Most, P. J.,
                      Horn, K.,
                      Xu, Y.,
                      Fuchsberger, C.,
                      Sedaghat, S.,
                      Afaq, S.,
                      Amin, N.,
                      Ã„rnlÃ¶v, J.,
                      Bakker, S. J. L.,
                      Bansal, N.,
                      Baptista, D.,
                      Bergmann, S.,
                      Biggs, M. L.,
                      Biino, G.,
                      Boerwinkle, E.,
                      Bottinger, E. P.,
                      Boutin, T. S.,
                      Brumat, M.,
                      Burkhardt, R.,
                      Campana, E.,
                      Campbell, A.,
                      Campbell, H.,
                      Carroll, R. J.,
                      Catamo, E.,
                      Chambers, J. C.,
                      Ciullo, M.,
                      Concas, M. P.,
                      Coresh, J.,
                      Corre, T.,
                      Cusi, D.,
                      Felicita, S. C.,
                      Borst, M. H.,
                      De Grandi, A.,
                      Mutsert, R.,
                      Vries, A. P. J.,
                      Delgado, G.,
                      Demirkan, A.,
                      Devuyst, O.,
                      Dittrich, K.,
                      Eckardt, K.-U.,
                      Ehret, G.,
                      Endlich, K.,
                      Evans, M. K.,
                      Gansevoort, R. T.,
                      Gasparini, P.,
                      Giedraitis, V.,
                      Gieger, C.,
                      Girotto, G.,
                      GÃ¶gele, M.,
                      Gordon, S. D.,
                      Gudbjartsson, D. F.,
                      Gudnason, V.,
                      Haller, T.,
                      Hamet, P.,
                      Harris, T. B.,
                      Hayward, C.,
                      Hicks, A. A.,
                      Hofer, E.,
                      Holm, H.,
                      Huang, W.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Hwang, S.-J.,
                      Ikram, M. A.,
                      Lewis, R. M.,
                      Ingelsson, E.,
                      Jakobsdottir, J.,
                      Jonsdottir, I.,
                      Jonsson, H.,
                      Joshi, P. K.,
                      Josyula, N. S.,
                      Jung, B.,
                      KÃ¤hÃ¶nen, M.,
                      Kamatani, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kerr, S. M.,
                      Kiess, W.,
                      Kleber, M. E.,
                      Koenig, W.,
                      Kooner, J. S.,
                      KÃ¶rner, A.,
                      Kovacs, P.,
                      KrÃ¤mer, B. K.,
                      Kronenberg, F.,
                      Kubo, M.,
                      KÃ¼hnel, B.,
                      La Bianca, M.,
                      Lange, L. A.,
                      Lehne, B.,
                      LehtimÃ¤ki, T.,
                      Liu, J.,
                      Loeffler, M.,
                      Loos, R. J. F.,
                      LyytikÃ¤inen, L.-P.,
                      Magi, R.,
                      Mahajan, A.,
                      Martin, N. G.,
                      MÃ¤rz, W.,
                      Mascalzoni, D.,
                      Matsuda, K.,
                      Meisinger, C.,
                      Meitinger, T.,
                      Metspalu, A.,
                      Milaneschi, Y.,
                      Oâ€™Donnell, C. J.,
                      Wilson, O. D.,
                      Gaziano, J. M.,
                      Mishra, P. P.,
                      Mohlke, K. L.,
                      Mononen, N.,
                      Montgomery, G. W.,
                      Mook-Kanamori, D. O.,
                      MÃ¼ller-Nurasyid, M.,
                      Nadkarni, G. N.,
                      Nalls, M. A.,
                      Nauck, M.,
                      Nikus, K.,
                      Ning, B.,
                      Nolte, I. M.,
                      Noordam, R.,
                      Oâ€™Connell, J. R.,
                      Olafsson, I.,
                      Padmanabhan, S.,
                      Penninx, B. W. J. H.,
                      Perls, T.,
                      Peters, A.,
                      Pirastu, M.,
                      Pirastu, N.,
                      Pistis, G.,
                      Polasek, O.,
                      Ponte, B.,
                      Porteous, D. J.,
                      Poulain, T.,
                      Preuss, M. H.,
                      Rabelink, T. J.,
                      Raffield, L. M.,
                      Raitakari, O. T.,
                      Rettig, R.,
                      Rheinberger, M.,
                      Rice, K. M.,
                      Rizzi, F.,
                      Robino, A.,
                      Rudan, I.,
                      Krajcoviechova, A.,
                      Cifkova, R.,
                      Rueedi, R.,
                      Ruggiero, D.,
                      Ryan, K. A.,
                      Saba, Y.,
                      Salvi, E.,
                      Schmidt, H.,
                      Schmidt, R.,
                      Shaffer, C. M.,
                      Smith, A. V.,
                      Smith, B. H.,
                      Spracklen, C. N.,
                      Strauch, K.,
                      Stumvoll, M.,
                      Sulem, P.,
                      Tajuddin, S. M.,
                      Teren, A.,
                      Thiery, J.,
                      Thio, C. H. L.,
                      Thorsteinsdottir, U.,
                      Toniolo, D.,
                      TÃ¶njes, A.,
                      Tremblay, J.,
                      Uitterlinden, A. G.,
                      Vaccargiu, S.,
                      Harst, P.,
                      Duijn, C. M.,
                      Verweij, N.,
                      VÃ¶lker, U.,
                      Vollenweider, P.,
                      Waeber, G.,
                      Waldenberger, M.,
                      Whitfield, J. B.,
                      Wild, S. H.,
                      Wilson, J. F.,
                      Yang, Q.,
                      Zhang, W.,
                      Zonderman, A. B.,
                      Bochud, M.,
                      Wilson, J. G.,
                      Pendergrass, S. A.,
                      Ho, K.,
                      Parsa, A.,
                      Pramstaller, P. P.,
                      Psaty, B. M.,
                      BÃ¶ger, C. A.,
                      Snieder, H.,
                      Butterworth, A. S.,
                      Okada, Y.,
                      Edwards, T. L.,
                      Stefansson, K.,
                      Susztak, K.,
                      Scholz, M.,
                      Heid, I. M.,
                      Hung, A. M.,
                      Teumer, A.,
                      Pattaro, C.,
                      Woodward, O. M.,
                      Vitart, V.,
                    </span>
                    and KÃ¶ttgen, A.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>51</b>,
                    1459â€“1474
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqTP" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31578528" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-019-0504-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Elevated serum urate levels cause gout and correlate with cardiometabolic diseases via poorly understood mechanisms. We performed a trans-ancestry genome-wide association study of serum urate in 457,690 individuals, identifying 183 loci (147 previously unknown) that improve the prediction of gout in an independent cohort of 334,880 individuals. Serum urate showed significant genetic correlations with many cardiometabolic traits, with genetic causality analyses supporting a substantial role for pleiotropy. Enrichment analysis, fine-mapping of urate-associated loci and colocalization with gene expression in 47 tissues implicated the kidney and liver as the main target organs and prioritized potentially causal genes and variants, including the transcriptional master regulators in the liver and kidney, HNF1A and HNF4A. Experimental validation showed that HNF4A transactivated the promoter of ABCG2, encoding a major urate transporter, in kidney cells, and that HNF4A p.Thr139Ile is a functional variant. Transcriptional coregulation within and across organs may be a general mechanism underlying the observed pleiotropy between urate and cardiometabolic traits.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-019-0504-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Akiyama2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-019-12276-5" target="_blank" rel="noopener">Characterizing rare and low-frequency height-associated variants in the Japanese population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <span class="coauthor">
                      Ishigaki, K.,
                    </span>
                    <span class="coauthor">
                      Sakaue, S.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Momozawa, Y.,
                      Horikoshi, M.,
                      Hirata, M.,
                      Matsuda, K.,
                      Ikegawa, S.,
                      Takahashi, A.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Suzuki, S.,
                      Matsui, D.,
                      Naito, M.,
                      Yamaji, T.,
                      Iwasaki, M.,
                      Sawada, N.,
                      Tanno, K.,
                      Sasaki, M.,
                      Hozawa, A.,
                      Minegishi, N.,
                      Wakai, K.,
                      Tsugane, S.,
                      Shimizu, A.,
                      Yamamoto, M.,
                      Okada, Y.,
                      Murakami, Y.,
                      Kubo, M.,
                    </span>
                    and Kamatani, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>10</b>,
                    4393
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqTX" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31562340" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://www.riken.jp/press/2019/20191002_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-019-12276-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Human height is a representative phenotype to elucidate genetic architecture. However, the majority of large studies have been performed in European population. To investigate the rare and low-frequency variants associated with height, we construct a reference panel (N = 3,541) for genotype imputation by integrating the whole-genome sequence data from 1,037 Japanese with that of the 1000 Genomes Project, and perform a genome-wide association study in 191,787 Japanese. We report 573 height-associated variants, including 22 rare and 42 low-frequency variants. These 64 variants explain 1.7% of the phenotypic variance. Furthermore, a gene-based analysis identifies two genes with multiple height-increasing rare and low-frequency nonsynonymous variants (SLC27A3 and CYP26B1; PSKAT-O &lt; 2.5 \times 10-6). Our analysis shows a general tendency of the effect sizes of rare variants towards increasing height, which is contrary to findings among Europeans, suggesting that height-associated rare variants are under different selection pressure in Japanese and European populations.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-019-12276-5" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Wuttke2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-019-0407-x" target="_blank" rel="noopener">A catalog of genetic loci associated with kidney function from analyses of a million individuals</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Wuttke, M.,
                    </span>
                    <span class="coauthor">
                      Li, Y.,
                    </span>
                    <span class="coauthor">
                      Li, M.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Sieber, K. B.,
                      Feitosa, M. F.,
                      Gorski, M.,
                      Tin, A.,
                      Wang, L.,
                      Chu, A. Y.,
                      Hoppmann, A.,
                      Kirsten, H.,
                      Giri, A.,
                      Chai, J.-F.,
                      Sveinbjornsson, G.,
                      Tayo, B. O.,
                      Nutile, T.,
                      Fuchsberger, C.,
                      Marten, J.,
                      Cocca, M.,
                      Ghasemi, S.,
                      Xu, Y.,
                      Horn, K.,
                      Noce, D.,
                      Most, P. J.,
                      Sedaghat, S.,
                      Yu, Z.,
                      Akiyama, M.,
                      Afaq, S.,
                      Ahluwalia, T. S.,
                      Almgren, P.,
                      Amin, N.,
                      Ã„rnlÃ¶v, J.,
                      Bakker, S. J. L.,
                      Bansal, N.,
                      Baptista, D.,
                      Bergmann, S.,
                      Biggs, M. L.,
                      Biino, G.,
                      Boehnke, M.,
                      Boerwinkle, E.,
                      Boissel, M.,
                      Bottinger, E. P.,
                      Boutin, T. S.,
                      Brenner, H.,
                      Brumat, M.,
                      Burkhardt, R.,
                      Butterworth, A. S.,
                      Campana, E.,
                      Campbell, A.,
                      Campbell, H.,
                      Canouil, M.,
                      Carroll, R. J.,
                      Catamo, E.,
                      Chambers, J. C.,
                      Chee, M.-L.,
                      Chee, M.-L.,
                      Chen, X.,
                      Cheng, C.-Y.,
                      Cheng, Y.,
                      Christensen, K.,
                      Cifkova, R.,
                      Ciullo, M.,
                      Concas, M. P.,
                      Cook, J. P.,
                      Coresh, J.,
                      Corre, T.,
                      Sala, C. F.,
                      Cusi, D.,
                      Danesh, J.,
                      Daw, E. W.,
                      Borst, M. H.,
                      De Grandi, A.,
                      Mutsert, R.,
                      Vries, A. P. J.,
                      Degenhardt, F.,
                      Delgado, G.,
                      Demirkan, A.,
                      Di Angelantonio, E.,
                      Dittrich, K.,
                      Divers, J.,
                      Dorajoo, R.,
                      Eckardt, K.-U.,
                      Ehret, G.,
                      Elliott, P.,
                      Endlich, K.,
                      Evans, M. K.,
                      Felix, J. F.,
                      Foo, V. H. X.,
                      Franco, O. H.,
                      Franke, A.,
                      Freedman, B. I.,
                      Freitag-Wolf, S.,
                      Friedlander, Y.,
                      Froguel, P.,
                      Gansevoort, R. T.,
                      Gao, H.,
                      Gasparini, P.,
                      Gaziano, J. M.,
                      Giedraitis, V.,
                      Gieger, C.,
                      Girotto, G.,
                      Giulianini, F.,
                      GÃ¶gele, M.,
                      Gordon, S. D.,
                      Gudbjartsson, D. F.,
                      Gudnason, V.,
                      Haller, T.,
                      Hamet, P.,
                      Harris, T. B.,
                      Hartman, C. A.,
                      Hayward, C.,
                      Hellwege, J. N.,
                      Heng, C.-K.,
                      Hicks, A. A.,
                      Hofer, E.,
                      Huang, W.,
                      Hutri-KÃ¤hÃ¶nen, N.,
                      Hwang, S.-J.,
                      Ikram, M. A.,
                      Indridason, O. S.,
                      Ingelsson, E.,
                      Ising, M.,
                      Jaddoe, V. W. V.,
                      Jakobsdottir, J.,
                      Jonas, J. B.,
                      Joshi, P. K.,
                      Josyula, N. S.,
                      Jung, B.,
                      KÃ¤hÃ¶nen, M.,
                      Kamatani, Y.,
                      Kammerer, C. M.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kastarinen, M.,
                      Kerr, S. M.,
                      Khor, C.-C.,
                      Kiess, W.,
                      Kleber, M. E.,
                      Koenig, W.,
                      Kooner, J. S.,
                      KÃ¶rner, A.,
                      Kovacs, P.,
                      Kraja, A. T.,
                      Krajcoviechova, A.,
                      Kramer, H.,
                      KrÃ¤mer, B. K.,
                      Kronenberg, F.,
                      Kubo, M.,
                      KÃ¼hnel, B.,
                      Kuokkanen, M.,
                      Kuusisto, J.,
                      La Bianca, M.,
                      Laakso, M.,
                      Lange, L. A.,
                      Langefeld, C. D.,
                      Lee, J. J.-M.,
                      Lehne, B.,
                      LehtimÃ¤ki, T.,
                      Lieb, W.,
                      Lim, S.-C.,
                      Lind, L.,
                      Lindgren, C. M.,
                      Liu, J.,
                      Liu, J.,
                      Loeffler, M.,
                      Loos, R. J. F.,
                      Lucae, S.,
                      Lukas, M. A.,
                      LyytikÃ¤inen, L.-P.,
                      MÃ¤gi, R.,
                      Magnusson, P. K. E.,
                      Mahajan, A.,
                      Martin, N. G.,
                      Martins, J.,
                      MÃ¤rz, W.,
                      Mascalzoni, D.,
                      Matsuda, K.,
                      Meisinger, C.,
                      Meitinger, T.,
                      Melander, O.,
                      Metspalu, A.,
                      Mikaelsdottir, E. K.,
                      Milaneschi, Y.,
                      Miliku, K.,
                      Mishra, P. P.,
                      Mohlke, K. L.,
                      Mononen, N.,
                      Montgomery, G. W.,
                      Mook-Kanamori, D. O.,
                      Mychaleckyj, J. C.,
                      Nadkarni, G. N.,
                      Nalls, M. A.,
                      Nauck, M.,
                      Nikus, K.,
                      Ning, B.,
                      Nolte, I. M.,
                      Noordam, R.,
                      Oâ€™Connell, J.,
                      Oâ€™Donoghue, M. L.,
                      Olafsson, I.,
                      Oldehinkel, A. J.,
                      Orho-Melander, M.,
                      Ouwehand, W. H.,
                      Padmanabhan, S.,
                      Palmer, N. D.,
                      Palsson, R.,
                      Penninx, B. W. J. H.,
                      Perls, T.,
                      Perola, M.,
                      Pirastu, M.,
                      Pirastu, N.,
                      Pistis, G.,
                      Podgornaia, A. I.,
                      Polasek, O.,
                      Ponte, B.,
                      Porteous, D. J.,
                      Poulain, T.,
                      Pramstaller, P. P.,
                      Preuss, M. H.,
                      Prins, B. P.,
                      Province, M. A.,
                      Rabelink, T. J.,
                      Raffield, L. M.,
                      Raitakari, O. T.,
                      Reilly, D. F.,
                      Rettig, R.,
                      Rheinberger, M.,
                      Rice, K. M.,
                      Ridker, P. M.,
                      Rivadeneira, F.,
                      Rizzi, F.,
                      Roberts, D. J.,
                      Robino, A.,
                      Rossing, P.,
                      Rudan, I.,
                      Rueedi, R.,
                      Ruggiero, D.,
                      Ryan, K. A.,
                      Saba, Y.,
                      Sabanayagam, C.,
                      Salomaa, V.,
                      Salvi, E.,
                      Saum, K.-U.,
                      Schmidt, H.,
                      Schmidt, R.,
                      SchÃ¶ttker, B.,
                      Schulz, C.-A.,
                      Schupf, N.,
                      Shaffer, C. M.,
                      Shi, Y.,
                      Smith, A. V.,
                      Smith, B. H.,
                      Soranzo, N.,
                      Spracklen, C. N.,
                      Strauch, K.,
                      Stringham, H. M.,
                      Stumvoll, M.,
                      Svensson, P. O.,
                      Szymczak, S.,
                      Tai, E.-S.,
                      Tajuddin, S. M.,
                      Tan, N. Y. Q.,
                      Taylor, K. D.,
                      Teren, A.,
                      Tham, Y.-C.,
                      Thiery, J.,
                      Thio, C. H. L.,
                      Thomsen, H.,
                      Thorleifsson, G.,
                      Toniolo, D.,
                      TÃ¶njes, A.,
                      Tremblay, J.,
                      Tzoulaki, I.,
                      Uitterlinden, A. G.,
                      Vaccargiu, S.,
                      Dam, R. M.,
                      Harst, P.,
                      Duijn, C. M.,
                      Velez Edward, D. R.,
                      Verweij, N.,
                      Vogelezang, S.,
                      VÃ¶lker, U.,
                      Vollenweider, P.,
                      Waeber, G.,
                      Waldenberger, M.,
                      Wallentin, L.,
                      Wang, Y. X.,
                      Wang, C.,
                      Waterworth, D. M.,
                      Bin Wei, W.,
                      White, H.,
                      Whitfield, J. B.,
                      Wild, S. H.,
                      Wilson, J. F.,
                      Wojczynski, M. K.,
                      Wong, C.,
                      Wong, T.-Y.,
                      Xu, L.,
                      Yang, Q.,
                      Yasuda, M.,
                      Yerges-Armstrong, L. M.,
                      Zhang, W.,
                      Zonderman, A. B.,
                      Rotter, J. I.,
                      Bochud, M.,
                      Psaty, B. M.,
                      Vitart, V.,
                      Wilson, J. G.,
                      Dehghan, A.,
                      Parsa, A.,
                      Chasman, D. I.,
                      Ho, K.,
                      Morris, A. P.,
                      Devuyst, O.,
                      Akilesh, S.,
                      Pendergrass, S. A.,
                      Sim, X.,
                      BÃ¶ger, C. A.,
                      Okada, Y.,
                      Edwards, T. L.,
                      Snieder, H.,
                      Stefansson, K.,
                      Hung, A. M.,
                      Heid, I. M.,
                      Scholz, M.,
                      Teumer, A.,
                      KÃ¶ttgen, A.,
                    </span>
                    and Pattaro, C.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>51</b>,
                    957â€“972
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqUe" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31152163" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-019-0407-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Chronic kidney disease (CKD) is responsible for a public health burden with multi-systemic complications. Through trans-ancestry meta-analysis of genome-wide association studies of estimated glomerular filtration rate (eGFR) and independent replication (n = 1,046,070), we identified 264 associated loci (166 new). Of these, 147 were likely to be relevant for kidney function on the basis of associations with the alternative kidney function marker blood urea nitrogen (n = 416,178). Pathway and enrichment analyses, including mouse models with renal phenotypes, support the kidney as the main target organ. A genetic risk score for lower eGFR was associated with clinically diagnosed CKD in 452,264 independent individuals. Colocalization analyses of associations with eGFR among 783,978 European-ancestry individuals and gene expression across 46 human tissues, including tubulo-interstitial and glomerular kidney compartments, identified 17 genes differentially expressed in kidney. Fine-mapping highlighted missense driver variants in 11 genes and kidney-specific regulatory variants. These results provide a comprehensive priority list of molecular targets for translational research.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-019-0407-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Nakatochi2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s42003-019-0339-0" target="_blank" rel="noopener">Genome-wide meta-analysis identifies multiple novel loci associated with serum uric acid levels in Japanese individuals</a></div>
                  <div class="author">
                    <span class="coauthor">
                      *Nakatochi, M.,
                    </span>
                    <em>*Kanai, M.</em>,
                    <span class="coauthor">
                      *Nakayama, A.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      *Hishida, A.,
                      *Kawamura, Y.,
                      Ichihara, S.,
                      Akiyama, M.,
                      Ikezaki, H.,
                      Furusyo, N.,
                      Shimizu, S.,
                      Yamamoto, K.,
                      Hirata, M.,
                      Okada, R.,
                      Kawai, S.,
                      Kawaguchi, M.,
                      Nishida, Y.,
                      Shimanoe, C.,
                      Ibusuki, R.,
                      Takezaki, T.,
                      Nakajima, M.,
                      Takao, M.,
                      Ozaki, E.,
                      Matsui, D.,
                      Nishiyama, T.,
                      Suzuki, S.,
                      Takashima, N.,
                      Kita, Y.,
                      Endoh, K.,
                      Kuriki, K.,
                      Uemura, H.,
                      Arisawa, K.,
                      Oze, I.,
                      Matsuo, K.,
                      Nakamura, Y.,
                      Mikami, H.,
                      Tamura, T.,
                      Nakashima, H.,
                      Nakamura, T.,
                      Kato, N.,
                      Matsuda, K.,
                      Murakami, Y.,
                      Matsubara, T.,
                      Naito, M.,
                      Kubo, M.,
                      Kamatani, Y.,
                      Shinomiya, N.,
                      Yokota, M.,
                      Wakai, K.,
                      Okada, Y.,
                    </span>
                    and Matsuo, H.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Communications Biology</em>
                    <b>2</b>,
                    115
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqUo" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/30993211" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://ndmc-ipb.browse.jp/DL/researchNDMC20190408HP.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s42003-019-0339-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Gout is a common arthritis caused by elevated serum uric acid (SUA) levels. Here we investigated loci influencing SUA in a genome-wide meta-analysis with 121,745 Japanese subjects. We identified 8948 variants at 36 genomic loci (P&lt;5 \times 10-8) including eight novel loci. Of these, missense variants of SESN2 and PNPLA3 were predicted to be damaging to the function of these proteins; another five loci-TMEM18, TM4SF4, MXD3-LMAN2, PSORS1C1-PSORS1C2, and HNF4A-are related to cell metabolism, proliferation, or oxidative stress; and the remaining locus, LINC01578, is unknown. We also identified 132 correlated genes whose expression levels are associated with SUA-increasing alleles. These genes are enriched for the UniProt transport term, suggesting the importance of transport-related genes in SUA regulation. Furthermore, trans-ethnic meta-analysis across our own meta-analysis and the Global Urate Genetics Consortium has revealed 15 more novel loci associated with SUA. Our findings provide insight into the pathogenesis, treatment, and prevention of hyperuricemia/gout.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s42003-019-0339-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Martin2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-019-0379-x" target="_blank" rel="noopener">Clinical use of current polygenic risk scores may exacerbate health disparities</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Martin, A. R.,
                    </span>
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Kamatani, Y.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Okada, Y.,
                      Neale, B. M.,
                    </span>
                    and Daly, M. J.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>51</b>,
                    584â€“591
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqUB" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/30926966" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/441261" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <a href="https://github.com/armartin/prs_disparities" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <a href="https://www.broadinstitute.org/news/study-highlights-need-increase-diversity-within-genetic-data-sets" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-019-0379-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Polygenic risk scores (PRS) are poised to improve biomedical outcomes via precision medicine. However, the major ethical and scientific challenge surrounding clinical implementation of PRS is that those available today are several times more accurate in individuals of European ancestry than other ancestries. This disparity is an inescapable consequence of Eurocentric biases in genome-wide association studies, thus highlighting that-unlike clinical biomarkers and prescription drugs, which may individually work better in some populations but do not ubiquitously perform far better in European populations-clinical uses of PRS today would systematically afford greater improvement for European-descent populations. Early diversifying efforts show promise in leveling this vast imbalance, even when non-European sample sizes are considerably smaller than the largest studies to date. To realize the full and equitable potential of PRS, greater diversity must be prioritized in genetic studies, and summary statistics must be publically disseminated to ensure that health disparities are not increased for those individuals already most underserved.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-019-0379-x" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Matoba2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41562-019-0557-y" target="_blank" rel="noopener">GWAS of smoking behaviour in 165,436 Japanese people reveals seven new loci and shared genetic architecture</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Matoba, N.,
                    </span>
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <span class="coauthor">
                      Ishigaki, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Takahashi, A.,
                      Momozawa, Y.,
                      Ikegawa, S.,
                      Ikeda, M.,
                      Iwata, N.,
                      Hirata, M.,
                      Matsuda, K.,
                      Kubo, M.,
                      Okada, Y.,
                    </span>
                    and Kamatani, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Human Behaviour</em>
                    <b>3</b>,
                    471â€“477
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqU0" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/31089300" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2019/20190326_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41562-019-0557-y" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Cigarette smoking is a risk factor for a wide range of human diseases1. To investigate the genetic components associated with smoking behaviours in the Japanese population, we conducted a genome-wide association study of four smoking-related traits using up to 165,436 individuals. In total, we identified seven new loci, including three loci associated with the number of cigarettes per day (EPHX2-CLU, RET and CUX2-ALDH2), three loci associated with smoking initiation (DLC1, CXCL12-TMEM72-AS1 and GALR1-SALL3) and LINC01793-MIR4432HG, associated with the age of smoking initiation. Of these, three loci (LINC01793-MIR4432HG, CXCL12-TMEM72-AS1 and GALR1-SALL3) were found by conducting an additional sex-stratified genome-wide association study. This additional analysis showed heterogeneity of effects between sexes. The cross-sex linkage disequilibrium score regression2,3 analysis also indicated that the genetic component of smoking initiation was significantly different between the sexes. Cross-trait linkage disequilibrium score regression analysis and trait-relevant tissue analysis showed that the number of cigarettes per day has a specific genetic background distinct from those of the other three smoking behaviours. We also report 11 diseases that share genetic basis with smoking behaviours. Although the current study should be carefully considered owing to the lack of replication samples, our findings characterized the genetic architecture of smoking behaviours. Further studies in East Asian populations are warranted to confirm our findings.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41562-019-0557-y" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Suzuki2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-018-0332-4" target="_blank" rel="noopener">Identification of 28 new susceptibility loci for type 2 diabetes in the Japanese population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Suzuki, K.,
                    </span>
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <span class="coauthor">
                      Ishigaki, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Hosoe, J.,
                      Shojima, N.,
                      Hozawa, A.,
                      Kadota, A.,
                      Kuriki, K.,
                      Naito, M.,
                      Tanno, K.,
                      Ishigaki, Y.,
                      Hirata, M.,
                      Matsuda, K.,
                      Iwata, N.,
                      Ikeda, M.,
                      Sawada, N.,
                      Yamaji, T.,
                      Iwasaki, M.,
                      Ikegawa, S.,
                      Maeda, S.,
                      Murakami, Y.,
                      Wakai, K.,
                      Tsugane, S.,
                      Sasaki, M.,
                      Yamamoto, M.,
                      Okada, Y.,
                      Kubo, M.,
                      Kamatani, Y.,
                      Horikoshi, M.,
                      Yamauchi, T.,
                    </span>
                    and Kadowaki, T.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>51</b>,
                    379â€“386
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqU2" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/30718926" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://www.amed.go.jp/news/release_20190205.html" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-018-0332-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>To understand the genetics of type 2 diabetes in people of Japanese ancestry, we conducted A meta-analysis of four genome-wide association studies (GWAS; 36,614 cases and 155,150 controls of Japanese ancestry). We identified 88 type 2 diabetes-associated loci (P 0.6) with the lead variants. Among the 28 missense variants, three previously unreported variants had distinct minor allele frequency (MAF) spectra between people of Japanese and European ancestry (MAFJPN &gt; 0.05 versus MAFEUR &lt; 0.01), including missense variants in genes related to pancreatic acinar cells (GP2) and insulin secretion (GLP1R). Transethnic comparisons of the molecular pathways identified from the GWAS results highlight both ethnically shared and heterogeneous effects of a series of pathways on type 2 diabetes (for example, monogenic diabetes and beta cells).</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-018-0332-4" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Hirata2019" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-018-0336-0" target="_blank" rel="noopener">Genetic and phenotypic landscape of the major histocompatibilty complex region in the Japanese population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Hirata, J.,
                    </span>
                    <span class="coauthor">
                      Hosomichi, K.,
                    </span>
                    <span class="coauthor">
                      Sakaue, S.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Nakaoka, H.,
                      Ishigaki, K.,
                      Suzuki, K.,
                      Akiyama, M.,
                      Kishikawa, T.,
                      Ogawa, K.,
                      Masuda, T.,
                      Yamamoto, K.,
                      Hirata, M.,
                      Matsuda, K.,
                      Momozawa, Y.,
                      Inoue, I.,
                      Kubo, M.,
                      Kamatani, Y.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>51</b>,
                    470â€“480
                    (2019)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqU4" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/30692682" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.med.osaka-u.ac.jp/activities/results/2019year/hirataokada1901" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-018-0336-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>To perform detailed fine-mapping of the major-histocompatibility-complex region, we conducted next-generation sequencing (NGS)-based typing of the 33 human leukocyte antigen (HLA) genes in 1,120 individuals of Japanese ancestry, providing a high-resolution allele catalog and linkage-disequilibrium structure of both classical and nonclassical HLA genes. Together with population-specific deep-whole-genome-sequencing data (n = 1,276), we conducted NGS-based HLA, single-nucleotide-variant and indel imputation of large-scale genome-wide-association-study data from 166,190 Japanese individuals. A phenome-wide association study assessing 106 clinical phenotypes identified abundant, significant genotype-phenotype associations across 52 phenotypes. Fine-mapping highlighted multiple association patterns conferring independent risks from classical HLA genes. Region-wide heritability estimates and genetic-correlation network analysis elucidated the polygenic architecture shared across the phenotypes.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-018-0336-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2018">2018</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2018b" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1093/bioinformatics/bty488" target="_blank" rel="noopener">Grimon: graphical interface to visualize multi-omics networks</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Maeda, Y.,
                    </span>
                    and Okada, Y.
                  </div>
                  <div class="periodical">
                    <em>Bioinformatics</em>
                    <b>34</b>,
                    3934â€“3936
                    (2018)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://academic.oup.com/bioinformatics/article-pdf/34/22/3934/26441493/bty488.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29931190" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://github.com/mkanai/grimon" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1093/bioinformatics/bty488" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Summary: Rapid advances in high-throughput sequencing technologies have enabled more efficient acquisition of massive amount of multi-omics data. However, interpretation of the underlying relationships across multi-omics networks has not been fully succeeded, partly due to the lack of effective methods in visualization. To aid interpretation of the results from such multi-omics data, we here present Grimon (Graphical interface to visualize multi-omics networks), an R package that visualizes high-dimensional multi-layered data sets in three-dimensional parallel coordinates. Grimon enables users to intuitively and interactively explore their analyzed data, helping their understanding of multiple inter-layer connections embedded in high-dimensional complex data. Availability and implementation: Grimon is freely available at https://github.com/mkanai/grimon as an R package with example omics data sets. Supplementary information: Supplementary data are available at bioinformatics online.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1093/bioinformatics/bty488" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Horikoshi2018" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-018-04398-z" target="_blank" rel="noopener">Elucidating the genetic architecture of reproductive ageing in the Japanese population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Horikoshi, M.,
                    </span>
                    <span class="coauthor">
                      Day, F. R.,
                    </span>
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Hirata, M.,
                      Kamatani, Y.,
                      Matsuda, K.,
                      Ishigaki, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Wright, H.,
                      Toro, C. A.,
                      Ojeda, S. R.,
                      Lomniczi, A.,
                      Kubo, M.,
                      Ong, K. K.,
                    </span>
                    and Perry, J. R. B.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>9</b>,
                    1977
                    (2018)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqVk" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29773799" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://doi.org/10.1101/182717" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">bioRxiv</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-018-04398-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Population studies elucidating the genetic architecture of reproductive ageing have been largely limited to European ancestries, restricting the generalizability of the findings and overlooking possible key genes poorly captured by common European genetic variation. Here, we report 26 loci (all P &lt; 5 \times 10-8) for reproductive ageing, i.e. puberty timing or age at menopause, in a non-European population (up to 67,029 women of Japanese ancestry). Highlighted genes for menopause include GNRH1, which supports a primary, rather than passive, role for hypothalamic-pituitary GnRH signalling in the timing of menopause. For puberty timing, we demonstrate an aetiological role for receptor-like protein tyrosine phosphatases by combining evidence across population genetics and pre- and peri-pubertal changes in hypothalamic gene expression in rodent and primate models. Furthermore, our findings demonstrate widespread differences in allele frequencies and effect estimates between Japanese and European associated variants, highlighting the benefits and challenges of large-scale trans-ethnic approaches.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-018-04398-z" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Okada2018" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41467-018-03274-0" target="_blank" rel="noopener">Deep whole-genome sequencing reveals recent selection signatures linked to evolution and disease risk of Japanese</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <span class="coauthor">
                      Momozawa, Y.,
                    </span>
                    <span class="coauthor">
                      Sakaue, S.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Ishigaki, K.,
                      Akiyama, M.,
                      Kishikawa, T.,
                      Arai, Y.,
                      Sasaki, T.,
                      Kosaki, K.,
                      Suematsu, M.,
                      Matsuda, K.,
                      Yamamoto, K.,
                      Kubo, M.,
                      Hirose, N.,
                    </span>
                    and Kamatani, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Communications</em>
                    <b>9</b>,
                    1631
                    (2018)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqVl" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29691385" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2018/20180424_2/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41467-018-03274-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Understanding natural selection is crucial to unveiling evolution of modern humans. Here, we report natural selection signatures in the Japanese population using 2234 high-depth whole-genome sequence (WGS) data (25.9\times). Using rare singletons, we identify signals of very recent selection for the past 2000-3000 years in multiple loci (ADH cluster, MHC region, BRAP-ALDH2, SERHL2). In large-scale genome-wide association study (GWAS) dataset (n = 171,176), variants with selection signatures show enrichment in heterogeneity of derived allele frequency spectra among the geographic regions of Japan, highlighted by two major regional clusters (Hondo and Ryukyu). While the selection signatures do not show enrichment in archaic hominin-derived genome sequences, they overlap with the SNPs associated with the modern human traits. The strongest overlaps are observed for the alcohol or nutrition metabolism-related traits. Our study illustrates the value of high-depth WGS to understand evolution and their relationship with disease risk.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-018-03274-0" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Malik2018" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-018-0058-3" target="_blank" rel="noopener">Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Malik, R.,
                    </span>
                    <span class="coauthor">
                      Chauhan, G.,
                    </span>
                    <span class="coauthor">
                      Traylor, M.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Sargurupremraj, M.,
                      Okada, Y.,
                      Mishra, A.,
                      Rutten-Jacobs, L.,
                      Giese, A.-K.,
                      Laan, S. W.,
                      Gretarsdottir, S.,
                      Anderson, C. D.,
                      Chong, M.,
                      Adams, H. H. H.,
                      Ago, T.,
                      Almgren, P.,
                      Amouyel, P.,
                      Ay, H.,
                      Bartz, T. M.,
                      Benavente, O. R.,
                      Bevan, S.,
                      Boncoraglio, G. B.,
                      Brown, R. D.,
                      Butterworth, A. S.,
                      Carrera, C.,
                      Carty, C. L.,
                      Chasman, D. I.,
                      Chen, W.-M.,
                      Cole, J. W.,
                      Correa, A.,
                      Cotlarciuc, I.,
                      Cruchaga, C.,
                      Danesh, J.,
                      Bakker, P. I. W.,
                      DeStefano, A. L.,
                      Hoed, M.,
                      Duan, Q.,
                      Engelter, S. T.,
                      Falcone, G. J.,
                      Gottesman, R. F.,
                      Grewal, R. P.,
                      Gudnason, V.,
                      Gustafsson, S.,
                      Haessler, J.,
                      Harris, T. B.,
                      Hassan, A.,
                      Havulinna, A. S.,
                      Heckbert, S. R.,
                      Holliday, E. G.,
                      Howard, G.,
                      Hsu, F.-C.,
                      Hyacinth, H. I.,
                      Ikram, M. A.,
                      Ingelsson, E.,
                      Irvin, M. R.,
                      Jian, X.,
                      JimÃ©nez-Conde, J.,
                      Johnson, J. A.,
                      Jukema, J. W.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Keene, K. L.,
                      Kissela, B. M.,
                      Kleindorfer, D. O.,
                      Kooperberg, C.,
                      Kubo, M.,
                      Lange, L. A.,
                      Langefeld, C. D.,
                      Langenberg, C.,
                      Launer, L. J.,
                      Lee, J.-M.,
                      Lemmens, R.,
                      Leys, D.,
                      Lewis, C. M.,
                      Lin, W.-Y.,
                      Lindgren, A. G.,
                      Lorentzen, E.,
                      Magnusson, P. K.,
                      Maguire, J.,
                      Manichaikul, A.,
                      McArdle, P. F.,
                      Meschia, J. F.,
                      Mitchell, B. D.,
                      Mosley, T. H.,
                      Nalls, M. A.,
                      Ninomiya, T.,
                      Oâ€™Donnell, M. J.,
                      Psaty, B. M.,
                      Pulit, S. L.,
                      RannikmÃ¤e, K.,
                      Reiner, A. P.,
                      Rexrode, K. M.,
                      Rice, K.,
                      Rich, S. S.,
                      Ridker, P. M.,
                      Rost, N. S.,
                      Rothwell, P. M.,
                      Rotter, J. I.,
                      Rundek, T.,
                      Sacco, R. L.,
                      Sakaue, S.,
                      Sale, M. M.,
                      Salomaa, V.,
                      Sapkota, B. R.,
                      Schmidt, R.,
                      Schmidt, C. O.,
                      Schminke, U.,
                      Sharma, P.,
                      Slowik, A.,
                      Sudlow, C. L. M.,
                      Tanislav, C.,
                      Tatlisumak, T.,
                      Taylor, K. D.,
                      Thijs, V. N. S.,
                      Thorleifsson, G.,
                      Thorsteinsdottir, U.,
                      Tiedt, S.,
                      Trompet, S.,
                      Tzourio, C.,
                      Duijn, C. M.,
                      Walters, M.,
                      Wareham, N. J.,
                      Wassertheil-Smoller, S.,
                      Wilson, J. G.,
                      Wiggins, K. L.,
                      Yang, Q.,
                      Yusuf, S.,
                      Bis, J. C.,
                      Pastinen, T.,
                      Ruusalepp, A.,
                      Schadt, E. E.,
                      Koplev, S.,
                      BjÃ¶rkegren, J. L. M.,
                      Codoni, V.,
                      Civelek, M.,
                      Smith, N. L.,
                      TrÃ©gouÃ«t, D. A.,
                      Christophersen, I. E.,
                      Roselli, C.,
                      Lubitz, S. A.,
                      Ellinor, P. T.,
                      Tai, E. S.,
                      Kooner, J. S.,
                      Kato, N.,
                      He, J.,
                      Harst, P.,
                      Elliott, P.,
                      Chambers, J. C.,
                      Takeuchi, F.,
                      Johnson, A. D.,
                      Sanghera, D. K.,
                      Melander, O.,
                      Jern, C.,
                      Strbian, D.,
                      Fernandez-Cadenas, I.,
                      Longstreth, W. T.,
                      Rolfs, A.,
                      Hata, J.,
                      Woo, D.,
                      Rosand, J.,
                      Pare, G.,
                      Hopewell, J. C.,
                      Saleheen, D.,
                      Stefansson, K.,
                      Worrall, B. B.,
                      Kittner, S. J.,
                      Seshadri, S.,
                      Fornage, M.,
                      Markus, H. S.,
                      Howson, J. M. M.,
                      Kamatani, Y.,
                      Debette, S.,
                    </span>
                    and Dichgans, M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>50</b>,
                    524â€“537
                    (2018)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqVm" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29531354" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2018/20180313_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-018-0058-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Stroke has multiple etiologies, but the underlying genes and pathways are largely unknown. We conducted a multiancestry genome-wide-association meta-analysis in 521,612 individuals (67,162 cases and 454,450 controls) and discovered 22 new stroke risk loci, bringing the total to 32. We further found shared genetic variation with related vascular traits, including blood pressure, cardiac traits, and venous thromboembolism, at individual loci (n = 18), and using genetic risk scores and linkage-disequilibrium-score regression. Several loci exhibited distinct association and pleiotropy patterns for etiological stroke subtypes. Eleven new susceptibility loci indicate mechanisms not previously implicated in stroke pathophysiology, with prioritization of risk variants and genes accomplished through bioinformatics analyses using extensive functional datasets. Stroke risk loci were significantly enriched in drug targets for antithrombotic therapy.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-018-0058-3" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2018a" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/s41588-018-0047-6" target="_blank" rel="noopener">Genetic analysis of quantitative traits in the Japanese population links cell types to complex human diseases</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <span class="coauthor">
                      Takahashi, A.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Matoba, N.,
                      Momozawa, Y.,
                      Ikeda, M.,
                      Iwata, N.,
                      Ikegawa, S.,
                      Hirata, M.,
                      Matsuda, K.,
                      Kubo, M.,
                      Okada, Y.,
                    </span>
                    and Kamatani, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>50</b>,
                    390â€“400
                    (2018)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/clzFn" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29403010" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2018/20180206_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/s41588-018-0047-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Clinical measurements can be viewed as useful intermediate phenotypes to promote understanding of complex human diseases. To acquire comprehensive insights into the underlying genetics, here we conducted a genome-wide association study (GWAS) of 58 quantitative traits in 162,255 Japanese individuals. Overall, we identified 1,407 trait-associated loci (P &lt; 5.0 \times 10-8), 679 of which were novel. By incorporating 32 additional GWAS results for complex diseases and traits in Japanese individuals, we further highlighted pleiotropy, genetic correlations, and cell-type specificity across quantitative traits and diseases, which substantially expands the current understanding of the associated genetics and biology. This study identified both shared polygenic effects and cell-type specificity, represented by the genetic links among clinical measurements, complex diseases, and relevant cell types. Our findings demonstrate that even without prior biological knowledge of cross-phenotype relationships, genetics corresponding to clinical measurements successfully recapture those measurementsâ€™ relevance to diseases, and thus can contribute to the elucidation of unknown etiology and pathogenesis.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41588-018-0047-6" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Hirata2018" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.jid.2017.10.001" target="_blank" rel="noopener">Variants at HLA-A , HLA-C , and HLA-DQB1 confer risk of psoriasis vulgaris in Japanese</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Hirata, J.,
                    </span>
                    <span class="coauthor">
                      Hirota, T.,
                    </span>
                    <span class="coauthor">
                      Ozeki, T.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Sudo, T.,
                      Tanaka, T.,
                      Hizawa, N.,
                      Nakagawa, H.,
                      Sato, S.,
                      Mushiroda, T.,
                      Saeki, H.,
                      Tamari, M.,
                    </span>
                    and Okada, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Journal of Investigative Dermatology</em>
                    <b>138</b>,
                    542â€“548
                    (2018)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.jidonline.org/action/showPdf?pii=S0022-202X%2817%2933037-3" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/29031612" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.jid.2017.10.001" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Psoriasis vulgaris (PsV) is an autoimmune disease of skin and joints with heterogeneity in epidemiologic and genetic landscapes of global populations. We conducted an initial genome-wide association study and a replication study of PsV in the Japanese population (606 PsV cases and 2,052 controls). We identified significant associations of the single nucleotide polymorphisms with PsV risk at TNFAIP3-interacting protein 1and the major histocompatibility complex region (P = 3.7 \times 10-10 and 6.6 \times 10-15, respectively). By updating the HLA imputation reference panel of Japanese (n = 908) to expand HLA gene coverage, we fine-mapped the HLA variants associated with PsV risk. Although we confirmed the PsV risk of HLA-C*06:02 (odds ratio = 6.36, P = 0.0015), its impact was relatively small compared with those in other populations due to rare allele frequency in Japanese (0.4% in controls). Alternatively, HLA-A*02:07, which corresponds to the cysteine residue at HLA-A amino acid position 99 (HLA-A Cys99), demonstrated the most significant association with PsV (odds ratio = 4.61, P = 1.2 \times 10-10). In addition to HLA-A*02:07 and HLA-C*06:02, stepwise conditional analysis identified an independent PsV risk of HLA-DQ\beta1 Asp57 (odds ratio = 2.19, P = 1.9 \times 10-6). Our PsV genome-wide association study in Japanese highlighted the genetic architecture of PsV, including the identification of HLA risk variants.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.jid.2017.10.001" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2017">2017</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Akiyama2017" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/ng.3951" target="_blank" rel="noopener">Genome-wide association study identifies 112 new loci for body mass index in the Japanese population</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Akiyama, M.,
                    </span>
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Takahashi, A.,
                      Momozawa, Y.,
                      Ikeda, M.,
                      Iwata, N.,
                      Ikegawa, S.,
                      Hirata, M.,
                      Matsuda, K.,
                      Iwasaki, M.,
                      Yamaji, T.,
                      Sawada, N.,
                      Hachiya, T.,
                      Tanno, K.,
                      Shimizu, A.,
                      Hozawa, A.,
                      Minegishi, N.,
                      Tsugane, S.,
                      Yamamoto, M.,
                      Kubo, M.,
                    </span>
                    and Kamatani, Y.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>49</b>,
                    1458â€“1467
                    (2017)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqVZ" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28892062" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2017/20170912_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/ng.3951" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Obesity is a risk factor for a wide variety of health problems. In a genome-wide association study (GWAS) of body mass index (BMI) in Japanese people (n = 173,430), we found 85 loci significantly associated with obesity (P &lt; 5.0 \times 10-8), of which 51 were previously unknown. We conducted trans-ancestral meta-analyses by integrating these results with the results from a GWAS of Europeans and identified 61 additional new loci. In total, this study identifies 112 novel loci, doubling the number of previously known BMI-associated loci. By annotating associated variants with cell-type-specific regulatory marks, we found enrichment of variants in CD19+ cells. We also found significant genetic correlations between BMI and lymphocyte count (P = 6.46 \times 10-5, rg = 0.18) and between BMI and multiple complex diseases. These findings provide genetic evidence that lymphocytes are relevant to body weight regulation and offer insights into the pathogenesis of obesity.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/ng.3951" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Sudo2017" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1177/0022034517715432" target="_blank" rel="noopener">Association of NOD2 Mutations with Aggressive Periodontitis</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Sudo, T.,
                    </span>
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <span class="coauthor">
                      Ozaki, K.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Urayama, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kobayashi, H.,
                      Gokyu, M.,
                      Izumi, Y.,
                    </span>
                    and Tanaka, T.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Journal of Dental Research</em>
                    <b>96</b>,
                    1100â€“1105
                    (2017)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28682159" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.tmd.ac.jp/press-release/20170706_1/index.html" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1177/0022034517715432" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Aggressive periodontitis (AgP) is characterized by rapid alveolar bone destruction and tooth loss early in life, and its etiology remains unclear. To explore the genetic risk factors of AgP, we performed genome-wide single-nucleotide polymorphism genotyping for identity-by-descent mapping and identified 32 distinct candidate loci, followed by whole exome sequencing with 2 pedigrees of AgP consisting of 3 cases and 1 control in 1 family and 2 sibling cases in the other. After variant filtering procedures and validation by targeted Sanger sequencing, we identified 2 missense mutations at 16q12 in NOD2 (p.Ala110Thr and p.Arg311Trp), which encodes nucleotide-binding oligomerization domain protein 2. We further examined 94 genetically unrelated AgP patients by targeted sequencing of NOD2 and found that 2 patients among them also carried the p.Arg311Trp variant. Furthermore, we found 3 additional missense mutations in this gene (p.His370Tyr, p.Arg459Cys, and p.Ala868Thr). These mutations either had not been previously observed or are extremely rare (frequency &lt;0.001) in Asian populations. NOD2 plays a crucial role in innate immunity as an intracellular receptor initiating nuclear factor \kappaB-dependent and mitogen-activated protein kinase-dependent gene transcription. These results demonstrated NOD2 as a novel gene involved in AgP.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1177/0022034517715432" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2016">2016</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Okada2016b" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1016/j.ajhg.2016.06.019" target="_blank" rel="noopener">Contribution of a Non-classical HLA Gene, HLA-DOA, to the Risk of Rheumatoid Arthritis</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <span class="coauthor">
                      Suzuki, A.,
                    </span>
                    <span class="coauthor">
                      Ikari, K.,
                    </span>
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Terao, C.,
                      Kochi, Y.,
                      Ohmura, K.,
                      Higasa, K.,
                      Akiyama, M.,
                      Ashikawa, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Hirata, J.,
                      Suita, N.,
                      Teo, Y.-Y.,
                      Xu, H.,
                      Bae, S.-C.,
                      Takahashi, A.,
                      Momozawa, Y.,
                      Matsuda, K.,
                      Momohara, S.,
                      Taniguchi, A.,
                      Yamada, R.,
                      Mimori, T.,
                      Kubo, M.,
                      Brown, M. A.,
                      Raychaudhuri, S.,
                      Matsuda, F.,
                      Yamanaka, H.,
                      Kamatani, Y.,
                    </span>
                    and Yamamoto, K.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>The American Journal of Human Genetics</em>
                    <b>99</b>,
                    366â€“374
                    (2016)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://www.cell.com/ajhg/pdfExtended/S0002-9297(16)30223-3" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27486778" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2016/20160805_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1016/j.ajhg.2016.06.019" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>Despite the progress in human leukocyte antigen (HLA) causal variant mapping, independent localization of major histocompatibility complex (MHC) risk from classical HLA genes is challenging. Here, we conducted a large-scale MHC fine-mapping analysis of rheumatoid arthritis (RA) in a Japanese population (6,244 RA cases and 23,731 controls) population by using HLA imputation, followed by a multi-ethnic validation study including east Asian and European populations (n = 7,097 and 23,149, respectively). Our study identified an independent risk of a synonymous mutation at HLA-DOA, a non-classical HLA gene, on anti-citrullinated protein autoantibody (ACPA)-positive RA risk (p = 1.4 \times 10(-9)), which demonstrated a cis-expression quantitative trait loci (cis-eQTL) effect on HLA-DOA expression. Trans-ethnic comparison revealed different linkage disequilibrium (LD) patterns in HLA-DOA and HLA-DRB1, explaining the observed HLA-DOA variant risk heterogeneity among ethnicities, which was most evident in the Japanese population. Although previous HLA fine-mapping studies have identified amino acid polymorphisms of the classical HLA genes as driving genetic susceptibility to disease, our study additionally identifies the dosage contribution of a non-classical HLA gene to disease etiology. Our study contributes to the understanding of HLA immunology in human diseases and suggests the value of incorporating additional ancestry in MHC fine-mapping.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.ajhg.2016.06.019" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Kanai2016" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/jhg.2016.72" target="_blank" rel="noopener">Empirical estimation of genome-wide significance thresholds based on the 1000 Genomes Project data set</a></div>
                  <div class="author">
                    <em>Kanai, M.</em>,
                    <span class="coauthor">
                      Tanaka, T.,
                    </span>
                    and Okada, Y.
                  </div>
                  <div class="periodical">
                    <em>Journal of Human Genetics</em>
                    <b>61</b>,
                    861â€“866
                    (2016)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqWl" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27305981" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="https://github.com/mkanai/GWAS_threshold_scripts" class="btn btn-sm z-depth-0" role="button" target="_blank">Code</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/jhg.2016.72" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>To assess the statistical significance of associations between variants and traits, genome-wide association studies (GWAS) should employ an appropriate threshold that accounts for the massive burden of multiple testing in the study. Although most studies in the current literature commonly set a genome-wide significance threshold at the level of P=5.0 \times 10-8, the adequacy of this value for respective populations has not been fully investigated. To empirically estimate thresholds for different ancestral populations, we conducted GWAS simulations using the 1000 Genomes Phase 3 data set for Africans (AFR), Europeans (EUR), Admixed Americans (AMR), East Asians (EAS) and South Asians (SAS). The estimated empirical genome-wide significance thresholds were Psig=3.24 \times 10-8 (AFR), 9.26 \times 10-8 (EUR), 1.83 \times 10-7 (AMR), 1.61 \times 10-7 (EAS) and 9.46 \times 10-8 (SAS). We additionally conducted trans-ethnic meta-analyses across all populations (ALL) and all populations except for AFR (\DeltaAFR), which yielded Psig=3.25 \times 10-8 (ALL) and 4.20 \times 10-8 (\DeltaAFR). Our results indicate that the current threshold (P=5.0 \times 10-8) is overly stringent for all ancestral populations except for Africans; however, we should employ a more stringent threshold when conducting a meta-analysis, regardless of the presence of African samples.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/jhg.2016.72" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Okada2016a" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/srep22223" target="_blank" rel="noopener">Significant impact of miRNAâ€“target gene networks on genetics of human complex traits</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <span class="coauthor">
                      Muramatsu, T.,
                    </span>
                    <span class="coauthor">
                      Suita, N.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Kawakami, E.,
                      Iotchkova, V.,
                      Soranzo, N.,
                      Inazawa, J.,
                    </span>
                    and Tanaka, T.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Scientific Reports</em>
                    <b>6</b>,
                    22223
                    (2016)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqWo" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/26927695" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.tmd.ac.jp/archive-tmdu/kouhou/20160301.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/srep22223" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>The impact of microRNA (miRNA) on the genetics of human complex traits, especially in the context of miRNA-target gene networks, has not been fully assessed. Here, we developed a novel analytical method, MIGWAS, to comprehensively evaluate enrichment of genome-wide association study (GWAS) signals in miRNA-target gene networks. We applied the method to the GWAS results of the 18 human complex traits from &gt;1.75 million subjects, and identified significant enrichment in rheumatoid arthritis (RA), kidney function, and adult height (P &lt; 0.05/18 = 0.0028, most significant enrichment in RA with P = 1.7 \times 10(-4)). Interestingly, these results were consistent with current literature-based knowledge of the traits on miRNA obtained through the NCBI PubMed database search (adjusted P = 0.024). Our method provided a list of miRNA and target gene pairs with excess genetic association signals, part of which included drug target genes. We identified a miRNA (miR-4728-5p) that downregulates PADI2, a novel RA risk gene considered as a promising therapeutic target (rs761426, adjusted P = 2.3 \times 10(-9)). Our study indicated the significant impact of miRNA-target gene networks on the genetics of human complex traits, and provided resources which should contribute to drug discovery and nucleic acid medicine.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/srep22223" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
          <h2 class="year" id="2015">2015</h2>
          <ol class="bibliography">
            <li>
              <div class="row">
                <div class="col-sm-1 abbr">
                </div>
                <div id="Okada2015" class="col-sm-8">
                  <div class="title"><a href="https://doi.org/10.1038/ng.3310" target="_blank" rel="noopener">Construction of a population-specific HLA imputation reference panel and its application to Gravesâ€™ disease risk in Japanese</a></div>
                  <div class="author">
                    <span class="coauthor">
                      Okada, Y.,
                    </span>
                    <span class="coauthor">
                      Momozawa, Y.,
                    </span>
                    <span class="coauthor">
                      Ashikawa, K.,
                    </span>
                    <em>Kanai, M.</em>,
                    <a class="coauthor etal visible" role="button">...,</a>
                    <span class="coauthor etal">
                      Matsuda, K.,
                      Kamatani, Y.,
                      Takahashi, A.,
                    </span>
                    and Kubo, M.
                    <a class="coauthor etal" role="button">[Show fewer authors]</a>
                  </div>
                  <div class="periodical">
                    <em>Nature Genetics</em>
                    <b>47</b>,
                    798â€“802
                    (2015)
                  </div>
                  <div class="links">
                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                    <a href="https://rdcu.be/cmqWr" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/26029868" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener">PubMed</a>
                    <a href="http://www.riken.jp/pr/press/2015/20150602_1/" class="btn btn-sm z-depth-0" role="button" target="_blank">Press</a>
                    <div class="altmetric-badge">
                      <div data-badge-popover="bottom" data-badge-type="1" data-doi="10.1038/ng.3310" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                    </div>
                  </div>
                  <!-- Hidden abstract block -->
                  <div class="abstract hidden">
                    <p>To fine map association signals of human leukocyte antigen (HLA) variants in the major histocompatibility complex (MHC) region, we constructed a Japanese population-specific reference panel (n = 908). We conducted trans-ancestry comparisons of linkage disequilibrium (LD) and haplotype structure for HLA variants using an entropy-based LD measurement, É›, and a visualization tool to capture high-dimensional variables. Our Japanese reference panel exhibited stronger LD between HLA genes than European or other East Asian populations, characterized by one population-specific common long-range HLA haplotype. We applied HLA imputation to genome-wide association study (GWAS) data for Gravesâ€™ disease in Japanese (n = 9,003) and found that amino acid polymorphisms of multiple class I and class II HLA genes independently contribute to disease risk (HLA-DPB1, HLA-A, HLA-B and HLA-DRB1; P &lt; 2.3 \times 10(-6)), with the strongest impact at HLA-DPB1 (P = 1.6 \times 10(-42)). Our study illustrates the value of population-specific HLA reference panels.</p>
                  </div>
                </div>
                <div class="col-sm-1">
                  <div class="altmetric-donut">
                    <div data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/ng.3310" data-hide-no-mentions="true" data-hide-less-than="50" class="altmetric-embed"></div>
                  </div>
                </div>
              </div>
            </li>
          </ol>
        </div>
      </article>
    </div>
  </div>
  <!-- Footer -->
  <div class="social">
    <div class="contact-icons">
      <a href="mailto:%6D%6B%61%6E%61%69@%62%72%6F%61%64%69%6E%73%74%69%74%75%74%65.%6F%72%67"><i class="fas fa-envelope"></i></a>
      <a href="https://orcid.org/0000-0001-5165-4408" target="_blank" title="ORCID"><i class="ai ai-orcid"></i></a>
      <a href="https://scholar.google.com/citations?user=jb3AUMsAAAAJ" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
      <a href="https://github.com/mkanai" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
      <a href="https://www.linkedin.com/in/masakanai" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
      <a href="https://twitter.com/masakanai" target="_blank" title="Twitter"><i class="fab fa-twitter"></i></a>
      <a href="/assets/pdf/CV.pdf" target="_blank" title="CV"><i class="ai ai-cv"></i></a>
    </div>
  </div>
  <footer class="fixed-bottom">
    <div class="container mt-0">
      &copy; Copyright 2026 Masahiro  Kanai.
      Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank">al-folio</a> theme.
      Last updated: <a href="https://github.com/mkanai/mkanai.github.io/commits/master" rel="noopener" target="_blank">February 26, 2026</a>.
    </div>
  </footer>
</body>
<!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>
<!-- Load Altmetric -->
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  $(document).ready(function() {
    var navbarHeight = $('#navbar').outerHeight(true);
    $('body').css({ 'padding-top': navbarHeight });
    $('progress-container').css({ 'padding-top': navbarHeight });
    var progressBar = $('#progress');
    progressBar.css({ 'top': navbarHeight });
    var getMax = function() { return $(document).height() - $(window).height(); }
    var getValue = function() { return $(window).scrollTop(); }
    // Check if the browser supports the progress element.
    if ('max' in document.createElement('progress')) {
      // Set the 'max' attribute for the first time.
      progressBar.attr({ max: getMax() });
      progressBar.attr({ value: getValue() });

      $(document).on('scroll', function() {
        // On scroll only the 'value' attribute needs to be calculated.
        progressBar.attr({ value: getValue() });
      });

      $(window).resize(function() {
        var navbarHeight = $('#navbar').outerHeight(true);
        $('body').css({ 'padding-top': navbarHeight });
        $('progress-container').css({ 'padding-top': navbarHeight });
        progressBar.css({ 'top': navbarHeight });
        // On resize, both the 'max' and 'value' attributes need to be calculated.
        progressBar.attr({ max: getMax(), value: getValue() });
      });
    } else {
      var max = getMax(), value, width;
      var getWidth = function() {
        // Calculate the window width as a percentage.
        value = getValue();
        width = (value/max) * 100;
        width = width + '%';
        return width;
      }
      var setWidth = function() { progressBar.css({ width: getWidth() }); };
      setWidth();
      $(document).on('scroll', setWidth);
      $(window).on('resize', function() {
        // Need to reset the 'max' attribute.
        max = getMax();
        setWidth();
      });
    }
  });
</script>
<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>
<!-- Load search bib -->
<script src="/assets/js/search_bibliographies.js"></script>
</html>